# DNA Extraction and Polymerase Chain Reaction 

## DNA
Deoxyribonucleic acid DNA) is a thread-like chain of nucleotides carrying the genetic instructions used in the growth, development, functioning and reproduction of all known living organisms and many viruses. DNA and ribonucleic acid (RNA) are nucleic acids; alongside proteins, lipids and complex carbohydrates (polysaccharides), they are one of the four major types of macromolecules that are essential for all known forms of life. Most DNA molecules consist of two biopolymer strands coiled around each other to form a double helix.

The two DNA strands are called polynucleotides since they are composed of simpler monomer units called nucleotides. Each nucleotide is composed of one of four nitrogen-containing nucleobases (cytosine [C], guanine [G], adenine [A] or thymine [T]), a sugar called deoxyribose, and a phosphate group. The nucleotides are joined to one another in a chain by covalent bonds between the sugar of one nucleotide and the phosphate of the next, resulting in an alternating sugar-phosphate backbone. The nitrogenous bases of the two separate polynucleotide strands are bound together, according to base pairing rules (A with T and C with G), with hydrogen bonds to make double-stranded DNA.

The complementary nitrogenous bases are divided into two groups, pyrimidines and purines. In a DNA molecule, the pyrimidines are thymine and cytosine, the purines are adenine and guanine.

DNA stores biological information. The DNA backbone is resistant to cleavage, and both strands of the double-stranded structure store the same biological information. This information is replicated as and when the two strands separate. A large part of DNA (more than 98% for humans) is non-coding, meaning that these sections do not serve as patterns for protein sequences.

The two strands of DNA run in opposite directions to each other and are thus antiparallel. Attached to each sugar is one of four types of nucleobases (informally, bases). It is the sequence of these four nucleobases along the backbone that encodes biological information. RNA strands are created using DNA strands as a template in a process called transcription. Under the genetic code, these RNA strands are translated to specify the sequence of amino acids within proteins in a process called translation.

Within eukaryotic cells, DNA is organized into long structures called chromosomes. During cell division these chromosomes are duplicated in the process of DNA replication, providing each cell its own complete set of chromosomes. Eukaryotic organisms (animals, plants, fungi and protists) store most of their DNA inside the cell nucleus and some of their DNA in organelles, such as mitochondria or chloroplasts. In contrast prokaryotes (bacteria and archaea) store their DNA only in the cytoplasm. Within the eukaryotic chromosomes, chromatin proteins such as histones compact and organize DNA. These compact structures guide the interactions between DNA and other proteins, helping control which parts of the DNA are transcribed.

DNA was first isolated by [Friedrich Miescher](https://en.wikipedia.org/wiki/Friedrich_Miescher) in 1869. Its molecular structure was first identified by [James Watson](https://en.wikipedia.org/wiki/James_Watsoni) and [Francis Crick](https://en.wikipedia.org/wiki/Francis_Crick) at the Cavendish Laboratory within the University of Cambridge in 1953, whose model-building efforts were guided by X-ray diffraction data acquired by [Raymond Gosling](https://en.wikipedia.org/wiki/Raymond_Gosling), who was a post-graduate student of [Rosalind Franklin](https://en.wikipedia.org/wiki/Rosalind_Franklin).

DNA isolation is a process of purification of DNA from sample using a combination of physical and chemical methods. Currently it is a routine procedure in molecular biology. 


## Polymerase chain reaction

[Polymerase chain reaction](https://en.wikipedia.org/wiki/Polymerase_chain_reaction) (PCR) is a technique used in molecular biology to amplify a single copy or a few copies of a segment of DNA across several orders of magnitude, generating thousands to millions of copies of a particular DNA sequence. PCR was developed in 1983 by [Kary Mullis](https://en.wikipedia.org/wiki/Kary_Mullis) when he was an employee of the [Cetus Corporation](https://en.wikipedia.org/wiki/Cetus_Corporation), one of the first biotech companies. He won the Nobel Prize in Chemistry in 1993 for the invention of PCR. It is an easy, cheap, and reliable way to repeatedly replicate a focused segment of DNA, a concept which is applicable to numerous fields in modern biology and related sciences. PCR is probably the most widely used technique in molecular biology and has revolutionized the field.

PCR is now a common and often indispensable technique used in clinical and research laboratories for a broad variety of applications. These include DNA cloning for sequencing, gene cloning and manipulation, gene mutagenesis; construction of DNA-based phylogenies, or functional analysis of genes; diagnosis and monitoring of hereditary diseases; amplification of ancient DNA; analysis of genetic fingerprints for DNA profiling (for example, in forensic science and parentage testing); and detection of pathogens in nucleic acid tests for the diagnosis of infectious diseases. In 1993, Mullis was awarded the Nobel Prize in Chemistry along with Michael Smith for his work on PCR.

PCR is now a common and often indispensable technique used in clinical and research laboratories for a broad variety of applications. These include DNA cloning for sequencing, gene cloning and manipulation, gene mutagenesis; construction of DNA-based phylogenies, or functional analysis of genes; diagnosis and monitoring of hereditary diseases; amplification of ancient DNA; analysis of genetic fingerprints for DNA profiling (for example, in forensic science and parentage testing); and detection of pathogens in nucleic acid tests for the diagnosis of infectious diseases.

The vast majority of PCR methods rely on thermal cycling, which involves exposing the reactants to cycles of repeated heating and cooling, permitting different temperature-dependent reactions—specifically, DNA melting and enzyme-driven DNA replication—to quickly proceed many times in sequence. Primers (short DNA fragments) containing sequences complementary to the target region, along with a [DNA polymerase](https://en.wikipedia.org/wiki/DNA_polymerase) (e.g. Taq polymerase), after which the method is named, enable selective and repeated amplification. As PCR progresses, the DNA generated is itself used as a template for replication, setting in motion a chain reaction in which the original DNA template is exponentially amplified. The simplicity of the basic principle underlying PCR means it can be extensively modified to perform a wide array of genetic manipulations. PCR is not generally considered to be a recombinant DNA method, as it does not involve cutting and pasting DNA, only amplification of existing sequences.

Almost all PCR applications employ a heat-stable DNA polymerase, such as Taq polymerase, an enzyme originally isolated from the thermophilic bacterium *Thermus aquaticus*. If heat-susceptible DNA polymerase is used, it will denature every cycle at the denaturation step. Before the use of Taq polymerase, DNA polymerase had to be manually added every cycle, which was a tedious and costly process. This DNA polymerase enzymatically assembles a new DNA strand from free nucleotides, the building blocks of DNA, by using single-stranded DNA as a template and DNA oligonucleotides (the primers mentioned above) to initiate DNA synthesis.

In the first step, the two strands of the DNA double helix are physically separated at a high temperature in a process called DNA melting. In the second step, the temperature is lowered and the two DNA strands become templates for DNA polymerase to selectively amplify the target DNA. The selectivity of PCR results from the use of primers that are complementary to sequence around the DNA region targeted for amplification under specific thermal cycling conditions.

The PCR, like recombinant DNA technology, has had an enormous impact in both basic and diagnostic aspects of molecular biology because it can produce large amounts of a specific DNA fragment from small amounts of a complex template. [Recombinant DNA](https://en.wikipedia.org/wiki/Recombinant_DNA) techniques create molecular clones by conferring on a specific sequence the ability to replicate by inserting it into a vector and introducing the vector into a host cell. PCR represents a form of “in vitro cloning” that can generate, as well as modify, DNA fragments of defined length and sequence in a simple automated reaction. In addition to its many applications in basic molecular biological research, PCR promises to play a critical role in the identification of medically important sequences as well as an important diagnostic one in their detection.

## DNA Extraction
## Experimental Procedures

1. Obtain two microcentrifuge tubes and label them with your initials. 
2. Add 200 µl of lysis buffer into each tube.
3. Weigh 50–100 mg of plant tissue. Record the weight of the tissue.
   
Table: (\#table:plant) Plant material record.

|Plant Name|Part of Plant (Leaf, Root, etc.)|Weight (mg)|
|:---------|:-------------------------------|:----------|
|	         | 		                    		    |           |
|	         | 		                    		    |           |
|	         | 		                    		    |           |
|	         | 		                    		    |           |

4. Chop the plant tissue into small (1-2 mm in diameter) pieces using a razor blade. 
5. Add the plant material to the microcentrifuge tubes with the lysis buffer.
5. Using a micropestle, carefully grind the plant tissue into fine particles (until no more chunks are visible). Be careful not to spill the lysis buffer. If any material gets stuck at the bottom of the tube use a clean pipet tip to dislodge it and continue grinding. 
6. Add an additional 500 µl of lysis buffer.
7. Grind until the lysate is homogeneous.
7. Close the microcentrifuge tube.
8. Spin for 5 min at full speed in a microcentrifuge at room temperature. Make sure to place the tubes in the rotor so that the microcentrifuge is balanced.  
9. While tubes are centrifuging, label two microcentrifuge tubes for each extract.
10. Add 500 µl of 70% ethanol to each tube.
11. Retrieve  your tubes from the microcentrifuge.
12. Carefully remove 400 µl of supernatant without disturbing the pellet and add it to the new tubes containing the 500 µl of 70% ethanol. Avoid transferring any solid plant material to the ethanol; if necessary, recentrifuge the lysate.
Mix the lysate and ethanol thoroughly by pipetting up and down. Cap the tubes.

```{block2, type='rmdimportant'}
**If a precipitate is visible, spin the microcentrifuge tube again for 5 min in a microcentrifuge at full speed to pellet the precipitated material and use the supernatant for the next stages.**
```
10. Label the top of a mini column for each sample with your initials. Place the columns in 2 ml capless collection tubes.
11. Remove 800 µl of cleared plant lysate from your microcentrifuge tubea and aad it to the column.

 ```{block2, type='rmdimportant'}
**Be sure that the lysate that is added to the column is clear and does not contain any precipitate. If necessary, centrifuge the lysate again to pellet any plant material prior to loading the lysate on the column.**
```
 
12. Place the capless collection tubes containing the columns into the microcentrifuge. Ensure that the centrifuge is balanced. Spin for 1 min at full speed at room temperature.
13. Discard the flowthrough from the collection tubes.
   
```{block2, type='rmdimportant'}
**If all of the supernatant does not pass through column, centrifuge again for 1 min. If there is any supernatant left in the column, carefully remove and discard it. Do not disrupt the bed of the column.**
```

13. Add 700 µl of wash buffer to each column. 
14. Spin at full speed at room temperature for 1 min. Discard the flowthrough.
15. Repeat steps 11 to 14 two more times for a total of 3 washes. 
14. After the final wash step, discard the flowthrough and put the columns back into the collection tubes. Centrifuge the columns for 2 min at full speed in the microcentrifuge at room temperature. This step removes the residual ethanol which will interfere with PCR if present.
15.Label a fresh microcentrifuge tube and transfer the column to the new tube.
16. Obtain the sterile water from the 70°C heat block.
17. Immediately add 80 µl of 70°C sterile water to the bed of each column, making certain that the water wets the column bed and let it sit for 1 min.
17. Place the column in the microcentrifuge tube into the microcentrifuge. Orient the loose cap of the microcentrifuge tube downward, toward the center of the rotor, to minimize friction and damage to the cap during centrifugation. 
18. Spin at full speed in the microcentrifuge at room temperature for 2 min. 
19. Remove the spin column from the microcentrifuge tube.

## GAPDH Polymerase Chain Reaction
## Experimental Procedures

1. Obtain a PCR tube.
2. Label the tube with your initials.
3. Add 20 µl of 2× MMIP (**m**aster **m**ix with **i**nitial **p**rimers) into each PCR tube.
4. Add 15 µl of sterile water to each tube.
5. Add 5 µl of the DNA template to each tube and gently pipet up and down to mix reagents.
6. Place your PCR tubes into the thermal cycler.
7. The PCR reaction will run for the next several hours using the program listed in Table \@ref(table:pcr).
8. The laboratory technicians will remove the PCR tubes from the thermocycler after the PCR has finished and store the tubes at -20°C until we continue our experiments next week.

Table: (\#tab:pcr) PCR protocol for GAPDH amplification.
 
|PCR Step            |Temperature|Time  |Number of cycles
|:-------------------|:----------|:-----|:--------------
|Initial denaturation|95°C       |5 min |1
|Denaturation        |95°C       |1 min |40
|Annealing           |52°C       |1 min |40
|Extension           |72°C       |2 min |40
|Final extension     |72°C       |6 min |1
|Hold                |15°C       |hold  |


