\documentclass[]{book}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdftitle={Laboratory Manual For SCI200 Genetics at Roxbury Community College},
            pdfauthor={Nikolaus Sucher},
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{longtable,booktabs}
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{5}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}

  \title{Laboratory Manual For SCI200 Genetics at Roxbury Community College}
    \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
    \author{Nikolaus Sucher}
    \preauthor{\centering\large\emph}
  \postauthor{\par}
    \date{}
    \predate{}\postdate{}
  
\usepackage{geometry}
%\usepackage{fontspec}
\usepackage{booktabs}
\usepackage{longtable}
\usepackage{siunitx}
\usepackage{framed}
\usepackage{xcolor}
 \definecolor{shadecolor}{RGB}{248,248,248}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{morefloats}
\usepackage[bf,singlelinecheck=on]{caption}
\usepackage{parskip}
 \setlength{\parindent}{15pt}
 % should be last package call in the preamble (?)
 \usepackage{hyperref}
  \urlstyle{same}

%\renewcommand{\textfraction}{0.05}
%\renewcommand{\topfraction}{0.8}
%\renewcommand{\bottomfraction}{0.8}
%\renewcommand{\floatpagefraction}{0.75}

\renewenvironment{quote}{\begin{VF}}{\end{VF}}
\let\oldhref\href
\renewcommand{\href}[2]{#2\footnote{\url{#1}}}

\ifxetex
  \usepackage{letltxmacro}
  \setlength{\XeTeXLinkMargin}{1pt}
  \LetLtxMacro\SavedIncludeGraphics\includegraphics
  \def\includegraphics#1#{% #1 catches optional stuff (star/opt. arg.)
    \IncludeGraphicsAux{#1}%
  }%
  \newcommand*{\IncludeGraphicsAux}[2]{%
    \XeTeXLinkBox{%
      \SavedIncludeGraphics#1{#2}%
    }%
  }%
\fi

\makeatletter
\newenvironment{kframe}{%
\medskip{}
\setlength{\fboxsep}{.8em}
 \def\at@end@of@kframe{}%
 \ifinner\ifhmode%
  \def\at@end@of@kframe{\end{minipage}}%
  \begin{minipage}{\columnwidth}%
 \fi\fi%
 \def\FrameCommand##1{\hskip\@totalleftmargin \hskip-\fboxsep
 \colorbox{shadecolor}{##1}\hskip-\fboxsep
     % There is no \\@totalrightmargin, so:
     \hskip-\linewidth \hskip-\@totalleftmargin \hskip\columnwidth}%
 \MakeFramed {\advance\hsize-\width
   \@totalleftmargin\z@ \linewidth\hsize
   \@setminipage}}%
 {\par\unskip\endMakeFramed%
 \at@end@of@kframe}
\makeatother

\newenvironment{Shaded}{\begin{kframe}}{\end{kframe}}

\newenvironment{rmdblock}[1]
  {
  \begin{itemize}
  \renewcommand{\labelitemi}{
    \raisebox{-.7\height}[0pt][0pt]{
      {\setkeys{Gin}{width=3em,keepaspectratio}\includegraphics{images/#1}}
    }
  }
  \setlength{\fboxsep}{1em}
  \begin{kframe}
  \item
  }
  {
  \end{kframe}
  \end{itemize}
  }

\newenvironment{rmdnote}
  {\begin{rmdblock}{note}}
  {\end{rmdblock}}

\newenvironment{rmdcaution}
  {\begin{rmdblock}{caution}}
  {\end{rmdblock}}

\newenvironment{rmdimportant}
  {\begin{rmdblock}{important}}
  {\end{rmdblock}}

\newenvironment{rmdtip}
  {\begin{rmdblock}{tip}}
  {\end{rmdblock}}

\newenvironment{rmdwarning}
  {\begin{rmdblock}{warning}}
  {\end{rmdblock}}

%\usepackage{makeidx}
% \makeindex

\usepackage{amsthm}
\makeatletter
\def\thm@space@setup{%
  \thm@preskip=8pt plus 2pt minus 4pt
  \thm@postskip=\thm@preskip
}
\makeatother

\usepackage{amsthm}
\newtheorem{theorem}{Theorem}[chapter]
\newtheorem{lemma}{Lemma}[chapter]
\theoremstyle{definition}
\newtheorem{definition}{Definition}[chapter]
\newtheorem{corollary}{Corollary}[chapter]
\newtheorem{proposition}{Proposition}[chapter]
\theoremstyle{definition}
\newtheorem{example}{Example}[chapter]
\theoremstyle{definition}
\newtheorem{exercise}{Exercise}[chapter]
\theoremstyle{remark}
\newtheorem*{remark}{Remark}
\newtheorem*{solution}{Solution}
\let\BeginKnitrBlock\begin \let\EndKnitrBlock\end
\begin{document}
\maketitle

{
\setcounter{tocdepth}{1}
\tableofcontents
}
\listoftables
\listoffigures
\chapter*{Welcome}\label{welcome}
\addcontentsline{toc}{chapter}{Welcome}

This is the \textbf{Laboratory Manual} for the Genetics course (SCI200)
at RCC.

\begin{center}\includegraphics[width=0.7\linewidth]{cover} \end{center}

This work is licensed under the
\href{https://creativecommons.org/licenses/by-sa/3.0/deed.en}{Creative
Commons Attribution-Share Alike 3.0 Unported} United States License.

\chapter*{Acknowledgements}\label{acknowledgements}
\addcontentsline{toc}{chapter}{Acknowledgements}

The creation of this manual was greatly facilitated and owes a major
debt to \href{https://www.wikipedia.org}{Wikipedia} and its large number
of voluntary contributors. I very liberally copied from many Wikipedia
pages and then remixed, edited, adapted and added text. With your
continued support and help this manual can only get better over time. I
urge you to email me with your criticisms and suggestions at
\href{mailto:nsucher@rcc.mass.edu}{\nolinkurl{nsucher@rcc.mass.edu}}.
This manual is made available as an open educational resource under
\href{https://creativecommons.org/licenses/by-sa/3.0/deed.en}{Creative
Commons Attribution-Share Alike 3.0 Unported} United States License for
others to do as I did and improve and adapt to specific requirements.

\chapter{Course Description}\label{course-description}

\section{Instructional Objectives:}\label{instructional-objectives}

At the end of the course, students will be able to

\begin{itemize}
\tightlist
\item
  contrast the role of mitosis and meiosis in cellular reproduction in
  eukaryotes.
\item
  describe how the concept of the ``gene'' has originated and evolved
  over time.
\item
  identify eukaryotic chromosomes and describe their structure and
  function.
\item
  explain how DNA stores information, read the genetic code, and
  describe how it is translated and transmitted to the offspring.
\item
  define genetic mutations and describe how they happen and how they can
  be prevented and give examples of how they influence the function of
  cells and can lead to disease.
\item
  outline how we can isolate, sequence, manipulate and move DNA between
  cells and organisms.
\item
  define the terms genome and genomics, evaluate genomics-related
  information and discuss the far-reaching consequences for our society
  of the use of genomics.
\item
  outline molecular mechanisms of gene expression.
\item
  give examples of complex traits and discuss current models of their
  genetic basis.
\item
  outline the role of genes in the development of organisms and give
  some specific examples.
\item
  discuss how variation in the types and frequencies of ``genes'' in
  organisms and populations forms the substrate for natural selection
  and evolution.
\end{itemize}

\section{Student Learning Outcomes:}\label{student-learning-outcomes}

At successful completion of the course, students will be able to

\begin{itemize}
\tightlist
\item
  describe the molecular and cellular mechanisms explaining why children
  resemble their parents to a certain degree yet are also distinctly
  different both from them and (if applicable) their own siblings.
\item
  give examples and evaluate the strength of the relationship between
  specific genes and health disorders.
\item
  demonstrate ``information literacy'' and critically analyze reports
  describing genetics-related research.
\item
  discuss the science underlying purported or real medical and
  scientific breakthroughs or doomsday scenarios related to the
  manipulation of cells and organisms at the level of DNA.
\item
  articulate and appraise the personal and societal implications of
  genetic testing and genetic engineering.
\item
  embark on the next level of training or start work in the life
  sciences or related fields.
\end{itemize}

\chapter{How to do well in this
class}\label{how-to-do-well-in-this-class}

\section*{Before the lab}\label{before-the-lab}
\addcontentsline{toc}{section}{Before the lab}

\begin{itemize}
\tightlist
\item
  Know what's coming up. Each week, look at the schedule to know what
  lab is coming up.
\item
  Be prepared. Read the chapter in the manual \textbf{before} you come
  to the lab.
\item
  In your own words, explain to yourself or someone else what the
  upcoming lab is about.
\end{itemize}

\section*{During the lab}\label{during-the-lab}
\addcontentsline{toc}{section}{During the lab}

\begin{itemize}
\tightlist
\item
  Read the lab instructions carefully.
\item
  Follow the instructions carefully.
\item
  Make sure that you know what you are doing and why you are doing it.
\item
  Don't be afraid to ask your instructor for help or clarifications.
\end{itemize}

\section*{After the lab}\label{after-the-lab}
\addcontentsline{toc}{section}{After the lab}

\begin{itemize}
\tightlist
\item
  Ask yourself: what was this lab about? What did we do? How did we do
  it?
\item
  In your own words, tell yourself or someone else what you learned in
  the lab.
\end{itemize}

\chapter{Lab Safety}\label{lab-safety}

The laboratory classes are hands-on. Some classes require the use of
hazardous chemicals and materials. Safety in the classroom is the \#1
priority for students and faculty. To ensure a safe science laboratory,
a list of rules must be followed at all times.

Please watch the \href{https//youtu.be/NxcsyTv7stQ}{RCC safety video}

\BeginKnitrBlock{rmdimportant}
\textbf{Prior to your participation in the science lab course, you must
read this safety document and sign and return the acknowledgment and
agreement page.}
\EndKnitrBlock{rmdimportant}

\section{General rules}\label{general-rules}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  NO FOOD, BEVERAGES, GUM, in the labs. Cell phone usage is also
  prohibited in the lab.
\item
  Conduct yourself in a responsible manner at all times in the lab.
  Horseplay, pranks and practical jokes are prohibited and will not be
  tolerated. If you participate in inappropriate behavior the INSTRUCTOR
  HAS THE RIGHT TO ASK YOU TO LEAVE THE lab.
\item
  Students cannot be in the lab without an instructor present.
\item
  Read all lab procedures, precautions, and equipment instructions
  thoroughly before each lab. Follow all written and verbal instructions
  carefully. Perform only those experiments authorized by the
  instructor. If during the lab you don't understand, stop and ask the
  instructor before proceeding. Never do anything in the lab that is
  outside of your instructors directions or that is not in your lab
  procedure.
\item
  Do not begin lab activities; touch any chemicals or equipment until
  you are instructed to do so.
\item
  Work areas should be kept organized and clean at all times. Only
  necessary items (lab notebook, worksheets, etc.) should be on your
  workbench. Backpacks and purses must be stored under the benches or
  against the walls. CLEAN ALL OF YOUR WORK SURFACES AND EQUIPMENT AT
  THE END OF THE EXPERIMENT. Safely dispose of waste in its proper
  container and place glassware in the grey bins by the sink. DO NOT
  STACK GLASSWARE. If the bin is full ask the instructor for another
  bin.
\item
  Keep aisles clear. Push lab stools under the lab benches when not in
  use.
\item
  Know where the safety equipment is and how to use it. This includes
  the first aid kit, eyewash station, safety shower, fire extinguisher,
  and fire blanket. Know the location of the fire alarm, and emergency
  phone. In the event of a fire drill during lab time containers must be
  closed, gas valves off, fume hood, and all electrical equipment must
  be turned off.
\item
  NEVER DISPOSE OF ANYTHING IN THE SINK. All materials are to be
  disposed of in the proper hazardous waste containers with the
  assistance of the instructor. All waste containers must be closed and
  placed inside a secondary containment bin.
\item
  As classes in these labs use toxic chemicals, keep your hands away
  from your face, eyes and mouth while working in the lab. Always wash
  your hands thoroughly with warm water and soap before leaving the lab
  to prevent injury or illness. This is part of proper lab procedure.
\item
  Students are not permitted in the prep room areas (between the lab
  rooms).
\item
  Handle all living organisms used for lab experiments in a respectful,
  humane manner.
\item
  Microscopes must be properly cleaned, the electrical cords properly
  wrapped, and returned to their places with their protective covers.
\end{enumerate}

Disposal of all hazardous waste is ONLY to be handled by the instructor
and in a manner consistent with federal, state and local hazardous waste
disposal regulations. Organic solvents are never to be disposed of down
the sink; receptacles will be provided as needed for their collection.
All hazardous chemical substances must be placed in the appropriate type
of container and labeled with chemical, name and date, sealed and placed
upright in a gray plastic bin.

\section{Class dissections}\label{class-dissections}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{13}
\tightlist
\item
  Preserved biological specimens should be treated with respect and
  disposed of in a clear plastic bag and placed inside the hazardous
  waste drum located in the classroom. This container must be sealed at
  the end of each dissection.
\item
  When using sharp objects always carry the tips pointed down and away
  from you. When dissecting, cut away from your body. Grasp the
  instrument only by the handle. Never try to catch falling sharp
  instruments or glassware. When you are finished dissecting, wash and
  dry your instruments and dissecting pan before storing them in the
  proper location. Do not leave any instruments in the sink.
\end{enumerate}

\section{Clothing}\label{clothing}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{15}
\tightlist
\item
  Students must wear lab goggles when using chemicals, or using heat. NO
  EXCEPTIONS! Lab coats are mandatory for Anatomy and Physiology,
  Biology, Microbiology, Chemistry, and Biotechnology labs, except for
  when lecturing and working on dry lab (for example, looking at models
  or prepared slides) activities.
\item
  Gloves should be worn when handling solutions, solids, specimens, etc.
\item
  Proper dress should always be observed in the lab. Long hair must be
  tied back. Loose or baggy clothing (especially sleeves), dangling
  jewelry, hats, shorts, short skirts, bare mid riffs, high heels,
  sleeveless shirts, and open toed or open heeled shoes and sandals are
  prohibited in the lab. Failure to comply may result in expulsion from
  class.
\end{enumerate}

\section{Handling chemicals}\label{handling-chemicals}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{17}
\tightlist
\item
  Always work in a well-ventilated area. Use the fume hood when working
  with volatile substances or poisonous vapors, or any chemical with an
  odor.
\item
  Never smell a chemical by sniffing. Use your hand to wave the chemical
  towards your nose.
\item
  DO NOT TASTE, TOUCH or smell anything unless instructed to do so. You
  should wear a lab apron and gloves at all times. Your instructor will
  tell you the proper gloves to wear depending on the chemical being
  used.
\item
  CHECK EACH LABEL TWICE before removing any of its contents. Take only
  what is needed of each chemical. NEVER return unused chemicals to
  their original container; put it in the waste container.
\item
  Do not use your fingers to transfer solid chemicals. Use a scoop or
  spatula.
\item
  Use a rubber bulb, pipette, or pi-pump when transferring liquid
  chemicals. NEVER USE YOUR MOUTH TO PIPETTE!
\item
  When transferring reagents hold the containers away from your body
  while working on the bench.
\item
  Acids must be handled with extreme care. You will be shown the proper
  method for diluting strong acids. ALWAYS ADD ACID TO WATER, swirl or
  invert the solution and be careful of the heat produced particularly
  with sulfuric acid.
\item
  Handle hazardous liquids over a pan to contain spills.
\item
  Never handle flammable liquids anywhere near an open flame or heat
  source.
\item
  Be careful when transporting chemicals across the lab. Hold securely
  and walk carefully.
\item
  NEVER POUR CHEMICALS INTO SINK. Waste should be disposed of in the
  proper hazardous waste container provided.
\end{enumerate}

\section{Glassware}\label{glassware}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{29}
\tightlist
\item
  Never handle broken glass with bare hands. Use a brush and a dustpan
  to clean up broken glass. Place uncontaminated broken glass in the
  white and blue broken glass receptacles. Contaminated trash goes in
  the biohazard bin.
\item
  Fill wash bottles only with distilled water and use only as intended,
  e.g.; rinsing glassware, adding water to a container.
\item
  Never use chipped, cracked or dirty glassware to avoid shattering.
\item
  Never immerse hot glassware in cold water. It may shatter.
\item
  Never place dirty glassware with the clean glassware. All dirty
  glassware should be placed in gray wash bins. DO NOT STACK DIRTY
  GLASSWARE IN BINS.
\end{enumerate}

\section{Heating substances}\label{heating-substances}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{34}
\tightlist
\item
  Exercise extreme caution when using a gas burner. Be careful to keep
  hair, loose clothing and hands away from flames at all times. Wear
  safety goggles. Do not put any substances into the flame unless
  specifically instructed to do so. Never reach over an exposed flame.
  The instructor will provide a demonstration of the proper way to
  operate a Bunsen burner. Never leave a lighted burner or hot plate
  unattended. Always turn the burner or hot plate off when not in use.
\item
  Do not point the open end of a test tube being heated at yourself or
  anyone else. Never look into a container that's being heated.
\item
  Heated metals and glass remain hot for a very long time. They should
  be set aside to cool and picked up with caution. Use tongs or heat
  protective gloves.
\end{enumerate}

\section{Handling microbiology
materials}\label{handling-microbiology-materials}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{37}
\tightlist
\item
  Please be aware that micro labs include work with pathogenic
  organisms. Be alert. Conduct yourself in a responsible manner at all
  times.
\item
  If you spill anything notify your instructor immediately. There are
  special procedures to be followed for spills containing
  microorganisms.
\item
  A lab coat must be worn during lab activities. Lab coats/aprons may
  never leave the lab. If you must leave the lab during a class then
  your lab coat must be removed.
\item
  Gloves must be worn at all times when working with bacteria. Gloves
  need to be disposed of in the biohazard waste container.
\item
  All contaminated waste must be disposed of in the biohazard container.
  Do not overfill biohazard containers.
\item
  You must spray down your lab bench with Lysol after each lab. Do not
  wipe with paper towels. The bench surface must remain wet for at least
  five minutes for the Lysol to destroy any micro organisms.
\item
  Wash your hands thoroughly before and after each lab as well as before
  you leave the lab for any reason.
\item
  Dispose of contaminated broken glass in the biohazard bin. Please wrap
  the broken glass in paper towels before disposal so that the broken
  glass doesn't cut through the bag. Dispose of uncontaminated glass in
  the white and blue cardboard glass boxes.
\end{enumerate}

\chapter*{Classical Genetics}\label{classical-genetics}
\addcontentsline{toc}{chapter}{Classical Genetics}

\href{https://en.wikipedia.org/wiki/Genetics}{Genetics} is, generally,
the study of genes, genetic variation, and heredity. In the first four
lab sessions of this course, we will study
\href{https://en.wikipedia.org/wiki/Classical_genetics}{classical
genetics}. This is the oldest form of genetics that began with
\href{https://en.wikipedia.org/wiki/Gregor_Mendel}{Gregor Mendel's}
experiments that formulated and defined a fundamental biological concept
now referred to as Mendelian Inheritance. In the second part of this
course, we will learn and apply some of the fundamental techniques of
\href{https://en.wikipedia.org/wiki/Molecular_genetics}{molecular
genetics}.

Classical genetics is the part of genetics that is about the
transmission of genetic traits via the acts of
\href{https://en.wikipedia.org/wiki/Reproduction}{reproduction}. The
process by which characteristics are passed down from parents to their
offspring is called heredity. In the sense of classical genetics,
variation is known as the lack of resemblance in related individuals and
can be categorized as discontinuous or continuous. The
\href{https://en.wikipedia.org/wiki/Phenotype}{phenotype} is a general
term that defines an individual's visible, physical traits. The
\href{https://en.wikipedia.org/wiki/Genotype}{genotype} of an offspring
is known as its genetic makeup.

\chapter{Mitosis and Meiosis}\label{mitosis-and-meiosis}

\href{https://en.wikipedia.org/wiki/Mitosis}{Mitosis} is the part of the
cell cycle when replicated chromosomes are separated into two new
nuclei. In general, mitosis (division of the nucleus) is preceded by the
S stage of interphase (during which the DNA is replicated) and is often
accompanied or followed by cytokinesis, which divides the cytoplasm,
organelles and cell membrane into two new cells containing roughly equal
shares of these cellular components. Mitosis and cytokinesis together
define the mitotic (M) phase of an animal cell cycle (the division of
the mother cell into two daughter cells genetically identical to each
other). The process of mitosis is divided into stages corresponding to
the completion of one set of activities and the start of the next. These
stages are prophase, pro-metaphase, metaphase, anaphase, and telophase.

\href{https://en.wikipedia.org/wiki/Meiosis}{Meiosis} is a specialized
type of cell division that reduces the chromosome number by half,
creating four haploid cells, each genetically distinct from the parent
cell that gave rise to them. This process occurs in all sexually
reproducing single-celled and multicellular eukaryotes, including
animals, plants, and fungi. Errors in meiosis resulting in aneuploidy
are the leading known cause of miscarriage and the most frequent genetic
cause of developmental disabilities. In meiosis, DNA replication is
followed by two rounds of cell division to produce four daughter cells,
each with half the number of chromosomes as the original parent cell.
The two meiotic divisions are known as Meiosis I and Meiosis II. Before
meiosis begins, during S phase of the cell cycle, the DNA of each
chromosome is replicated so that it consists of two identical sister
chromatids, which remain held together through sister chromatid
cohesion. This S-phase can be referred to as ``premeiotic S-phase'' or
``meiotic S-phase''. Immediately following DNA replication, meiotic
cells enter a prolonged G2-like stage known as meiotic prophase. During
this time, homologous chromosomes pair with each other and undergo
genetic recombination, a programmed process in which DNA is cut and then
repaired, which allows them to exchange some of their genetic
information. A subset of recombination events results in crossovers,
which create physical links known as chiasmata (singular: chiasma, for
the Greek letter Chi (X)) between the homologous chromosomes. In most
organisms, these links are essential to direct each pair of homologous
chromosomes to segregate away from each other during Meiosis I,
resulting in two haploid cells that have half the number of chromosomes
as the parent cell. During Meiosis II, the cohesion between sister
chromatids is released and they segregate from one another, as during
mitosis. In some cases, all four of the meiotic products form gametes
such as sperm, spores, or pollen. In female animals, three of the four
meiotic products are typically eliminated by extrusion into polar
bodies, and only one cell develops to produce an ovum. Because the
number of chromosomes is halved during meiosis, gametes can fuse
(i.e.~fertilization) to form a diploid zygote that contains two copies
of each chromosome, one from each parent. Thus, alternating cycles of
meiosis and fertilization enable sexual reproduction, with successive
generations maintaining the same number of chromosomes. For example,
diploid human cells contain 23 pairs of chromosomes including 1 pair of
sex chromosomes (46 total), half of maternal origin and half of paternal
origin. Meiosis produces haploid gametes (ova or sperm) that contain one
set of 23 chromosomes. When two gametes (an egg and a sperm) fuse, the
resulting zygote is once again diploid, with the mother and father each
contributing 23 chromosomes. This same pattern, but not the same number
of chromosomes, occurs in all organisms that utilize meiosis.

\section{View Prepared Slides}\label{view-prepared-slides}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  View the onion root tip and observe the different stages of mitosis
  (Figure \ref{fig:tip}).
\end{enumerate}

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/mitosis/Onion_root_tip} 

}

\caption{Onion root tip}\label{fig:tip}
\end{figure}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  View the fish blastodisc and observe the different stages of mitosis
  (Figure \ref{fig:blastodisc}).
\end{enumerate}

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/mitosis/Fish_blastodisc} 

}

\caption{Fish blastodisc}\label{fig:blastodisc}
\end{figure}

\section{Preparing an Onion root tip
squash}\label{preparing-an-onion-root-tip-squash}

\subsection{Experimental procedures}\label{experimental-procedures}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Obtain an onion bulb that shows some roots.
\item
  Cut off a root tip and place it on a clean slide.
\item
  Cut off 1mm to 2mm of the root tip and throw away the upper portion of
  the root.
\item
  Cover the root tip with four drops of 1 N HCl and warm the slide over
  an alcohol burner flame for 1 minute. Do not boil.
\item
  Blot off the excess HCl and cover the root tip with 0.5\% aqueous
  toluidine blue.
\item
  Again, pass the slide through the alcohol burner flame for 1 minute
  without boiling.
\item
  Blot off the excess stain, add a drop of fresh stain, and apply a
  coverslip.
\item
  Cover the slide with a paper towel and carefully squash the coverslip
  firmly with your thumb.
\item
  Examine the slide for the stages of mitosis as well as interphase and
  cytokinesis.
\end{enumerate}

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/mitosis/Onion_root_tip_spread} 

}

\caption{Several different phases of mitosis are visible in this onion root tip spread.}\label{fig:spread}
\end{figure}

\section{Review Questions}\label{review-questions}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  What is mitosis and what is its outcome?
\item
  What is meiosis and what is its outcome?
\item
  What is homologous recombination and what is its outcome?
\item
  What do the terms haploid and diploid mean?
\item
  Are you a haploid or a diploid organism?
\item
  What are gametes?
\item
  What is a zygote?
\end{enumerate}

\chapter{Chromosomes And Karyotypes}\label{chromosomes-and-karyotypes}

\section{Chromosomes}\label{chromosomes}

A \href{https://en.wikipedia.org/wiki/Chromosome}{chromosome} is a DNA
molecule with part or all of the genetic material (genome) of an
organism. Most eukaryotic chromosomes include packaging proteins which,
aided by chaperone proteins, bind to and condense the DNA molecule to
prevent it from becoming an unmanageable tangle. The word chromosome
comes from the Greek χρῶμα (chroma, ``colour'') and σῶμα (soma,
``body''), describing their strong staining by particular dyes. The term
was coined by
\href{https://en.wikipedia.org/wiki/Heinrich_Wilhelm_Gottfried_von_Waldeyer-Hartz}{Heinrich
Wilhelm Gottfried von Waldeyer-Hartz}, referring to the term chromatin,
which was introduced by
\href{https://en.wikipedia.org/wiki/Walther_Flemming}{Walther Flemming}.

Chromosomes are normally visible under a light microscope only when the
cell is undergoing the metaphase of cell division (where all chromosomes
are aligned in the center of the cell in their condensed form). Before
this happens, every chromosome is copied once (S phase), and the copy is
joined to the original by a centromere, resulting either in an X-shaped
structure (Figure \ref{fig:karyo}) if the centromere is located in the
middle of the chromosome or a two-arm structure if the centromere is
located near one of the ends. The original chromosome and the copy are
now called sister chromatids. During metaphase the X-shape structure is
called a metaphase chromosome. In this highly condensed form chromosomes
are easiest to distinguish and study. In animal cells, chromosomes reach
their highest compaction level in anaphase during segregation.

(ref:chromosome)
\href{https://commons.wikimedia.org/wiki/File:Chromosome.svg}{Diagram of
a replicated and condensed metaphase eukaryotic chromosome. (1)
Chromatid -- one of the two identical parts of the chromosome after S
phase. (2) Centromere -- the point where the two chromatids touch. (3)
Short (p) arm. (4) Long (q) arm.} By derivative work: Tryphon
(talk)*Chromosome-upright.png: Original version: Magnus Manske, this
version with upright chromosome: User:Dietzel65 {[}GFDL
(\url{http://www.gnu.org/copyleft/fdl.html}) or CC-BY-SA-3.0
(\url{http://creativecommons.org/licenses/by-sa/3.0/}){]}, via Wikimedia
Commons

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/karyotyping/Chromosome} 

}

\caption{(ref:chromosome)}\label{fig:chromosome}
\end{figure}

Chromosomal recombination during meiosis and subsequent sexual
reproduction play a significant role in genetic diversity. If these
structures are manipulated incorrectly, through processes known as
chromosomal instability and translocation, the cell may undergo mitotic
catastrophe and die. Mutations in the cell can allow it to
inappropriately evade apoptosis and lead to the progression of cancer.

Some use the term chromosome in a wider sense, to refer to the
individualized portions of chromatin in cells, either visible or not
under light microscopy. Others use the concept in a narrower sense, to
refer to the individualized portions of chromatin during cell division,
visible under light microscopy due to high condensation.

The prokaryotes -- bacteria and archaea -- typically have a single
circular chromosome, but many variations exist. The chromosomes of most
bacteria can range in size from only 130,000 base pairs in the
endosymbiotic bacteria \emph{Candidatus Hodgkinia cicadicola} to more
than 14,000,000 base pairs in the soil-dwelling bacterium
\emph{Sorangium cellulosum}. Spirochaetes of the genus \emph{Borrelia}
are a notable exception to this arrangement, with bacteria such as
\emph{Borrelia burgdorferi}, the cause of Lyme disease, containing a
single linear chromosome.

Chromosomes in eukaryotes are composed of chromatin fiber. Chromatin
fiber is made of nucleosomes (histone octamers with part of a DNA strand
attached to and wrapped around it). Chromatin fibers are packaged by
proteins into a condensed structure called chromatin. Chromatin contains
the vast majority of DNA and a small amount inherited maternally, can be
found in the mitochondria. Chromatin is present in most cells, with a
few exceptions, for example, red blood cells.

Chromatin allows the very long DNA molecules to fit into the cell
nucleus. During cell division chromatin condenses further to form
microscopically visible chromosomes. The structure of chromosomes varies
through the cell cycle. During cellular division chromosomes are
replicated, divided, and passed successfully to their daughter cells so
as to ensure the genetic diversity and survival of their progeny.
Chromosomes may exist as either duplicated or unduplicated. Unduplicated
chromosomes are single double helixes, whereas duplicated chromosomes
contain two identical copies (called chromatids or sister chromatids)
joined by a centromere. Eukaryotes possess multiple large linear
chromosomes contained in the cells' nucleus. Each chromosome has one
centromere, with one or two arms projecting from the centromere,
although, under most circumstances, these arms are not visible as such.
In addition, most eukaryotes have a small circular mitochondrial genome,
and some eukaryotes may have additional small circular or linear
cytoplasmic chromosomes.Eukaryotes (cells with nuclei such as those
found in plants, fungi, and animals) possess multiple large linear
chromosomes contained in the cell's nucleus. Each chromosome has one
centromere, with one or two arms projecting from the centromere,
although, under most circumstances, these arms are not visible as such.
In addition, most eukaryotes have a small circular mitochondrial genome,
and some eukaryotes may have additional small circular or linear
cytoplasmic chromosomes.

In the nuclear chromosomes of eukaryotes, the uncondensed DNA exists in
a semi-ordered structure, where it is wrapped around histones
(structural proteins), forming a composite material called chromatin.

Chromosomes in humans are divided into two types: autosomes (body
chromosome(s)) and allosome (sex chromosome(s)). Certain genetic traits
are linked to a person's sex and are passed on through the sex
chromosomes. The autosomes contain the rest of the genetic hereditary
information. All act in the same way during cell division. Human cells
have 23 pairs of chromosomes (22 pairs of autosomes and one pair of sex
chromosomes), giving a total of 46 per cell. In addition to these, human
cells have many hundreds of copies of the mitochondrial genome.

\section{Karyotypes}\label{karyotypes}

A \href{https://en.wikipedia.org/wiki/Karyotype}{karyotype} is the
number and appearance of chromosomes in the nucleus of a eukaryotic
cell. The term is also used for the complete set of chromosomes in a
species or in an individual organism and for a test that detects this
complement or measures the number. The name was coined by the German
anatomist, Heinrich von Waldeyer in 1888. It is New Latin from Ancient
Greek κάρυον karyon, ``kernel'', ``seed'', or ``nucleus'', and τύπος
typos, ``general form'').

Karyotypes describe the chromosome count of an organism and what these
chromosomes look like under a light microscope. Attention is paid to
their length, the position of the centromeres, banding pattern, any
differences between the sex chromosomes, and any other physical
characteristics. The preparation and study of karyotypes is part of
cytogenetics.

The study of whole sets of chromosomes is sometimes known as karyology.
The chromosomes are depicted (by rearranging a photomicrograph) in a
standard format known as a karyogram or idiogram: in pairs, ordered by
size and position of centromere for chromosomes of the same size as
shown in Figure \ref{fig:karyo}.

(ref:karyo)
\href{https://commons.wikimedia.org/wiki/File:NHGRI_human_male_karyotype.png}{Karyogram
of human male using Giemsa staining}

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/karyotyping/human_male_karyotype} 

}

\caption{(ref:karyo)}\label{fig:karyo}
\end{figure}

The basic number of chromosomes in the somatic cells of an individual or
a species is called the somatic number and is designated 2n. In the
germ-line (the sex cells) the chromosome number is n (humans: n = 23).
Thus, in humans 2n = 46.

So, in normal diploid organisms, autosomal chromosomes are present in
two copies. There may, or may not, be sex chromosomes. Polyploid cells
have multiple copies of chromosomes and haploid cells have single
copies.

The study of karyotypes is important for cell biology and genetics, and
the results may be used in evolutionary biology (karyosystematics) and
medicine. Karyotypes can be used for many purposes; such as to study
chromosomal aberrations, cellular function, taxonomic relationships, and
to gather information about past evolutionary events.

The German scientists Matthias
\href{https://en.wikipedia.org/wiki/Matthias_Jakob_Schleiden}{Jakob
Schleiden}, \href{https://en.wikipedia.org/wiki/Rudolf_Virchow}{Rudolf
Virchow} and \href{https://en.wikipedia.org/wiki/Otto_Bütschli}{Otto
Bütschli} were among the first scientists who recognized the structures
now familiar as chromosomes. Chromosomes were first observed in plant
cells by \href{https://en.wikipedia.org/wiki/Carl_Nägeli}{Carl Wilhelm
von Nägeli} in 1842. Their behavior in animal (salamander) cells was
described by Walther Flemming, the discoverer of mitosis, in 1882.

The next stage took place after the development of genetics in the early
20th century, when it was appreciated that chromosomes (that can be
observed by karyotype) were the carrier of genes.

In his famous textbook The Cell in Development and Heredity, Wilson
linked together the independent work of
\href{https://en.wikipedia.org/wiki/Theodor_Boveri}{Theodor Boveri} and
\href{https://en.wikipedia.org/wiki/Walter_Sutton}{Walter Sutton} (both
around 1902) by naming the chromosome theory of inheritance the
\href{https://en.wikipedia.org/wiki/Boveri–Sutton_chromosome_theory}{Boveri--Sutton
chromosome theory} (the names are sometimes reversed).

Investigation into the human karyotype took many years to settle the
most basic question: how many chromosomes does a normal diploid human
cell contain? In 1912, Hans von Winiwarter reported 47 chromosomes in
spermatogonia and 48 in oogonia, concluding an XX/XO sex determination
mechanism. The number of human chromosomes was published in 1923 by
Theophilus Painter. By inspection through the microscope, he counted 24
pairs, which would mean 48 chromosomes. His error was copied by others
and it was not until 1956 that the true number, 46, was determined by
Indonesia-born cytogeneticist
\href{https://en.wikipedia.org/wiki/Joe_Hin_Tjio}{Joe Hin Tjio}. He was
responsible for finding a new approach that settled the issue:

\begin{itemize}
\tightlist
\item
  Using cells in tissue culture
\item
  Pretreating cells in a hypotonic solution, which swells them and
  spreads the chromosomes
\item
  Arresting mitosis in metaphase by a solution of colchicine
\item
  Squashing the preparation on the slide forcing the chromosomes into a
  single plane
\item
  Cutting up a photomicrograph and arranging the result into an
  indisputable karyogram.
\end{itemize}

The work took place in 1955, and was published in 1956. The karyotype of
humans includes only 46 chromosomes. Rather interestingly, the great
apes have 48 chromosomes. Human chromosome 2 is now known to be a result
of an end-to-end fusion of two ancestral ape chromosomes.

\section{Observations on karyotypes}\label{observations-on-karyotypes}

\subsection{Staining}\label{staining}

The study of karyotypes is made possible by staining. Usually, a
suitable dye, such as Giemsa, is applied after cells have been arrested
during cell division by a solution of colchicine usually in metaphase or
prometaphase when most condensed. In order for the Giemsa stain to
adhere correctly, all chromosomal proteins must be digested and removed.
For humans, white blood cells are used most frequently because they are
easily induced to divide and grow in tissue culture. Sometimes
observations may be made on non-dividing (interphase) cells. The sex of
an unborn fetus can be determined by observation of interphase cells
(see amniotic centesis and Barr body).

\subsection{Observations}\label{observations}

Six different characteristics of karyotypes are usually observed and
compared:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Differences in absolute sizes of chromosomes. Chromosomes can vary in
  absolute size by as much as twenty-fold between genera of the same
  family. For example, the legumes \emph{Lotus tenuis} and \emph{Vicia
  faba} each have six pairs of chromosomes, yet \emph{V. faba}
  chromosomes are many times larger. These differences probably reflect
  different amounts of DNA duplication.
\item
  Differences in the position of centromeres. These differences probably
  came about through translocations.
\item
  Differences in relative size of chromosomes. These differences
  probably arose from segmental interchange of unequal lengths.
\item
  Differences in basic number of chromosomes. These differences could
  have resulted from successive unequal translocations which removed all
  the essential genetic material from a chromosome, permitting its loss
  without penalty to the organism (the dislocation hypothesis) or
  through fusion. Humans have one pair fewer chromosomes than the great
  apes. Human chromosome 2 appears to have resulted from the fusion of
  two ancestral chromosomes, and many of the genes of those two original
  chromosomes have been translocated to other chromosomes.
\item
  Differences in number and position of satellites. Satellites are small
  bodies attached to a chromosome by a thin thread.
\item
  Differences in degree and distribution of heterochromatic regions.
  Heterochromatin stains darker than euchromatin. Heterochromatin is
  packed tighter. Heterochromatin consists mainly of genetically
  inactive and repetitive DNA sequences as well as containing a larger
  amount of Adenine-Thymine pairs. Euchromatin is usually under active
  transcription and stains much lighter as it has less affinity for the
  giemsa stain. Euchromatin regions contain larger amounts of
  Guanine-Cytosine pairs. The staining technique using giemsa staining
  is called G banding and therefore produces the typical ``G-Bands''.
\end{enumerate}

A full account of a karyotype may therefore include the number, type,
shape and banding of the chromosomes, as well as other cytogenetic
information.

Variation is often found:

\begin{itemize}
\tightlist
\item
  between the sexes,
\item
  between the germ-line and soma (between gametes and the rest of the
  body),
\item
  between members of a population (chromosome polymorphism), in
  geographic specialization, and in mosaics or otherwise abnormal
  individuals. Human karyotype.
\end{itemize}

\section{Human karyotype}\label{human-karyotype}

The normal human karyotypes contain 22 pairs of autosomal chromosomes
and one pair of sex chromosomes (allosomes; Figure \ref{fig:karyo}).
Normal karyotypes for females contain two X chromosomes and are denoted
46,XX; males have both an X and a Y chromosome denoted 46,XY. Any
variation from the standard karyotype may lead to developmental
abnormalities.

\subsection{Ploidy}\label{ploidy}

\href{https://en.wikipedia.org/wiki/Ploidy}{Ploidy} is the number of
complete sets of chromosomes in a cell.

\subsection{Aneuploidy}\label{aneuploidy}

\href{https://en.wikipedia.org/wiki/Aneuploidy}{Aneuploidy} is the
condition in which the chromosome number in the cells is not the typical
number for the species. This would give rise to a chromosome abnormality
such as an extra chromosome or one or more chromosomes lost.
Abnormalities in chromosome number usually cause a defect in
development. \href{https://en.wikipedia.org/wiki/Down_syndrome}{Down
syndrome} (Figure \ref{fig:Down}) and
\href{https://en.wikipedia.org/wiki/Turner_syndrome}{Turner syndrome}
are examples of this.

\section{Chromosome abnormalities}\label{chromosome-abnormalities}

Chromosome abnormalities can be numerical, as in the presence of extra
or missing chromosomes, or structural, as in derivative chromosome,
translocations, inversions, large-scale deletions or duplications.
Numerical abnormalities, also known as aneuploidy, often occur as a
result of nondisjunction during meiosis in the formation of a gamete;
trisomies, in which three copies of a chromosome are present instead of
the usual two, are common numerical abnormalities. Structural
abnormalities often arise from errors in homologous recombination. Both
types of abnormalities can occur in gametes and therefore will be
present in all cells of an affected person's body, or they can occur
during mitosis and give rise to a genetic mosaic individual who has some
normal and some abnormal cells.

\subsection{Chromosomal abnormalities that lead to disease in humans
include}\label{chromosomal-abnormalities-that-lead-to-disease-in-humans-include}

\begin{itemize}
\tightlist
\item
  Turner syndrome (Figure \ref{fig:Turner})results from a single X
  chromosome (45,X or 45,X0).
\item
  \href{https://en.wikipedia.org/wiki/Klinefelter_syndrome}{Klinefelter
  syndrome} (Figure \ref{fig:Klinefelter}), the most common male
  chromosomal disease, otherwise known as 47,XXY, is caused by an extra
  X chromosome.
\item
  \href{https://en.wikipedia.org/wiki/Edwards_syndrome}{Edwards
  syndrome} is caused by trisomy (three copies) of chromosome 18.
\item
  Down syndrome (Figure \ref{fig:Down}), a common chromosomal disease,
  is caused by trisomy of chromosome 21.
\item
  \href{https://en.wikipedia.org/wiki/Patau_syndrome}{Patau syndrome} is
  caused by trisomy of chromosome 13.
\item
  \href{https://en.wikipedia.org/wiki/Trisomy_9}{Trisomy 9}, believed to
  be the 4th most common trisomy, has many long lived affected
  individuals but only in a form other than a full trisomy, such as
  trisomy 9p syndrome or mosaic trisomy 9. They often function quite
  well, but tend to have trouble with speech.
\item
  Also documented are trisomy 8 and trisomy 16, although they generally
  do not survive to birth.
\end{itemize}

(ref:Turner)
\href{https://commons.wikimedia.org/wiki/File:45,X.jpg}{45,X karyotype,
showing an unpaired X at the lower right}

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/karyotyping/45,X} 

}

\caption{(ref:Turner)}\label{fig:Turner}
\end{figure}

(ref:Klinefelter)
\href{https://commons.wikimedia.org/wiki/File:Human_chromosomesXXY01.png}{Klinefelter
syndrome karyotype (XXY)}

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/karyotyping/Human_chromosomesXXY01} 

}

\caption{(ref:Klinefelter)}\label{fig:Klinefelter}
\end{figure}

(ref:Down)
\href{https://commons.wikimedia.org/wiki/File:Down_Syndrome_Karyotype.png}{Karyotype
for trisomy Down syndrome: notice the three copies of chromosome 21}

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/karyotyping/Down_Syndrome_Karyotype} 

}

\caption{(ref:Down)}\label{fig:Down}
\end{figure}

Some disorders arise from loss of just a piece of one chromosome,
including

\begin{itemize}
\tightlist
\item
  \href{https://en.wikipedia.org/wiki/Cri_du_chat_syndrome}{Cri du chat}
  (cry of the cat), from a truncated short arm on chromosome 5. The name
  comes from the babies' distinctive cry, caused by abnormal formation
  of the larynx.
\item
  1p36 Deletion syndrome, from the loss of part of the short arm of
  chromosome 1.
\item
  Angelman syndrome -- 50\% of cases have a segment of the long arm of
  chromosome 15 missing; a deletion of the maternal genes, example of
  imprinting disorder.
\item
  Prader-Willi syndrome -- 50\% of cases have a segment of the long arm
  of chromosome 15 missing; a deletion of the paternal genes, example of
  imprinting disorder.
\item
  Chromosomal abnormalities can also occur in cancerous cells of an
  otherwise genetically normal individual; one well-documented example
  is the Philadelphia chromosome, a translocation mutation commonly
  associated with chronic myelogenous leukemia and less often with acute
  lymphoblastic leukemia.
\end{itemize}

\section{Chromosome banding}\label{chromosome-banding}

Chromosomes display a banded pattern when treated with some stains.
Bands are alternating light and dark stripes that appear along the
lengths of chromosomes. Unique banding patterns are used to identify
chromosomes and to diagnose chromosomal aberrations, including
chromosome breakage, loss, duplication, translocation or inverted
segments. A range of different chromosome treatments produce a range of
banding patterns: G-bands, R-bands, C-bands, Q-bands, T-bands and
NOR-bands.

Cytogenetics employs several techniques to visualize different aspects
of chromosomes.

\subsection{Types of banding}\label{types-of-banding}

\begin{itemize}
\tightlist
\item
  G-banding is obtained with Giemsa stain following digestion of
  chromosomes with trypsin. It yields a series of lightly and darkly
  stained bands --- the dark regions tend to be heterochromatic,
  late-replicating and AT rich. The light regions tend to be
  euchromatic, early-replicating and GC rich. This method will normally
  produce 300--400 bands in a normal, human genome.
\item
  R-banding is the reverse of G-banding (the R stands for ``reverse'').
  The dark regions are euchromatic (guanine-cytosine rich regions) and
  the bright regions are heterochromatic (thymine-adenine rich regions).
\item
  C-banding: Giemsa binds to constitutive heterochromatin, so it stains
  centromeres.The name is derived from centromeric or constitutive
  heterochromatin. The preparations undergo alkaline denaturation prior
  to staining leading to an almost complete depurination of the DNA.
  After washing the probe the remaining DNA is renatured again and
  stained with Giemsa solution consisting of methylene azure, methylene
  violet, methylene blue, and eosin. Heterochromatin binds a lot of the
  dye, while the rest of the chromosomes absorb only little of it. The
  C-bonding proved to be especially well-suited for the characterization
  of plant chromosomes.
\item
  Q-banding is a fluorescent pattern obtained using quinacrine for
  staining. The pattern of bands is very similar to that seen in
  G-banding.They can be recognized by a yellow fluorescence of differing
  intensity. Most part of the stained DNA is heterochromatin. Quinacrin
  (atebrin) binds both regions rich in AT and in GC, but only the
  AT-quinacrin-complex fluoresces. Since regions rich in AT are more
  common in heterochromatin than in euchromatin, these regions are
  labelled preferentially. The different intensities of the single bands
  mirror the different contents of AT. Other fluorochromes like DAPI or
  Hoechst 33258 lead also to characteristic, reproducible patterns. Each
  of them produces its specific pattern. In other words: the properties
  of the bonds and the specificity of the fluorochromes are not
  exclusively based on their affinity to regions rich in AT. Rather, the
  distribution of AT and the association of AT with other molecules like
  histones, for example, influences the binding properties of the
  fluorochromes.
\item
  T-banding: visualize telomeres. Silver staining: Silver nitrate stains
  the nucleolar organization region-associated protein. This yields a
  dark region where the silver is deposited, denoting the activity of
  rRNA genes within the NOR.
\end{itemize}

\subsection{Classic karyotype
cytogenetics}\label{classic-karyotype-cytogenetics}

In the ``classic'' (depicted) karyotype, a dye, often Giemsa
(G-banding), less frequently mepacrine (quinacrine), is used to stain
bands on the chromosomes (Figure \ref{fig:karyo}). Giemsa is specific
for the phosphate groups of DNA. Quinacrine binds to the
adenine-thymine-rich regions. Each chromosome has a characteristic
banding pattern that helps to identify them; both chromosomes in a pair
will have the same banding pattern.

Karyotypes are arranged with the short arm of the chromosome on top, and
the long arm on the bottom. Some karyotypes call the short and long arms
p and q, respectively. In addition, the differently stained regions and
sub-regions are given numerical designations from proximal to distal on
the chromosome arms. For example, Cri du chat syndrome involves a
deletion on the short arm of chromosome 5. It is written as 46,XX,5p-.
The critical region for this syndrome is deletion of p15.2 (the locus on
the chromosome), which is written as 46,XX,del(5)(p15.2).

\subsection{Multicolor FISH (mFISH) and spectral karyotype (SKY
technique)}\label{multicolor-fish-mfish-and-spectral-karyotype-sky-technique}

Multicolor FISH and the older spectral karyotyping are molecular
cytogenetic techniques used to simultaneously visualize all the pairs of
chromosomes in an organism in different colors (Figure \ref{fig:sky}).
Fluorescently labeled probes for each chromosome are made by labeling
chromosome-specific DNA with different fluorophores. Because there are a
limited number of spectrally distinct fluorophores, a combinatorial
labeling method is used to generate many different colors. Fluorophore
combinations are captured and analyzed by a fluorescence microscope
using up to 7 narrow-banded fluorescence filters or, in the case of
spectral karyotyping, by using an interferometer attached to a
fluorescence microscope. In the case of an mFISH image, every
combination of fluorochromes from the resulting original images is
replaced by a pseudo color in a dedicated image analysis software. Thus,
chromosomes or chromosome sections can be visualized and identified,
allowing for the analysis of chromosomal rearrangements. In the case of
spectral karyotyping, image processing software assigns a pseudo color
to each spectrally different combination, allowing the visualization of
the individually colored chromosomes. Multicolor FISH is used to
identify structural chromosome aberrations in cancer cells and other
disease conditions when Giemsa banding or other techniques are not
accurate enough.

(ref:sky)
\href{https://en.wikipedia.org/wiki/File:Spectralkaryotype98-300.jpg}{Spectral
human karyotype}

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/karyotyping/Spectralkaryotype98-300} 

}

\caption{(ref:sky)}\label{fig:sky}
\end{figure}

\section{Determine the karyotype}\label{determine-the-karyotype}

\subsection{Experimental procedures}\label{experimental-procedures-1}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Obtain a photomicrograph of a metaphase chromosome spread.
\item
  Write down the code associated with the photomicrograph.
\item
  Using scissors carefully cut out the largest chromosome on the page
  and paste it onto the ``data page''.
\item
  Find the matching chromosome and paste it next to the first
  chromosome.
\item
  Repeat these steps until all chromosomes have been pasted onto the
  data page by decreasing size (as shown in Figure \ref{fig:karyo}).
  \textbf{Cut out only one chromosome at a time or you will get confused
  and likely lose chromosomes.}
\item
  Determine the number of chromosomes on the data page. Any extra
  chromosome(s) are indication of an aneuploidy.
\item
  Determine the type of sex chromosomes on the data page.
\item
  Write down the karyotype.
\item
  Write down the suggested diagnosis.
\end{enumerate}

\section{Review Questions}\label{review-questions-1}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
\end{enumerate}

\chapter{Mendelian Genetics}\label{mendelian-genetics}

In this laboratory session, we will use maize \emph{Zea mays} subsp.
mays, from Spanish: maíz after Taíno mahiz), also known as corn to study
\href{https://en.wikipedia.org/wiki/Mendelian_inheritance}{Mendelian
inheritance}. This cereal grain was first domesticated by indigenous
peoples in southern Mexico about 10,000 years ago. The leafy stalk of
the plant produces separate pollen and ovuliferous inflorescences or
ears, which are fruits, yielding kernels or seeds. Maize has become a
staple food in many parts of the world, with total production surpassing
that of wheat or rice. However, not all of this maize is consumed
directly by humans. Some of the maize production is used for corn
ethanol, animal feed and other maize products, such as corn starch and
corn syrup. The six major types of corn are dent corn, flint corn, pod
corn, popcorn, flour corn, and sweet corn.

The principles of Mendelian inheritance were named for and first derived
by \href{https://en.wikipedia.org/wiki/Gregor_Mendel}{Gregor Johann
Mendel}, a nineteenth-century Moravian monk who formulated his ideas
after conducting simple hybridisation experiments with pea plants
(\emph{Pisum sativum}) he had planted in the garden of his monastery.
Between 1856 and 1863, Mendel cultivated and tested some 5,000 pea
plants. From these experiments, he induced two generalizations which
later became known as Mendel's Principles of Heredity or Mendelian
inheritance (Table \ref{tab:mendel}). He described these principles in a
two-part paper, Versuche über Pflanzen-Hybriden (Experiments on Plant
Hybridization), that he read to the Natural History Society of Brno on 8
February and 8 March 1865, and which was published in 1866. Mendel's
conclusions were largely ignored by the vast majority of scientists at
the time. In 1900, however, his work was ``re-discovered'' by three
European scientists, Hugo de Vries, Carl Correns, and Erich von
Tschermak.

Mendel discovered that, when he crossed purebred white flower and purple
flower pea plants (the parental or P generation), the result was not a
blend. Rather than being a mix of the two, the offspring (known as the
F1 generation) was purple-flowered. When Mendel self-fertilized the F1
generation pea plants, he obtained a purple flower to white flower ratio
in the F2 generation of 3 to 1. In the first experiment, we will examine
the F2 generation resulting from the F1 generation obtained from a
parental generation of yellow and purple corn.

He then conceived the idea of heredity units, which he called
``factors''. Mendel found that there are alternative forms of
factors---now called genes---that account for variations in inherited
characteristics. For example, the gene for flower color in pea plants
exists in two forms, one for purple and the other for white. The
alternative ``forms'' are now called alleles. For each biological trait,
an organism inherits two alleles, one from each parent. These alleles
may be the same or different. An organism that has two identical alleles
for a gene is said to be homozygous for that gene (and is called a
homozygote). An organism that has two different alleles for a gene is
said be heterozygous for that gene (and is called a heterozygote).

Mendel hypothesized that allele pairs separate randomly, or segregate,
from each other during the production of gametes: egg and sperm. Because
allele pairs separate during gamete production, a sperm or egg carries
only one allele for each inherited trait. When sperm and egg unite at
fertilization, each contributes its allele, restoring the paired
condition in the offspring. This is called the Law of Segregation.
Mendel also found that each pair of alleles segregates independently of
the other pairs of alleles during gamete formation. This is known as the
Law of Independent Assortment. In the second experiment, we will observe
this law exemplified by a dihybrid cross of corn.

The genotype of an individual is made up of the many alleles it
possesses. An individual's physical appearance, or phenotype, is
determined by its alleles as well as by its environment. The presence of
an allele does not mean that the trait will be expressed in the
individual that possesses it. If the two alleles of an inherited pair
differ (the heterozygous condition), then one determines the organism's
appearance and is called the dominant allele; the other has no
noticeable effect on the organism's appearance and is called the
recessive allele. Thus, in the example above the dominant purple flower
allele will hide the phenotypic effects of the recessive white flower
allele. This is known as the Law of Dominance but it is not a
transmission law: it concerns the expression of the genotype. The upper
case letters are used to represent dominant alleles whereas the
lowercase letters are used to represent recessive alleles.

\begin{longtable}[]{@{}ll@{}}
\caption{\label{tab:mendel} Mendel's Laws Of Inheritance.}\tabularnewline
\toprule
\begin{minipage}[b]{0.05\columnwidth}\raggedright\strut
Law\strut
\end{minipage} & \begin{minipage}[b]{0.14\columnwidth}\raggedright\strut
Definition\strut
\end{minipage}\tabularnewline
\midrule
\endfirsthead
\toprule
\begin{minipage}[b]{0.05\columnwidth}\raggedright\strut
Law\strut
\end{minipage} & \begin{minipage}[b]{0.14\columnwidth}\raggedright\strut
Definition\strut
\end{minipage}\tabularnewline
\midrule
\endhead
\begin{minipage}[t]{0.05\columnwidth}\raggedright\strut
Law of segregation\strut
\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
During gamete formation, the alleles for each gene segregate from each
other so that each gamete carries only one allele for each gene.\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.05\columnwidth}\raggedright\strut
Law of independent assortment\strut
\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
Genes for different traits can segregate independently during the
formation of gametes.\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.05\columnwidth}\raggedright\strut
Law of dominance\strut
\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
Some alleles are dominant while others are recessive; an organism with
at least one dominant allele will display the effect of the dominant
allele.\strut
\end{minipage}\tabularnewline
\bottomrule
\end{longtable}

\section{Punnett Square}\label{punnett-square}

The \href{https://en.wikipedia.org/wiki/Punnett_square}{Punnett square}
(Figures \ref{fig:punnett} and \ref{fig:punnettF1}) is a visual
representation of Mendelian inheritance and used to predict an outcome
of a particular cross or breeding experiment. It is named after
\href{https://en.wikipedia.org/wiki/Reginald_Punnett}{Reginald C.
Punnett}, who devised the approach. In our first experiment, both
parents are homozygous, one carrying two copies of the dominant allele
(R), the other two copies of the recessive (r) allele. Each parent can
only make gametes that have either the R (purple) or r (yellow) allele.
The Punnett square for the parental cross is shown in Figure
\ref{fig:punnett}

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/mendel/Punnett} 

}

\caption{Punnett square for homozygous cross.}\label{fig:punnett}
\end{figure}

The squares containing the single letters represent the possible
gametes. The squares with two letters represent the zygotes resulting
from the combination of the respective gametes. It can be easily seen
that all offspring will be heterozygous (Rr) and therefore purple. The
Punnett square for the F1 cross is depicted in Figure
\ref{fig:punnettF1}

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/mendel/PunnettF1} 

}

\caption{Punnett square for heterozygous cross.}\label{fig:punnettF1}
\end{figure}

\section{Test cross}\label{test-cross}

A \href{https://en.wikipedia.org/wiki/Test_cross}{test cross}, first
introduced by Gregor Mendel, involves the breeding of an individual with
a phenotypically recessive individual, in order to determine the
zygosity of the former by analyzing proportions of offspring phenotypes.
Zygosity can either be heterozygous or homozygous. Those that are
heterozygous have one dominant and one recessive allele. Individuals
that are homozygous dominant have two dominant alleles, and those that
are homozygous recessive have two recessive alleles.

The genotype that an offspring has for each of its genes is determined
by the alleles inherited from its parents. The combination of alleles is
a result of the maternal and paternal chromosomes contributed from each
gamete at fertilization of that offspring. During meiosis in gametes,
homologous chromosomes experience genetic recombination and segregate
randomly into haploid daughter cells, each with a unique combination of
maternally and paternally coded genes. Dominant alleles will override
the expression of recessive alleles.

Test crosses are used to test an individual's genotype by crossing it
with an individual of a known genotype. Individuals that show the
recessive phenotype are known to have a homozygous recessive genotype.
Individuals that show the dominant phenotype, however, may either be
homozygous dominant or heterozygous. The phenotypically dominant
organism is the individual in question in a test cross. The purpose of a
test cross is to determine if this individual is homozygous dominant or
heterozygous.

Test crosses involve breeding the individual in question with another
individual that expresses a recessive version of the same trait.
Analyzing the proportions of dominant and recessive offspring determines
if the individual in question is homozygous dominant or heterozygous. If
all offspring from the test cross display the dominant phenotype, the
individual in question is homozygous dominant; if half the offspring
display dominant phenotypes and half display recessive phenotypes, then
the individual is heterozygous. Since the homozygous recessive
individual can only pass on recessive alleles, the alleles the
individual in question passes on determine the phenotypes of the
offspring.

\section{Monohybrid Cross (Experiment
1)}\label{monohybrid-cross-experiment-1}

A \href{https://en.wikipedia.org/wiki/Monohybrid_cross}{monohybrid
cross} is a mating between two individuals with different variations at
one genetic trait of interest. The character(s) being studied in a
monohybrid cross are governed by two or multiple variations for a single
locus. A cross between two parents possessing a pair of contrasting
characters is known as monohybrid cross. To carry out such a cross, each
parent is chosen to be homozygous or true breeding for a given trait
(locus). When a cross satisfies the conditions for a monohybrid cross,
it is usually detected by a characteristic distribution of
second-generation (F2) offspring that is sometimes called the monohybrid
ratio.

Generally, the monohybrid cross is used to determine the dominance
relationship between two alleles. The cross begins with the parental (P)
generation. One parent is homozygous for one allele, and the other
parent is homozygous for the other allele. The offspring make up the
first filial (F1) generation. Every member of the F1 generation is
heterozygous and the phenotype of the F1 generation expresses the
dominant trait. Crossing two members of the F1 generation produces the
second filial (F2) generation. Probability theory predicts that three
quarters of the F2 generation will have the dominant allele's phenotype.
And the remaining quarter of the F2s will have the recessive allele's
phenotype. This predicted 3:1 phenotypic ratio assumes Mendelian
inheritance.

In the first experiment, we will study the result obtained from a
monohybrid cross. A strain of corn producing pure purple kernels (RR) is
crossed with a strain producing pure yellow kernels (rr). Purple is
dominant with the resulting F1 ears all bearing purple kernels. These
plants that are heterozygous for a single trait are called monohybrids.
When the F1 is self-pollinated, the resulting F2 ears bear both purple
and yellow kernels (Figure \ref{fig:monohybrid}).

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/mendel/Monohybrid_cross} 

}

\caption{Monohybrid cross}\label{fig:monohybrid}
\end{figure}

\subsection{Experimental Procedures}\label{experimental-procedures-2}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Count the number of purple and yellow kernels on one row of the F2 ear
  without removing the kernels.
\item
  Determine the ratio of purple to yellow.
\item
  Now tabulate the numbers obtained by each of your class mates in Table
  \ref{tab:mono} and add these figures to get a total.
\item
  Using the total numbers, determine a ratio of purple to yellow.
\end{enumerate}

\begin{longtable}[]{@{}cccc@{}}
\caption{\label{tab:mono} Monohybrid Cross.}\tabularnewline
\toprule
Row \# & purple & yellow & ratio\tabularnewline
\midrule
\endfirsthead
\toprule
Row \# & purple & yellow & ratio\tabularnewline
\midrule
\endhead
1 & & &\tabularnewline
2 & & &\tabularnewline
3 & & &\tabularnewline
4 & & &\tabularnewline
5 & & &\tabularnewline
6 & & &\tabularnewline
7 & & &\tabularnewline
8 & & &\tabularnewline
9 & & &\tabularnewline
Total & & &\tabularnewline
\bottomrule
\end{longtable}

\section{Dihybrid Cross (Experiment
2)}\label{dihybrid-cross-experiment-2}

In the second experiment, we will study the result obtained from a
\href{https://en.wikipedia.org/wiki/Dihybrid_cross}{dihybrid cross}. A
dihybrid cross is a cross between two different lines (varieties,
strains) that differ in two observed traits. In the name ``Dihybrid
cross'', the ``di'' indicates that there are two traits involved (in our
example designated R and Su), the ``hybrid'' means that each trait has
two different alleles (in our example R and r, or Su and su), and
``cross'' means that there are two individuals who are combining or
``crossing'' their genetic information. In our example, a pure strain of
corn producing purple-starchy kernels (RR SuSu) is crossed with a pure
strain producing yellow-sweet (rr susu). The starchy seeds are smooth,
the sweet seeds are wrinkled. The resulting F1 ears all bear
purple-starchy (smooth) kernels. Plants that are heterozygous for two
traits are called dihybrids. When the F1 is self-pollinated, the
resulting F2 generation contains various combinations (Figure
\ref{fig:dihybrid}).

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/mendel/Dihybrid_cross} 

}

\caption{Dihybrid cross}\label{fig:dihybrid}
\end{figure}

The rules of meiosis, as they apply to the dihybrid, are codified in
Mendel's first law and Mendel's second law, which are also called the
Law of Segregation and the Law of Independent Assortment, respectively
(Table \ref{tab:mendel}). For genes on separate chromosomes, each allele
pair showed independent segregation. If the first filial generation (F1
generation) produces four identical offspring, the second filial
generation, which occurs by crossing the members of the first filial
generation, shows a phenotypic (appearance) ratio of \textbf{9:3:3:1},
where:

\begin{itemize}
\tightlist
\item
  the \textbf{9} represents the proportion of individuals displaying
  both dominant traits
\item
  the first \textbf{3} represents the individuals displaying the first
  dominant trait and the second recessive trait
\item
  the second \textbf{3} represents those displaying the first recessive
  trait and second dominant trait
\item
  the \textbf{1} represents the homozygous, displaying both recessive
  traits.
\end{itemize}

\subsection{Experimental Procedures}\label{experimental-procedures-3}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Carefully count the number of kernels of each phenotype appearing on a
  row of F2 ear. Tabulate the results and determine the totals and total
  ratios in Table \ref{tab:di}.
\end{enumerate}

\begin{longtable}[]{@{}clllll@{}}
\caption{\label{tab:di} Dihybrid Cross.}\tabularnewline
\toprule
\begin{minipage}[b]{0.05\columnwidth}\centering\strut
Row \#\strut
\end{minipage} & \begin{minipage}[b]{0.19\columnwidth}\raggedright\strut
purple and starchy (smooth)\strut
\end{minipage} & \begin{minipage}[b]{0.19\columnwidth}\raggedright\strut
purple and sweet (wrinkled)\strut
\end{minipage} & \begin{minipage}[b]{0.19\columnwidth}\raggedright\strut
yellow and starchy (smooth)\strut
\end{minipage} & \begin{minipage}[b]{0.19\columnwidth}\raggedright\strut
yellow and sweet (wrinkled)\strut
\end{minipage} & \begin{minipage}[b]{0.04\columnwidth}\raggedright\strut
ratio\strut
\end{minipage}\tabularnewline
\midrule
\endfirsthead
\toprule
\begin{minipage}[b]{0.05\columnwidth}\centering\strut
Row \#\strut
\end{minipage} & \begin{minipage}[b]{0.19\columnwidth}\raggedright\strut
purple and starchy (smooth)\strut
\end{minipage} & \begin{minipage}[b]{0.19\columnwidth}\raggedright\strut
purple and sweet (wrinkled)\strut
\end{minipage} & \begin{minipage}[b]{0.19\columnwidth}\raggedright\strut
yellow and starchy (smooth)\strut
\end{minipage} & \begin{minipage}[b]{0.19\columnwidth}\raggedright\strut
yellow and sweet (wrinkled)\strut
\end{minipage} & \begin{minipage}[b]{0.04\columnwidth}\raggedright\strut
ratio\strut
\end{minipage}\tabularnewline
\midrule
\endhead
\begin{minipage}[t]{0.05\columnwidth}\centering\strut
1\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.04\columnwidth}\raggedright\strut
\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.05\columnwidth}\centering\strut
2\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.04\columnwidth}\raggedright\strut
\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.05\columnwidth}\centering\strut
3\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.04\columnwidth}\raggedright\strut
\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.05\columnwidth}\centering\strut
4\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.04\columnwidth}\raggedright\strut
\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.05\columnwidth}\centering\strut
5\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.04\columnwidth}\raggedright\strut
\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.05\columnwidth}\centering\strut
6\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.04\columnwidth}\raggedright\strut
\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.05\columnwidth}\centering\strut
7\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.04\columnwidth}\raggedright\strut
\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.05\columnwidth}\centering\strut
8\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.04\columnwidth}\raggedright\strut
\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.05\columnwidth}\centering\strut
9\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.04\columnwidth}\raggedright\strut
\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.05\columnwidth}\centering\strut
Total\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.19\columnwidth}\raggedright\strut
\strut
\end{minipage} & \begin{minipage}[t]{0.04\columnwidth}\raggedright\strut
\strut
\end{minipage}\tabularnewline
\bottomrule
\end{longtable}

\section{Review Questions}\label{review-questions-2}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  What is a gene?
\item
  What is an allele?
\item
  What are dominant and recessive alleles?
\item
  What is the genotype of an organism?
\item
  What is a trait?
\item
  What is the phenotype of an organism?
\item
  What is the genotype of the F1 generation of the monohybrid cross?
\item
  What is the phenotype of the F1 generation monohybrid cross?
\item
  What are the possible maternal and paternal genotypes of the F1
  gametes monohybrid cross?
\item
  What is the genotype of the parents of the dihybrid cross?
\item
  What are the phenotypes of the parents of the dihybrid cross?
\item
  What are the possible genotypes of the parent gametes of the dihybrid
  cross?
\item
  What is the genotype of the F1 generation of the dihybrid cross?
\item
  What is the phenotype of the F1 generation dihybrid cross?
\item
  What are the possible maternal and paternal genotypes of the F1
  gametes dihybrid cross?
\end{enumerate}

\chapter*{Molecular Genetics}\label{molecular-genetics}
\addcontentsline{toc}{chapter}{Molecular Genetics}

In the remaining 8 laboratory sessions of this course, we will clone a
portion of a glyceraldehyde-3-phosphate dehydrogenase (GAPDH) gene from
plants, insert this gene fragment into a plasmid vector, and analyze the
sequences of resulting clones using bioinformatics.

Specifically, we will:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Learn and practice some essential skills for the molecular laboratory
  (Chapter \ref{molecular-laboratory-skills})
\item
  Identify the plant or plants to be studied, extract the gDNA, and
  amplify a portion of the GAPC gene using PCR (Chapter
  \ref{dna-extraction-and-polymerase-chain-reaction}).
\item
  Assess the results of PCR, purify the PCR products and perform nested
  PCR (Chapter \ref{exonuclease-i-digestion-and-nested-pcr}).
\item
  Ligate (insert) the GAPC gene fragment into a plasmid vector and
  transform bacteria with the plasmid (Chapter
  \ref{agarose-gel-electrophoresis-ligation-and-transformation}).
\item
  Isolate the plasmid from the bacteria and confirm the presence of the
  insert by restriction enzyme digestion (Chapter
  \ref{plasmid-purification-and-restriction-digest}).
\item
  Prepare plasmid for DNA sequencing and send to a facility for
  sequencing (Chapter \ref{dna-sequencing}).
\item
  Obtain the sequence of the cloned GAPC gene fragment and analyze the
  cloned gene using bioinformatics (Chapter \ref{bioinformatics}).
\end{enumerate}

While we will chose and provide the plants, the reagents used in the
experiments will come in the form of ready made kits. In the early days
of molecular biology, researchers had to prepare all required materials
themselves including the laborious purification of enzymes. Today,
virtually any reagent that is required can be purchased
``off-the-shelf'', prepackaged and ready to use from a great number of
suppliers. One of the oldest companies specialized in supplying
materials for life science research laboratories are the
\href{https://en.wikipedia.org/wiki/Bio-Rad_Laboratories}{BioRad
Laboratories Inc.}, which developed the curriculum for these experiments
and supplies the materials under Bio-Rad Explorer\textsubscript{TM}
Cloning and Sequencing Explorer Series.

As our hands-on-time in the laboratory is limited, the RCC laboratory
technicians will help us by performing some behind the scenes work
between the lab sessions.

\href{https://en.wikipedia.org/wiki/Glyceraldehyde_3-phosphate_dehydrogenase}{Glyceraldehyde
3-phosphate dehydrogenase} (abbreviated as GAPDH or less commonly as
G3PDH) (EC 1.2.1.12) is an enzyme of \textasciitilde{}37kDa that
catalyzes the sixth step of glycolysis and thus serves to break down
glucose for energy and carbon molecules. In addition to this long
established metabolic function, GAPDH has recently been implicated in
several non-metabolic processes, including transcription activation,
initiation of apoptosis, ER to Golgi vesicle shuttling, and fast axonal,
or axoplasmic transport.

Because the GAPDH gene is often stably and constitutively expressed at
high levels in most tissues and cells, it is considered a housekeeping
gene. For this reason, GAPDH is commonly used by biological researchers
as a loading control for western blot and as a control for qPCR. Plants
have multiple GAPDH genes; the specific ones that have been selected for
cloning are the GAPC genes.

\chapter{Molecular Laboratory Skills}\label{molecular-laboratory-skills}

In this laboratory you will learn how to handle small amounts of liquids
using pipettes. A \href{https://en.wikipedia.org/wiki/Pipette}{pipette}
is a laboratory tool commonly used in biology, chemistry, and medicine
to draw up, transport and dispense a measured volume of liquid. Pipettes
come in several designs for various purposes with differing levels of
accuracy and precision, from single piece glass pipettes to more complex
adjustable or electronic pipettes. Micropipettes are used to handle
small (1 - 1000 µL) or even very small (\textless{} 1 µL) amounts of
liquids. Macropipettes are used to handle larger volumes. Accurate
pipetting is crucial to the success of molecular and biochemical
experiments. Mistakes during pipetting may cause experiments to fail or
to make them irreproducible.

Commonly, we will use two main types of pipettes:

\begin{itemize}
\tightlist
\item
  graduated pipettes
\item
  air displacement micropipettes
\end{itemize}

\href{https://en.wikipedia.org/wiki/Graduated_pipette}{Graduated
pipettes} or serological pipettes are a type macropipette made of glass
or plastic consisting of a long tube with a series of graduations, as on
a graduated cylinder or burette, to indicate different calibrated
volumes. Today we use manual (e.g.~a rubber bulb (Figure
\ref{fig:bulb})) or automatic propipetters to generate negative or
positive pressure to draw up and dispense the liquids.

(ref:bulb)
\href{https://commons.wikimedia.org/wiki/File:Pipetteerballon.png}{A
rubber bulb propipetter.}

\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/skills/Pipetteerballon} 

}

\caption{(ref:bulb)}\label{fig:bulb}
\end{figure}

\href{https://en.wikipedia.org/wiki/Air_displacement_pipette}{Air
displacement micropipettes} are a type of adjustable micropipette that
deliver a measured volume of liquid; depending on size, it could be
between about 0.1 µL to 1000 µL (1 ml). These pipettes require
disposable tips that come in contact with the fluid. Micropipettes (also
referred to as pipettors) come in several standard sizes with disposable
plastic tips that are labeled in different colors (Table
\ref{tab:standard}).

\begin{longtable}[]{@{}lllll@{}}
\caption{\label{tab:standard} Standard micropipette volumes and
colors.}\tabularnewline
\toprule
Name & Min. volume (µL) & Max. volume (µL) & Color & tip size
(µL)\tabularnewline
\midrule
\endfirsthead
\toprule
Name & Min. volume (µL) & Max. volume (µL) & Color & tip size
(µL)\tabularnewline
\midrule
\endhead
P2 & 0.2 & 2 & white & 10\tabularnewline
P10 & 1 & 10 & white & 10\tabularnewline
P20 & 2 & 20 & yellow & 200\tabularnewline
P100 & 20 & 100 & yellow & 200\tabularnewline
P200 & 50 & 200 & yellow & 200\tabularnewline
P1000 & 200 & 1000 & blue & 1000\tabularnewline
P5000 & 1000 & 5000 & green & 5000\tabularnewline
P10000 & 1000 & 10000 & red & 10000\tabularnewline
\bottomrule
\end{longtable}

These pipettes operate by piston-driven air displacement (Figure
\ref{fig:micropipette}) . A vacuum is generated by the vertical travel
of a metal or ceramic piston within an airtight sleeve. As the piston
moves upward, driven by the depression of the plunger, a vacuum is
created in the space left vacant by the piston. The liquid around the
tip moves into this vacuum (along with the air in the tip) and can then
be transported and released as necessary. These pipettes are capable of
being very precise and accurate. However, since they rely on air
displacement, they are subject to inaccuracies caused by the changing
environment, particularly temperature and user technique. For these
reasons this equipment must be carefully maintained and calibrated, and
users must be trained to exercise correct and consistent technique.











\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{./figures/skills/Gilson_schematic} 

}

\caption{Schematic of an air displacement pipette. The
``Digital volume indicator'' is a dial display that indicates the digits
(i.e.~unrelated to electronic). The components vary between brands and
different volume sizes have different components, for example the piston
in a P2 is needle-like and can be separated with ease from the piston
assembly whereas in a P10 it is drum-like and over 1 cm in diameter and
is enclosed in plastic. By Squidonius {[}CC BY-SA 3.0{]}
(\url{https://creativecommons.org/licenses/by-sa/3.0}), from Wikimedia
Commons.}\label{fig:micropipette}
\end{figure}

\BeginKnitrBlock{rmdnote}
\textbf{The recommendation is for using a pipette whose size is nearest
to the volume being worked with.}
\EndKnitrBlock{rmdnote}

All adjustable pipettes should be calibrated and checked periodically.
The simplest way to accomplish this is to pipette a specific volume of a
solution with a known density (remember
\(density = \frac{mass\ (g)}{volume (ml)}\). The density of distilled
water is 1.0 g/ml. Therefore, the accuracy of the pipette or your
pipetting skills (if you use an already calibrated pipette as in these
experiments) can be checked by determining the mass of a specific volume
of water using an analytical balance. We will use this technique in the
following two pipetting exercises.

\section{Exercise 1: using graduated (serological)
macropipettes}\label{exercise-1-using-graduated-serological-macropipettes}

\subsection{Experimental procedures}\label{experimental-procedures-4}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Obtain a graduated pipette.
\item
  Notice that the tapered delivery tip at one end of the pipette.
\item
  Always hold the pipette upright.
\item
  Attach the rubber bulb or mechanical pipetter to the other (blunt) end
  of the pipette.
\item
  Dip the pipette into the solution without touching the bottom of the
  vessel containing the liquid you want to draw up.
\item
  Draw up the liquid until the meniscus reaches the position on the
  scale printed on the pipette corresponding to the desired volume.
\item
  Dispense the liquid into a weigh boat.
\item
  Determine the mass of the dispensed volume and record your result in
  Table \ref{tab:graduated}.
\item
  Empty the weigh boat, put it on the balance and set the balance to
  zero.
\item
  Calculate the average for each triplicate set, and the percentage
  error (relative to the average value) for each measurement and enter
  your results in Table \ref{tab:graduated}.
\end{enumerate}

\[ Error (\%) = \frac{expected\ mass - experimentally\ determined\ mass}{expected\ mass} \times 100\% \]

Your results should be within ±5\% of the correct volume set on the dial
of the pipette.

\begin{longtable}[]{@{}lllllll@{}}
\caption{\label{tab:graduated} Experimental determination of graduated
pipette volumes.}\tabularnewline
\toprule
Name & Volume (ml) & Trial 1 & Trial 2 & Trial 3 & Average & Error
(\%)\tabularnewline
\midrule
\endfirsthead
\toprule
Name & Volume (ml) & Trial 1 & Trial 2 & Trial 3 & Average & Error
(\%)\tabularnewline
\midrule
\endhead
1 ml & 0.1 & & & & &\tabularnewline
1 ml & 0.5 & & & & &\tabularnewline
1 ml & 1 & & & & &\tabularnewline
5 ml & 0.5 & & & & &\tabularnewline
5 ml & 1 & & & & &\tabularnewline
5 ml & 5 & & & & &\tabularnewline
10 ml & 1 & & & & &\tabularnewline
10 ml & 5 & & & & &\tabularnewline
10 ml & 10 & & & & &\tabularnewline
20 ml & 5 & & & & &\tabularnewline
20 ml & 10 & & & & &\tabularnewline
20 ml & 20 & & & & &\tabularnewline
\bottomrule
\end{longtable}

\section{Exercise 2: using air displacement
micropipettes}\label{exercise-2-using-air-displacement-micropipettes}

\subsection{Experimental procedures}\label{experimental-procedures-5}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Your laboratory instructor will demonstrate the use of the
  micropipette and analytical balance before you begin your own
  experiment.
\item
  Obtain a micropipette.
\item
  Identify the main parts of the micropipette (Figure
  \ref{fig:micropipette})

  \begin{itemize}
  \tightlist
  \item
    Plunger button
  \item
    Tip ejector button
  \item
    Volume adjustment dial
  \item
    Digital volume indicator Shaft
  \item
    Attachment point for a disposable tip
  \end{itemize}
\item
  Set the pipetting volume. Do not adjust the volume beyond the maximum
  setting! This will damage the pipettor.
\item
  Attach a tip to the pipette.
\item
  Hold the pipette vertically and press the plunger button to the first
  stop.
\item
  Dip the tip into the sample and slowly release it, moving the plunger
  button to the home position.
\item
  Press the plunger button all the way to the second stop to dispense
  the sample.
\item
  Now, carefully draw up and dispense the volumes of distilled water
  listed in Table \ref{tab:micro} with the indicated micropipette into
  the weigh boat. Always keep an eye on the tip and watch the sample:
  note when it enters the tip and enters the final location. Do not
  assume some ``got in'', especially with small volumes and clear
  solutions. Remember the approximate levels that particular volumes
  fill the tip -- this will allow you to check your pipetting visually.
\item
  Record the value of the mass of the dispensed water.
\item
  Set the balance back to zero.
\item
  Repeat twice.
\item
  Eject the tip.
\item
  Calculate the average for each triplicate set, and the percentage
  error (relative to the average value) for each measurement and enter
  your results in Table \ref{tab:micro}.
\end{enumerate}

\[ Error (\%) = \frac{expected\ mass - experimentally\ determined\ mass}{expected\ mass} \times 100\% \]

Your results should be within ±5\% of the correct volume set on the dial
of the pipette.

\begin{longtable}[]{@{}lllllll@{}}
\caption{\label{tab:micro} Experimental determination of micropipette
volumes.}\tabularnewline
\toprule
Name & Volume (µL) & Trial 1 & Trial 2 & Trial 3 & Average & Error
(\%)\tabularnewline
\midrule
\endfirsthead
\toprule
Name & Volume (µL) & Trial 1 & Trial 2 & Trial 3 & Average & Error
(\%)\tabularnewline
\midrule
\endhead
P10 & 5 & & & & &\tabularnewline
P10 & 10 & & & & &\tabularnewline
P20 & 5 & & & & &\tabularnewline
P20 & 10 & & & & &\tabularnewline
P20 & 20 & & & & &\tabularnewline
P200 & 20 & & & & &\tabularnewline
P200 & 50 & & & & &\tabularnewline
P200 & 200 & & & & &\tabularnewline
P1000 & 200 & & & & &\tabularnewline
P1000 & 500 & & & & &\tabularnewline
P1000 & 1000 & & & & &\tabularnewline
\bottomrule
\end{longtable}

Obtain an empty microcentrifuge tube and perform the follwoing pipetting
exercises transferring distilled water from a beaker to and from a
microcentrifuge tube:

\begin{itemize}
\item
  Use a fresh tip each time.
\item
  Do not touch the tube with the pipette, only with the tip.
\item
  Large volume:

  \begin{enumerate}
  \def\labelenumi{\arabic{enumi}.}
  \tightlist
  \item
    Add 265 µL to a tube
  \item
    Add 435 µL to the same tube
  \item
    Remove 615 µL from the tube
  \item
    Remove 32 µL from the tube
  \end{enumerate}
\item
  Small volume:

  \begin{enumerate}
  \def\labelenumi{\arabic{enumi}.}
  \tightlist
  \item
    Add 5.3 µL to a tube
  \item
    Add 15.5 µL to the same tube
  \item
    Remove 12.6 µL from the tube
  \item
    Remove 7.2 µL from the tube
  \end{enumerate}
\item
  Very small volume:

  \begin{enumerate}
  \def\labelenumi{\arabic{enumi}.}
  \tightlist
  \item
    Add 1.3 µL
  \item
    Add 3.8 µL
  \item
    Remove 4.2 µL
  \item
    Remove 0.7 µL
  \end{enumerate}
\end{itemize}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Hold the pipette almost vertically in the plam of your right hand (if
  you are right handed).
\item
  Depress plunger to first stop and hold. Dip tip 2-4 mm into the beaker
  with distilled water.
\item
  Gently release the plunger. If you release too quickly, you will
  create aerosols (small droplets) which will contaminate the pipette.
\item
  Wait for a second or so and confirm that all the liquid has been taken
  up.
\item
  Touch the pipette tip to the inside wall of the beaker to dislodge any
  remaining droplets adhering to the outside of the tip.
\item
  Check that there is no air space at the very end of the tip.
\item
  Put the tube containing the sample in the tube rack and pick up the
  tube that you want to add the sample to. Hold the tube between your
  thumb and index finger of your left hand.
\item
  To expel sample into the tube, touch the tip to wall of the tube and
  slowly depress the plunger to the first, and then to the second stop
  to expel the liquid.
\item
  Keeping the plunger at the second stop, slide the tip out of the
  fluid, along the tube wall and out of the tube.
\item
  Look for the small droplet of liquid in the tube to make sure that it
  was transferred.
\end{enumerate}

\section{Exercise 3: using a
microcentrifuge}\label{exercise-3-using-a-microcentrifuge}

A \href{https://en.wikipedia.org/wiki/Laboratory_centrifuge}{laboratory
centrifuge} is a piece of laboratory equipment, driven by a motor, which
spins liquid samples at high speed to separate heavy materials in a
substance from lighter materials in the same substance. The heavier
particles will clump at the bottom and form a pellet, while the solution
above the pellet is referred to as the supernatant. Centrifuge speeds
are measured in RPM (revolutions per minute) or RCF (relative
centrifugal force; often referred to as `g-force').

Like all other centrifuges, laboratory centrifuges work by the
sedimentation principle, where the centripetal acceleration is used to
separate substances of greater and lesser density.

Microcentrifuges sit on the bench top and are used to spin small (2 ml
or less) liquid samples at high speeds (up to tens of thousands times
g-force). Centrifugation of small samples is important for many
biological applications, such as pelleting nucleic acids or proteins
from solution. Generally, we also use ``microcenfuges'' to gather the
typically very small volumes of liquids that we work with into the
bottom of the tube.

All centrifuges must be balanced or they will be unstable and will not
spin at the correct speed. To balance a centrifuge, tubes are
symmetrically arranged around the center. An odd number of tubes can be
balanced with empty tubes or ``balance'' tubes filled with water (always
label the ALL tubes appropriately). It is a good practice to tubes
always place the tubes in the centrifuge such that the hinges face
outward. This way, you will know that the pellet forms directly below
the hinge of the microcentrigue tube which helps when the pellet is
small and difficult to see.

\subsection{Experimental procedures}\label{experimental-procedures-6}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Obtain an empty microcentrifuge tube.
\item
  Label your tube.
\item
  Transfer 200 µL of xxx to the microcentrifuge tube.
\item
  Put your tubes into the centrifuge with the hinge facing outward.
\item
  Make sure that the centrifuge is properly balanced.
\item
  Spin the tubes at 10,000 rpm for 30 seconds,
\item
  Use a pipette to transfer the supernatant to a new tube, being sure to
  leave the pellet untouched.
\item
  Eject the tip into container provided for discarded tubes, by pressing
  the tip ejector button.
\item
  Show your instructor the two tubes - one with supernatant, one with
  the silica resin pellet.
\item
  Your instructor will keep the tube with the silica resin. Discard the
  tube with the supernatant.
\end{enumerate}

\section{Review Questions}\label{review-questions-3}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
\end{enumerate}

\chapter{DNA Extraction and Polymerase Chain
Reaction}\label{dna-extraction-and-polymerase-chain-reaction}

\section{DNA}\label{dna}

Deoxyribonucleic acid DNA) is a thread-like chain of nucleotides
carrying the genetic instructions used in the growth, development,
functioning and reproduction of all known living organisms and many
viruses. DNA and ribonucleic acid (RNA) are nucleic acids; alongside
proteins, lipids and complex carbohydrates (polysaccharides), they are
one of the four major types of macromolecules that are essential for all
known forms of life. Most DNA molecules consist of two biopolymer
strands coiled around each other to form a double helix.

The two DNA strands are called polynucleotides since they are composed
of simpler monomer units called nucleotides. Each nucleotide is composed
of one of four nitrogen-containing nucleobases (cytosine {[}C{]},
guanine {[}G{]}, adenine {[}A{]} or thymine {[}T{]}), a sugar called
deoxyribose, and a phosphate group. The nucleotides are joined to one
another in a chain by covalent bonds between the sugar of one nucleotide
and the phosphate of the next, resulting in an alternating
sugar-phosphate backbone. The nitrogenous bases of the two separate
polynucleotide strands are bound together, according to base pairing
rules (A with T and C with G), with hydrogen bonds to make
double-stranded DNA.

The complementary nitrogenous bases are divided into two groups,
pyrimidines and purines. In a DNA molecule, the pyrimidines are thymine
and cytosine, the purines are adenine and guanine.

DNA stores biological information. The DNA backbone is resistant to
cleavage, and both strands of the double-stranded structure store the
same biological information. This information is replicated as and when
the two strands separate. A large part of DNA (more than 98\% for
humans) is non-coding, meaning that these sections do not serve as
patterns for protein sequences.

The two strands of DNA run in opposite directions to each other and are
thus antiparallel. Attached to each sugar is one of four types of
nucleobases (informally, bases). It is the sequence of these four
nucleobases along the backbone that encodes biological information. RNA
strands are created using DNA strands as a template in a process called
transcription. Under the genetic code, these RNA strands are translated
to specify the sequence of amino acids within proteins in a process
called translation.

Within eukaryotic cells, DNA is organized into long structures called
chromosomes. During cell division these chromosomes are duplicated in
the process of DNA replication, providing each cell its own complete set
of chromosomes. Eukaryotic organisms (animals, plants, fungi and
protists) store most of their DNA inside the cell nucleus and some of
their DNA in organelles, such as mitochondria or chloroplasts. In
contrast prokaryotes (bacteria and archaea) store their DNA only in the
cytoplasm. Within the eukaryotic chromosomes, chromatin proteins such as
histones compact and organize DNA. These compact structures guide the
interactions between DNA and other proteins, helping control which parts
of the DNA are transcribed.

DNA was first isolated by
\href{https://en.wikipedia.org/wiki/Friedrich_Miescher}{Friedrich
Miescher} in 1869. Its molecular structure was first identified by
\href{https://en.wikipedia.org/wiki/James_Watsoni}{James Watson} and
\href{https://en.wikipedia.org/wiki/Francis_Crick}{Francis Crick} at the
Cavendish Laboratory within the University of Cambridge in 1953, whose
model-building efforts were guided by X-ray diffraction data acquired by
\href{https://en.wikipedia.org/wiki/Raymond_Gosling}{Raymond Gosling},
who was a post-graduate student of
\href{https://en.wikipedia.org/wiki/Rosalind_Franklin}{Rosalind
Franklin}.

DNA isolation is a process of purification of DNA from sample using a
combination of physical and chemical methods. Currently it is a routine
procedure in molecular biology.

\section{Polymerase chain reaction}\label{polymerase-chain-reaction}

\href{https://en.wikipedia.org/wiki/Polymerase_chain_reaction}{Polymerase
chain reaction} (PCR) is a technique used in molecular biology to
amplify a single copy or a few copies of a segment of DNA across several
orders of magnitude, generating thousands to millions of copies of a
particular DNA sequence. PCR was developed in 1983 by
\href{https://en.wikipedia.org/wiki/Kary_Mullis}{Kary Mullis} when he
was an employee of the
\href{https://en.wikipedia.org/wiki/Cetus_Corporation}{Cetus
Corporation}, one of the first biotech companies. He won the Nobel Prize
in Chemistry in 1993 for the invention of PCR. It is an easy, cheap, and
reliable way to repeatedly replicate a focused segment of DNA, a concept
which is applicable to numerous fields in modern biology and related
sciences. PCR is probably the most widely used technique in molecular
biology and has revolutionized the field.

PCR is now a common and often indispensable technique used in clinical
and research laboratories for a broad variety of applications. These
include DNA cloning for sequencing, gene cloning and manipulation, gene
mutagenesis; construction of DNA-based phylogenies, or functional
analysis of genes; diagnosis and monitoring of hereditary diseases;
amplification of ancient DNA; analysis of genetic fingerprints for DNA
profiling (for example, in forensic science and parentage testing); and
detection of pathogens in nucleic acid tests for the diagnosis of
infectious diseases. In 1993, Mullis was awarded the Nobel Prize in
Chemistry along with Michael Smith for his work on PCR.

PCR is now a common and often indispensable technique used in clinical
and research laboratories for a broad variety of applications. These
include DNA cloning for sequencing, gene cloning and manipulation, gene
mutagenesis; construction of DNA-based phylogenies, or functional
analysis of genes; diagnosis and monitoring of hereditary diseases;
amplification of ancient DNA; analysis of genetic fingerprints for DNA
profiling (for example, in forensic science and parentage testing); and
detection of pathogens in nucleic acid tests for the diagnosis of
infectious diseases.

The vast majority of PCR methods rely on thermal cycling, which involves
exposing the reactants to cycles of repeated heating and cooling,
permitting different temperature-dependent reactions---specifically, DNA
melting and enzyme-driven DNA replication---to quickly proceed many
times in sequence. Primers (short DNA fragments) containing sequences
complementary to the target region, along with a
\href{https://en.wikipedia.org/wiki/DNA_polymerase}{DNA polymerase}
(e.g.~Taq polymerase), after which the method is named, enable selective
and repeated amplification. As PCR progresses, the DNA generated is
itself used as a template for replication, setting in motion a chain
reaction in which the original DNA template is exponentially amplified.
The simplicity of the basic principle underlying PCR means it can be
extensively modified to perform a wide array of genetic manipulations.
PCR is not generally considered to be a recombinant DNA method, as it
does not involve cutting and pasting DNA, only amplification of existing
sequences.

Almost all PCR applications employ a heat-stable DNA polymerase, such as
Taq polymerase, an enzyme originally isolated from the thermophilic
bacterium \emph{Thermus aquaticus}. If heat-susceptible DNA polymerase
is used, it will denature every cycle at the denaturation step. Before
the use of Taq polymerase, DNA polymerase had to be manually added every
cycle, which was a tedious and costly process. This DNA polymerase
enzymatically assembles a new DNA strand from free nucleotides, the
building blocks of DNA, by using single-stranded DNA as a template and
DNA oligonucleotides (the primers mentioned above) to initiate DNA
synthesis.

In the first step, the two strands of the DNA double helix are
physically separated at a high temperature in a process called DNA
melting. In the second step, the temperature is lowered and the two DNA
strands become templates for DNA polymerase to selectively amplify the
target DNA. The selectivity of PCR results from the use of primers that
are complementary to sequence around the DNA region targeted for
amplification under specific thermal cycling conditions.

The PCR, like recombinant DNA technology, has had an enormous impact in
both basic and diagnostic aspects of molecular biology because it can
produce large amounts of a specific DNA fragment from small amounts of a
complex template.
\href{https://en.wikipedia.org/wiki/Recombinant_DNA}{Recombinant DNA}
techniques create molecular clones by conferring on a specific sequence
the ability to replicate by inserting it into a vector and introducing
the vector into a host cell. PCR represents a form of ``in vitro
cloning'' that can generate, as well as modify, DNA fragments of defined
length and sequence in a simple automated reaction. In addition to its
many applications in basic molecular biological research, PCR promises
to play a critical role in the identification of medically important
sequences as well as an important diagnostic one in their detection.

\section{DNA Extraction}\label{dna-extraction}

\section{Experimental Procedures}\label{experimental-procedures-7}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Obtain two microcentrifuge tubes and label them with your initials.
\item
  Add 200 µl of lysis buffer into each tube.
\item
  Weigh 50--100 mg of plant tissue. Record the weight of the tissue.
\end{enumerate}

\begin{longtable}[]{@{}lll@{}}
\caption{\label{tab:plant} Plant material record.}\tabularnewline
\toprule
Plant Name & Part of Plant (Leaf, Root, etc.) & Weight
(mg)\tabularnewline
\midrule
\endfirsthead
\toprule
Plant Name & Part of Plant (Leaf, Root, etc.) & Weight
(mg)\tabularnewline
\midrule
\endhead
& &\tabularnewline
& &\tabularnewline
& &\tabularnewline
& &\tabularnewline
\bottomrule
\end{longtable}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{3}
\tightlist
\item
  Chop the plant tissue into small (1-2 mm in diameter) pieces using a
  razor blade.
\item
  Add the plant material to the microcentrifuge tubes with the lysis
  buffer.
\item
  Using a micropestle, carefully grind the plant tissue into fine
  particles (until no more chunks are visible). Be careful not to spill
  the lysis buffer. If any material gets stuck at the bottom of the tube
  use a clean pipet tip to dislodge it and continue grinding.
\item
  Add an additional 500 µl of lysis buffer.
\item
  Grind until the lysate is homogeneous.
\item
  Close the microcentrifuge tube.
\item
  Spin for 5 min at full speed in a microcentrifuge at room temperature.
  Make sure to place the tubes in the rotor so that the microcentrifuge
  is balanced.\\
\item
  While tubes are centrifuging, label two microcentrifuge tubes for each
  extract.
\item
  Add 500 µl of 70\% ethanol to each tube.
\item
  Retrieve your tubes from the microcentrifuge.
\item
  Carefully remove 400 µl of supernatant without disturbing the pellet
  and add it to the new tubes containing the 500 µl of 70\% ethanol.
  Avoid transferring any solid plant material to the ethanol; if
  necessary, recentrifuge the lysate. Mix the lysate and ethanol
  thoroughly by pipetting up and down. Cap the tubes.
\end{enumerate}

\BeginKnitrBlock{rmdimportant}
\textbf{If a precipitate is visible, spin the microcentrifuge tube again
for 5 min in a microcentrifuge at full speed to pellet the precipitated
material and use the supernatant for the next stages.}
\EndKnitrBlock{rmdimportant} 10. Label the top of a mini column for each
sample with your initials. Place the columns in 2 ml capless collection
tubes. 11. Remove 800 µl of cleared plant lysate from your
microcentrifuge tubea and aad it to the column.

\BeginKnitrBlock{rmdimportant}
\textbf{Be sure that the lysate that is added to the column is clear and
does not contain any precipitate. If necessary, centrifuge the lysate
again to pellet any plant material prior to loading the lysate on the
column.}
\EndKnitrBlock{rmdimportant}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{11}
\tightlist
\item
  Place the capless collection tubes containing the columns into the
  microcentrifuge. Ensure that the centrifuge is balanced. Spin for 1
  min at full speed at room temperature.
\item
  Discard the flowthrough from the collection tubes.
\end{enumerate}

\BeginKnitrBlock{rmdimportant}
\textbf{If all of the supernatant does not pass through column,
centrifuge again for 1 min. If there is any supernatant left in the
column, carefully remove and discard it. Do not disrupt the bed of the
column.}
\EndKnitrBlock{rmdimportant}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{12}
\tightlist
\item
  Add 700 µl of wash buffer to each column.
\item
  Spin at full speed at room temperature for 1 min. Discard the
  flowthrough.
\item
  Repeat steps 11 to 14 two more times for a total of 3 washes.
\item
  After the final wash step, discard the flowthrough and put the columns
  back into the collection tubes. Centrifuge the columns for 2 min at
  full speed in the microcentrifuge at room temperature. This step
  removes the residual ethanol which will interfere with PCR if present.
  15.Label a fresh microcentrifuge tube and transfer the column to the
  new tube.
\item
  Obtain the sterile water from the 70°C heat block.
\item
  Immediately add 80 µl of 70°C sterile water to the bed of each column,
  making certain that the water wets the column bed and let it sit for 1
  min.
\item
  Place the column in the microcentrifuge tube into the microcentrifuge.
  Orient the loose cap of the microcentrifuge tube downward, toward the
  center of the rotor, to minimize friction and damage to the cap during
  centrifugation.
\item
  Spin at full speed in the microcentrifuge at room temperature for 2
  min.
\item
  Remove the spin column from the microcentrifuge tube.
\end{enumerate}

\section{GAPDH Polymerase Chain
Reaction}\label{gapdh-polymerase-chain-reaction}

\section{Experimental Procedures}\label{experimental-procedures-8}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Obtain a PCR tube.
\item
  Label the tube with your initials.
\item
  Add 20 µl of 2× MMIP (\textbf{m}aster \textbf{m}ix with
  \textbf{i}nitial \textbf{p}rimers) into each PCR tube.
\item
  Add 15 µl of sterile water to each tube.
\item
  Add 5 µl of the DNA template to each tube and gently pipet up and down
  to mix reagents.
\item
  Place your PCR tubes into the thermal cycler.
\item
  The PCR reaction will run for the next several hours using the program
  listed in Table \ref{table:pcr}.
\item
  The laboratory technicians will remove the PCR tubes from the
  thermocycler after the PCR has finished and store the tubes at -20°C
  until we continue our experiments next week.
\end{enumerate}

\begin{longtable}[]{@{}llll@{}}
\caption{\label{tab:pcr} PCR protocol for GAPDH
amplification.}\tabularnewline
\toprule
PCR Step & Temperature & Time & Number of cycles\tabularnewline
\midrule
\endfirsthead
\toprule
PCR Step & Temperature & Time & Number of cycles\tabularnewline
\midrule
\endhead
Initial denaturation & 95°C & 5 min & 1\tabularnewline
Denaturation & 95°C & 1 min & 40\tabularnewline
Annealing & 52°C & 1 min & 40\tabularnewline
Extension & 72°C & 2 min & 40\tabularnewline
Final extension & 72°C & 6 min & 1\tabularnewline
Hold & 15°C & hold &\tabularnewline
\bottomrule
\end{longtable}

\chapter{Exonuclease I Digestion And Nested
PCR}\label{exonuclease-i-digestion-and-nested-pcr}

\section{Exonuclease I Digestion}\label{exonuclease-i-digestion}

Exonucleases are enzymes that work by cleaving nucleotides one at a time
from the end (exo) of a polynucleotide chain. A hydrolyzing reaction
that breaks phosphodiester bonds at either the 3' or the 5' end occurs.
Its close relative is the endonuclease, which cleaves phosphodiester
bonds in the middle (endo) of a polynucleotide chain. Exonuclease I
(Phosphodiesterase I) hydrolytically removes 5'-nucleotides successively
from the 3'-hydroxy termini of 3'-hydroxy-terminated oligonucleotides.
We use exonuclease I to digest the oligonucleotide primers from the firs
PCR reaction so that they will not allow for PCR amplification in the
second PCR. After the initial PCR primers have been digested,
exonuclease I also needs to be inactivated before it is introduced into
fresh PCR reactions to prevent it digesting the nested PCR primers.

\section{Experimental Procedures}\label{experimental-procedures-9}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Add 1 µl of exonuclease I to each PCR tube containing the amplified
  gDNA.
\item
  Mix well by pipetting up and down.
\item
  Incubate at 37°C for 15 min.
\item
  Incubate at 80°C for 15 min to heat-inactivate the exonuclease I
  enzyme.
\end{enumerate}

\section{Nested polymerase chain
reaction}\label{nested-polymerase-chain-reaction}

Nested polymerase chain reaction (Nested PCR) is a modification of
polymerase chain reaction intended to reduce non-specific binding in
products due to the amplification of unexpected primer binding sites.

Nested polymerase chain reaction involves two sets of primers, used in
two successive runs of polymerase chain reaction, the second set
intended to amplify a secondary target within the first run product.
This allows amplification for a low number of runs in the first round,
limiting non-specific products. The second nested primer set should only
amplify the intended product from the first round of amplification and
not non-specific product. This allows running more total cycles while
minimizing non-specific products. This is useful for very rare templates
or PCR with high background.

\section{Experimental Procedures}\label{experimental-procedures-10}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Label a microcentrifuge tube for each exonuclease-treated initial PCR
  tube.
\item
  To dilute each initial PCR sample 20× (to 1/50 the original
  concentration), add 98 µl of sterile water into each of the labeled
  microcentrifuge tubes.
\item
  Using a fresh tip each time, pipet 2 µl of the initial PCR into the
  corresponding microcentrifuge tube. Close the cap.
\item
  Vortex or flick the tube with your finger to mix. Spin briefly in a
  microfuge to collect the liquid at the bottom of the tube.
\item
  Label 3 fresh PCR tubes \#1-3 and your initials.
\item
  Place them in PCR tube adaptors on ice.
\item
  Pipet 20 µl of 2× MMNP into each of the three PCR tubes.
\item
  Using a fresh tip, pipet 20 µl of the initial PCR into tube \#1. Close
  the cap.
\item
  Use a fresh pipet tip to add 20 µl of plasmid DNA (positive control)
  to tube \#2. Gently pipet up and down to mix the reagents. Recap the
  tubes.
\item
  Use a fresh pipet tip to add 20 µl of plasmid DNA (positive control)
  to tube \#2.
\item
  Use a fresh pipet tip to add 20 µl of sterile water (negative control)
  to PCR tube \#3.
\item
  Place the PCR tubes into the thermal cycler. The PCR will run for the
  next several hours using the Nested GAPDH PCR program listed in Table
  \ref{tab:nestedpcr}.
\end{enumerate}

\begin{longtable}[]{@{}llll@{}}
\caption{\label{tab:nestedpcr} PCR protocol for second round (nested) GAPDH
amplification.}\tabularnewline
\toprule
PCR Step & Temperature & Time & Number of cycles\tabularnewline
\midrule
\endfirsthead
\toprule
PCR Step & Temperature & Time & Number of cycles\tabularnewline
\midrule
\endhead
Initial denaturation & 95°C & 5 min & 1\tabularnewline
Denaturation & 95°C & 1 min & 40\tabularnewline
Annealing & 52°C & 1 min & 40\tabularnewline
Extension & 72°C & 2 min & 40\tabularnewline
Final extension & 72°C & 6 min & 1\tabularnewline
Hold & 15°C & hold &\tabularnewline
\bottomrule
\end{longtable}

\chapter{Agarose Gel Electrophoresis, Ligation and
Transformation}\label{agarose-gel-electrophoresis-ligation-and-transformation}

\section{Agarose gel electrophoresis}\label{agarose-gel-electrophoresis}

Electrophoresis (from the Greek ``electrophoron'' meaning ``to bear
electrons'') is the motion of dispersed particles relative to a fluid
under the influence of a spatially uniform electric field.

Agarose gel electrophoresis is a method of gel electrophoresis used in
biochemistry, molecular biology, genetics, and clinical chemistry to
separate a mixed population of macromolecules such as DNA or proteins in
a matrix of agarose, one of the two main components of agar. The
proteins may be separated by charge and/or size, and the DNA and RNA
fragments by length. Biomolecules are separated by applying an electric
field to move the charged molecules through an agarose matrix, and the
biomolecules are separated by size in the agarose gel matrix.

Agarose gels are easy to cast and are particularly suitable for
separating DNA of size range most often encountered in laboratories. The
separated DNA is visualized using a stain, most commonly under UV light,
and the DNA fragments can be extracted from the gel with relative ease.
Most agarose gels used are made using 0.5--3\% agarose which is
dissolved in a suitable electrophoresis buffer.

Agarose gel is easy to cast, has relatively fewer charged groups, and is
particularly suitable for separating DNA of size range most often
encountered in laboratories, which accounts for the popularity of its
use. The separated DNA may be viewed with stain, most commonly under UV
light, and the DNA fragments can be extracted from the gel with relative
ease. Most agarose gels used are between 0.7--2\% dissolved in a
suitable electrophoresis buffer.

\section{Experimental Procedures}\label{experimental-procedures-11}

\BeginKnitrBlock{rmdnote}
Take special care when pipetting very small volumes. Make sure only the
soft stop of the pipet is used, when pulling up reagents even though it
may feel like a very small movement. Also, look at the end of the pipet
tip to be sure that the correct volume of reagent is in the tip. After
adding the reagent to the tube, be sure that the pipet tip is empty.
Never reuse a pipet tip.
\EndKnitrBlock{rmdnote}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  Get the Erlenmeyer flask containing 1 g of agarose powder
\item
  Add 100 ml of 1× TAE (Tris base, acetic acid, EDTA) running buffer.
\item
  Add 10 µl of SybrGreen™ dye (10,000× stock solution).
\item
  Heat in the microwave at full power for 1 minute.
\item
  Swirl to make sure that all powder has dissolved, and the solution is
  clear.
\item
  Add the comb into the comb slot.
\item
  Pour the solution onto the gel tray in the gel box.
\item
  Label a microcentrifuge tube for each PCR sample.
\item
  Transfer 20 µl of each of first-round PCR reaction into the labeled
  microcentrifuge tube. Add 4 µl of loading dye and stain into the tube.
  Pipette up and down to mix.
\item
  To assess the success of the nested PCR round, pipet 5 µl of each
  nested PCR into a microcentrifuge tube and mix it with 1 µl of 6×
  loading dye and stain. A smaller volume of the nested PCR is loaded
  because nested PCR is usually more efficient and produces very intense
  bands that can obscure bands in adjacent wells if samples are
  overloaded.
\item
  The gel will need about 30 minutes to solidify.
\item
  Start setting up the PCR product purification.
\end{enumerate}

\section{PCR Product Purification}\label{pcr-product-purification}

\section{Experimental Procedures}\label{experimental-procedures-12}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Label a PCR Kleen spin column and a capless collection tube with your
  initials.
\item
  Label a capped microcentrifuge tube with your initials, ``purified PCR
  product,'' and the plant name.
\item
  Resuspend the beads in the PCR Kleen spin column by vortexing. Return
  the beads to the bottom of the column with a sharp downward flick.
\item
  Snap the bottom off the spin column, remove the cap, and place the
  column in the capless collection tube. Discard cap and bottom of
  column.
\item
  Place the spin column, still in the capless collection tube, into the
  microcentrifuge. Make sure that your tube is placed in the rotor with
  another group's or with a balance tube so that the microcentrifuge is
  balanced. Centrifuge columns at 735×g for 2 min. Do not use the top
  speed of the microcentrifuge for this step.
\item
  Move the spin column to the labeled microcentrifuge tube. Discard the
  flowthrough and the collection tube.
\item
  Pipet 30 µl of the nested PCR product onto the top of the column bed
  in the spin column, without disturbing the resin (or column bed). Save
  5 µl of this original (unpurified) yellow PCR sample and mix it with 1
  µl of 6× loading dye and stain for gel electrophoresis.
\item
  Place the column in the labeled microcentrifuge tube into the
  microcentrifuge. It is best to orient the cap of the microcentrifuge
  tube downward, toward the center of the rotor, to minimize friction
  and damage to the cap during centrifugation. Centrifuge at 735×g for 2
  min. Make sure that another group's sample counterbalances the
  microcentrifuge.
\item
  Remove the spin column from the microcentrifuge tube and discard the
  column. Cap the microcentrifuge tube, which now contains the purified
  PCR product. Store at 4°C for up to 2 weeks or at --20°C long term.
\item
  Add 1 µl of 6× loading dye and stain into 5 µl of the purified sample.
\item
  Electrophorese 5 µl of this mix next to 5 µl of the unpurified sample
  on a 1\% agarose gel to visualize differences between the original and
  the purified sample.
\item
  Check if the agarose gel has solidified. If so, proceed to the next
  step or wait until the gel is ready.
\item
  Pour electrophoresis running buffer into the chamber until it just
  covers the gel by 1--2 mm.
\item
  Load 10 µl of the 500 bp molecular weight ruler into the first well.
\item
  Load 20 µl from each microcentrifuge tube containing an initial PCR
  with added loading dye. Load 5 µl from each tube containing a nested
  PCR with added loading dye into the wells of the gel according to your
  plan.
\item
  Connect your electrophoresis chamber to the power supply and turn on
  the power. Run the gel at 100 V for 30 min.
\item
  While the gel is running, start setting up the ligation reactions.
\end{enumerate}

\section{Ligation}\label{ligation}

In molecular biology,
\href{https://en.wikipedia.org/wiki/Ligation_(molecular_biology)}{ligation}
is the joining of two nucleic acid fragments through the action of an
enzyme. It is an essential laboratory procedure in the molecular cloning
of DNA whereby DNA fragments are joined together to create recombinant
DNA molecules, such as when a foreign DNA fragment is inserted into a
plasmid. The ends of DNA fragments are joined together by the formation
of phosphodiester bonds between the 3'-hydroxyl of one DNA terminus with
the 5'-phosphoryl of another.

\section{Experimental Procedures}\label{experimental-procedures-13}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Label a microcentrifuge tube with your initials and ``l'' for
  ``ligation.''
\item
  Briefly centrifuge the stock tubes containing the 2× ligation reaction
  buffer and proofreading polymerase in a microfuge to force contents to
  bottom of tubes.
\item
  Add 5.0 µl 2× ligation reaction buffer to the labeled microcentrifuge
  tube.
\item
  Add 1.0 µl purified PCR product
\item
  Add 2.5 µl sterile water
\item
  Add 0.5 µl proofreading polymerase
\item
  Close the cap and mix well. Centrifuge briefly in a microfuge to
  collect the contents at the bottom of the tube.
\item
  Incubate the tube at 70°C for 5 min. 70°C is the optimal temperature
  for the proofreading polymerase to blunt the PCR fragment.
\item
  Cool the tube on ice for 2 min. This recondenses water vapor to
  maintain the reaction volume. Centrifuge the tube briefly to collect
  the contents at the bottom of the tube. Maintain the tube at room
  temperature.
\item
  Add 0.5 µl T4 DNA ligase.
\item
  Add 0.5 µl pJet1.2 blunted vector.
\item
  Close the cap and mix well.
\item
  Centrifuge briefly in a microcentrifuge to collect the contents at the
  bottom of the tube.
\item
  Incubate the tube at room temperature for 5--10 min.
\item
  Pipet 5 µl of the ligation reaction into a fresh microcentrifuge tube
  labeled with your initials and ``t'' for ``transformation'' and store
  it on ice until needed for the transformation.
\item
  Take the gel out of the electrophoresis box to visualize the DNA bands
  and acquire an image of your gel.
\end{enumerate}

\section{Transformation}\label{transformation}

In molecular biology,
\href{https://en.wikipedia.org/wiki/Transformation_(genetics)}{transformation}
is the genetic alteration of a cell resulting from the direct uptake and
incorporation of exogenous genetic material from its surroundings
through the cell membrane(s). For transformation to take place, the
recipient bacteria must be in a state of competence, which might occur
in nature as a time-limited response to environmental conditions such as
starvation and cell density, and may also be induced in a laboratory.

Transformation is one of three processes for horizontal gene transfer,
in which exogenous genetic material passes from one bacterium to
another, the other two being conjugation (transfer of genetic material
between two bacterial cells in direct contact) and transduction
(injection of foreign DNA by a bacteriophage virus into the host
bacterium). In transformation, the genetic material passes through the
intervening medium, and uptake is completely dependent on the recipient
bacterium.

``Transformation'' may also be used to describe the insertion of new
genetic material into nonbacterial cells, including animal and plant
cells; however, because ``transformation'' has a special meaning in
relation to animal cells, indicating progression to a cancerous state,
the process is usually called ``transfection''. Transformation in
bacteria was first demonstrated in 1928 by the British bacteriologist
\href{https://en.wikipedia.org/wiki/Frederick_Griffith}{Frederick
Griffith}. Griffith was interested in determining whether injections of
heat-killed bacteria could be used to vaccinate mice against pneumonia.
However, he discovered that a non-virulent strain of \emph{Streptococcus
pneumonia} could be made virulent after being exposed to heat-killed
virulent strains. Griffith hypothesized that some ``transforming
principle'' from the heat-killed strain was responsible for making the
harmless strain virulent. In 1944 this ``transforming principle'' was
identified as being genetic by
\href{https://en.wikipedia.org/wiki/Oswald_Avery}{Oswald Avery},
\href{https://en.wikipedia.org/wiki/Colin_Munro_MacLeod}{Colin MacLeod},
and \href{https://en.wikipedia.org/wiki/Maclyn_McCarty}{Maclyn McCarty}.
They isolated DNA from a virulent strain of S. pneumoniae and using just
this DNA were able to make a harmless strain virulent. They called this
uptake and incorporation of DNA by bacteria ``transformation''. The
results of Avery et al.'s experiments were at first skeptically received
by the scientific community and it was not until the development of
genetic markers and the discovery of other methods of genetic transfer
(conjugation in 1947 and transduction in 1953) by
\href{https://en.wikipedia.org/wiki/Joshua_Lederberg}{Joshua Lederberg}
that Avery's experiments were accepted.

\section{Experimental Procedures}\label{experimental-procedures-14}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Add 1.5 ml C-growth medium into a 15 ml culture tube. Label tube with
  your initials and warm it at 37°C for at least 10 min.
\item
  Label 2 LB amp IPTG agar plates with your initials (on the bottom of
  the plate, not the lid). Also label one of the plates ``pGAP'' for the
  control plasmid and the other for your ligation (pJET + your plant
  name).
\item
  Pipet 150 µl of fresh \emph{E. coli} starter culture (inoculated one
  day prior) into the prewarmed C-growth medium and place in a 37°C
  incubator or water bath for 20--40 min shaking at 275 rpm.
\item
  Label a 1.5 microcentrifuge tube with your initials and ``competent
  cells.''
\item
  Prepare transformation buffer by combining 250 µl of transformation
  reagent A and 250 µl of transformation reagent B into a tube labeled
  ``TF buffer'' and mix thoroughly with a vortex mixer. Keep on ice
  until use.
\item
  After bacteria have grown in C-growth medium for 20--40 min at 37°C
  with shaking, transfer the entire culture to the tube labeled
  ``competent cells'' by decanting. Do not put the actively growing cell
  culture on ice at this step.
\item
  Centrifuge the bacterial culture in a microcentrifuge at top speed for
  1 min. Make sure that the microcentrifuge is balanced. Immediately put
  the pelleted bacterial culture on ice.
\item
  Locate the pellet of bacteria at the bottom of the tube. Remove the
  culture supernatant, avoiding the pellet, using a 1,000 µl pipet. Keep
  the cells on ice.
\item
  Pipet 300 µl of ice-cold transformation buffer into the
  microcentrifuge tube containing the bacterial pellet. Resuspend the
  pellet by gently pipetting up and down in the solution above the
  pellet with a 1,000 µl pipet, and gradually wear away the pellet from
  the bottom of the tube. Make sure that the bacteria are fully
  resuspended, with no clumps. Avoid removing the cells from the ice
  bucket for more than a few seconds.
\item
  Incubate the resuspended bacteria on ice for 5 min.
\item
  Centrifuge the bacteria in a microcentrifuge for 1 min, then place
  back in ice bucket immediately.
\end{enumerate}

\BeginKnitrBlock{rmdimportant}
Ensure that the bacteria are on ice immediately prior to and immediately
following centrifugation. If the centrifuge is not close to the lab
bench, take the entire ice bucket to the microcentrifuge so that the
bacteria are only out of the ice bucket for 1 minute. Use a refrigerated
microcentrifuge, if available.
\EndKnitrBlock{rmdimportant}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{11}
\tightlist
\item
  Remove the supernatant from the pellet using a 1,000 μl pipet.
\end{enumerate}

\BeginKnitrBlock{rmdimportant}
After this step, it is very important to keep the bacteria on ice as
much as possible during this procedure. Transformation efficiency will
be severely compromised if the cells warm up. It is very important to
treat the bacteria extremely gently during this procedure --- the
bacteria are very fragile and your transformation efficiency will be
compromised unless you are very gentle.
\EndKnitrBlock{rmdimportant}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{7}
\tightlist
\item
  Locate the pellet of bacteria at the bottom of the tube. Remove the
  culture
\item
  Pipet 120 μl of ice-cold transformation buffer onto the pellet and
  resuspend by gently pipetting up and down with a 200 μl pipet. Be sure
  that bacteria are fully resuspended with no clumps. Avoid removing the
  cells from the ice bucket for more than a few seconds.
\item
  Incubate the resuspended bacteria on ice for 5 min. The cells are now
  competent for transformation.
\item
  Label one microcentrifuge tube with your initials and ``pGAP T'' (for
  pGAP transformation) and another microcentrifuge tube with your
  initials and ``T'' (referred to below as the ``plant T'' tube).
\item
  Pipet 1 μl of control pGAP plasmid into the microcentrifuge tube
  labeled ``pGAP T.''
\item
  Using a fresh tip, pipet 50 μl of competent bacteria directly into the
  ice-cold ``plant T'' tube containing 5 μl of your ligation, and gently
  pipet up and down 2 times to mix.
\item
  Incubate the transformations for 10 min on ice.
\item
  Retrieve the warm LB amp IPTG agar plates from the 37°C incubator.
\item
  Pipet the entire volume of each transformation onto the corresponding
  labeled LB amp IPTG plate and, using an inoculation loop or a sterile
  spreader, very gently spread the bacteria around the plate ---
  remember that the bacteria are still very fragile! Once the plate is
  covered, stop spreading. Do not spread for more than 10 sec.~It is
  vital that the LB amp IPTG plates are warm at this step to ensure
  sufficiently high transformation efficiency. This is the heat shock
  for the transformation. Spreading the plate until it is dry will also
  reduce transformation efficiency.
\item
  Once the volume is absorbed in the agar, cover and place the LB amp
  IPTG plates upside down and incubate them overnight at 37°C.
\end{enumerate}

\chapter{Plasmid Purification And Restriction
Digest}\label{plasmid-purification-and-restriction-digest}

\section{Plasmid Purification}\label{plasmid-purification}

\section{Experimental Procedures}\label{experimental-procedures-15}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Label one capless collection tube, one plasmid mini column, and two
  microcentrifuge tubes for each miniprep culture with your initial and
  ``M'' for ``miniprep''.
\item
  Place each column in the appropriate capless collection tube.
\item
  Transfer 1.5 ml of each miniprep culture into one of the appropriately
  labeled microcentrifuge tubes by pipetting or decanting.
\item
  Centrifuge the microcentrifuge tubes for 1 min at top speed
  (\textgreater{}12,000 × g) to pellet the bacteria. Make sure that the
  microcentrifuge is balanced.
\item
  Locate the bacterial pellet and remove the supernatant from each tube
  using a 1,000 µl pipet; avoid touching or extracting the pellet.
  Dispose of the supernatant in the provided waste bottles in the fume
  hood.
\item
  Add the remaining 1.5 ml of the appropriate miniprep culture to the
  correct microcentrifuge tube containing the bacterial pellet,
  centrifuge for 1 min and remove supernatant.
\item
  Add 250 µl of resuspension solution to each tube. Resuspend bacterial
  pellet by pipetting up and down or vortexing. Use a fresh tip each
  time. Ensure that no clumps of bacteria remain.
\item
  Pipet 250 µl of lysis solution into each tube and mix by gently
  inverting 6--8 times. Do not pipet or vortex this lysate or you risk
  shearing (fragmenting) the bacterial gDNA, which could contaminate
  your plasmid preparation.
\item
  Wait for 5 min, then pipet 350 µl of neutralization solution into each
  tube and mix by gently inverting 6--8 times. Do not pipet or vortex
  this lysate. A white precipitate should form.
\item
  Centrifuge the tubes for 5 min at top speed. Make sure the
  microcentrifuge is balanced.
\item
  Pipet supernatant from the centrifuged tubes onto the appropriately
  labeled column. Avoid transferring any precipitate. If necessary,
  recentrifuge the microcentrifuge tubes. Discard the empty
  microcentrifuge tubes.
\item
  Centrifuge the columns in the capless collection tubes for 1 min at
  top speed.
\item
  Discard the flowthough from the collection tube and place the column
  back in the collection tube.
\item
  Pipet 750 µl of wash solution onto each column (ensure that the wash
  solution has had ethanol added to it prior to use).
\item
  Centrifuge columns in the capless collection tubes for 1 min at top
  speed.
\item
  Discard the flowthough from the collection tube.
\item
  Place the columns back into collection tubes and centrifuge for an
  additional 1 min to dry the column.
\item
  Transfer contents of each column to the appropriately labeled capped
  ``miniprep DNA'' microcentrifuge tube and pipet 100 µl of elution
  solution onto the column.
\item
  Let the elution solution be absorbed into the column for 2 min.
\item
  Place the column in the microcentrifuge tube into the centrifuge. It
  is best to orient the cap of the microcentrifuge tube downward, toward
  the center of the rotor, to minimize friction and damage to the cap
  during centrifugation.
\item
  Centrifuge the columns for 2 min.
\item
  Discard the columns and cap the tubes containing the eluted sample.
\item
  Proceed to setting up the restriction digest reactions.
\end{enumerate}

\section{Restriction Digest}\label{restriction-digest}

\section{Experimental Procedures}\label{experimental-procedures-16}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Label one microcentrifuge tube with your initials and ``master mix.''
\item
  Prepare a 2× master mix for BgI II restriction digestion reactions
  according to Table \ref{tab:digest} using stock reagents from the
  common workstation. Use a fresh tip for each reagent.
\item
  Label a microcentrifuge tube for each plasmid miniprep.
\item
  Prepare digestion reactions by adding 10 µl of the BgI II master mix
  and 10 µl of each plasmid DNA in the labeled microcentrifuge tubes.
\item
  Mix tube contents and spin briefly in a microcentrifuge to collect the
  contents at the bottom of the tube.
\item
  Incubate reactions at 37°C for 1 hr. Set timer to sound an alarm after
  1 hour.
\item
  While the restriction digest is incubating, prepare an agarose gel.
\item
  Get the Erlenmeyer flask containing 1 g of agarose powder
\item
  Add 100 ml of 1× TAE (Tris base, acetic acid, EDTA) running buffer.
\item
  Add 10 µl of SybrGreen™ dye (10,000× stock solution).
\item
  Heat in the microwave at full power for 1 minute.
\item
  Swirl to make sure that all powder has dissolved, and the solution is
  clear.
\item
  Add the comb into the comb slot.
\item
  Pour the solution onto the gel tray in the gel box.
\item
  The gel will need about 30 minutes to solidify.
\item
  When the timer indicates that the restriction digest has been
  incubating for 1 hour, combine 5 µl of miniprep DNA with 15 µl of
  sterile water.
\item
  Briefly centrifuge samples to force the contents to the bottom of the
  tube.
\item
  Add 3.5 µl of 6× loading dye and stain to each of the digested and
  undigested samples.
\item
  Load 20 µl of each sample and 10 µl of the 500 bp molecular weight
  ruler according to your plan on the 1\% agarose gel.
\item
  Connect the electrophoresis chamber to the power supply and turn on
  the power. Run the gel at 100 V for 30 min.
\item
  Visualize your bands and acquire an image of your gel. Paste the image
  into your lab notebook and label it accordingly.
\end{enumerate}

\begin{longtable}[]{@{}ll@{}}
\caption{\label{tab:digest} Composition of restriction digest master
mix.}\tabularnewline
\toprule
Reagent & Volume\tabularnewline
\midrule
\endfirsthead
\toprule
Reagent & Volume\tabularnewline
\midrule
\endhead
10× BgI II reaction buffer & 2 µl\tabularnewline
Sterile water & 7 µl\tabularnewline
BgI II enzyme & 1 µl\tabularnewline
Total & 10 µl\tabularnewline
\bottomrule
\end{longtable}

\chapter{DNA sequencing}\label{dna-sequencing}

DNA sequencing is the process of determining the precise order of
nucleotides within a DNA molecule. It includes any method or technology
that is used to determine the order of the four bases---adenine,
guanine, cytosine, and thymine---in a strand of DNA. The advent of rapid
DNA sequencing methods has greatly accelerated biological and medical
research and discovery.

Knowledge of DNA sequences has become indispensable for basic biological
research, and in numerous applied fields such as medical diagnosis,
biotechnology, forensic biology, virology and biological systematics.
The rapid speed of sequencing attained with modern DNA sequencing
technology has been instrumental in the sequencing of complete DNA
sequences, or genomes of numerous types and species of life, including
the human genome and other complete DNA sequences of many animal, plant,
and microbial species.

The first DNA sequences were obtained in the early 1970s by academic
researchers using laborious methods based on two-dimensional
chromatography.
\href{https://en.wikipedia.org/wiki/Frederick_Sanger}{Frederick Sanger}
published a method for ``DNA sequencing with chain-terminating
inhibitors'' in 1977.
\href{https://en.wikipedia.org/wiki/Walter_Gilbert}{Walter Gilbert} and
\href{https://en.wikipedia.org/wiki/Allan_Maxam}{Allan Maxam} at Harvard
also developed sequencing methods, including one for ``DNA sequencing by
chemical degradation''. The chain-termination method developed by
Frederick Sanger and coworkers in 1977 soon became the method of choice,
owing to its relative ease and reliability. When invented, the
chain-terminator method used fewer toxic chemicals and lower amounts of
radioactivity than the Maxam and Gilbert method. Because of its
comparative ease, the Sanger method was soon automated and was the
method used in the first generation of DNA sequencers.

Sanger sequencing is the method which prevailed from the 1980s until the
mid-2000s. Over that period, great advances were made in the technique,
such as fluorescent labelling, capillary electrophoresis, and general
automation. Advancements in sequencing were aided by the concurrent
development of recombinant DNA technology, allowing DNA samples to be
isolated from sources other than viruses.Following the development of
fluorescence-based sequencing methods with a DNA sequencer, DNA
sequencing has become easier and orders of magnitude faster.
High-throughput (formerly ``next-generation'') sequencing applies to
genome sequencing, genome resequencing, transcriptome profiling
(RNA-Seq), DNA-protein interactions (ChIP-sequencing), and epigenome
characterization.

The high demand for low-cost sequencing has driven the development of
high-throughput sequencing technologies that parallelize the sequencing
process, producing thousands or millions of sequences concurrently.
High-throughput sequencing technologies are intended to lower the cost
of DNA sequencing beyond what is possible with standard dye-terminator
methods. In ultra-high-throughput sequencing as many as 500,000
sequencing-by-synthesis operations may be run in parallel.

\section{Experimental Procedures}\label{experimental-procedures-17}

Each team will be assigned a group of wells on a 96-well plate for
setting up the sequencing reactions. The rows and columns of the 96-well
plate are by letters and numbers. The top left well is designated A1 and
the bottom right well is H12.

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Choose two of your plasmid minipreps to sequence.
\item
  Plan your experiment. You will prepare four sequencing samples for
  each of your two plasmid minipreps. You will combine each plasmid
  miniprep with each of the four different sequencing primers --- two
  forward primers and two reverse primers --- to ensure complete
  coverage of the insert. pJET SEQ F and pJET SEQ R anneal to the
  pJET1.2 vector, while GAP SEQ F and GAP SEQ R are degenerate primers
  homologous to internal GAPC sequences. Enter in Table \ref{tab:seq}
  below which plasmid was combined with which primer and added into each
  well. When you name your sequences (sequence wells), make sure you use
  the same names when submitting samples to the sequencing facility.
\item
  In your microcentrifuge tubes, combine 10 µl of the miniprep DNA with
  1 µl of sequencing primer. Pipet up and down to mix.
\item
  Pipet 10 µl of the plasmid/primer mixtures into the assigned wells of
  the 96-well plate. Write down the barcode number from the 96-well
  plate.
\item
  Once the entire class has added samples to the plate, seal the plate
  using the sealing film provided. Ensure a secure seal by rubbing
  extensively over the top of the plate with a gloved finger. It is
  essential to seal the plate completely so that the samples are not
  lost during transit.
\item
  Go to www.operon.com/bio-rad and fill out the sequencing request form.
\item
  The laboratory technicians will mail the plate to the sequencing
  facility.
\end{enumerate}

\begin{longtable}[]{@{}lll@{}}
\caption{\label{tab:seq} DNA Sequencing Reactions}\tabularnewline
\toprule
Well Identifier & Plasmid Name & Sequencing Primers and
Color\tabularnewline
\midrule
\endfirsthead
\toprule
Well Identifier & Plasmid Name & Sequencing Primers and
Color\tabularnewline
\midrule
\endhead
& &\tabularnewline
& &\tabularnewline
& &\tabularnewline
& &\tabularnewline
& &\tabularnewline
& &\tabularnewline
& &\tabularnewline
& &\tabularnewline
\bottomrule
\end{longtable}

\chapter{Bioinformatics}\label{bioinformatics}

Bioinformatics is an interdisciplinary field that develops methods and
software tools for understanding biological data. As an
interdisciplinary field of science, bioinformatics combines biology,
computer science, mathematics and statistics to analyze and interpret
biological data. Bioinformatics has been used for \emph{in silico}
analyses of biological queries using mathematical and statistical
techniques.

Bioinformatics is both an umbrella term for the body of biological
studies that use computer programming as part of their methodology, as
well as a reference to specific analysis ``pipelines'' that are
repeatedly used, particularly in the field of genomics. Common uses of
bioinformatics include the identification of candidate genes and single
nucleotide polymorphisms (SNPs). Often, such identification is made with
the aim of better understanding the genetic basis of disease, unique
adaptations, desirable properties (esp. in agricultural species), or
differences between populations. In a less formal way, bioinformatics
also tries to understand the organisational principles within nucleic
acid and protein sequences, called proteomics.


\end{document}
