<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Laboratory Manual For SCI200 Genetics at Roxbury Community College</title>
  <meta name="description" content="This laboratory manual will accompany you during your laboratory for Genetics.">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Laboratory Manual For SCI200 Genetics at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="cover.jpg" />
  <meta property="og:description" content="This laboratory manual will accompany you during your laboratory for Genetics." />
  <meta name="github-repo" content="nikolaussuchert/genetics" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Laboratory Manual For SCI200 Genetics at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This laboratory manual will accompany you during your laboratory for Genetics." />
  <meta name="twitter:image" content="cover.jpg" />

<meta name="author" content="Nikolaus Sucher">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="mitosis-and-meiosis.html">
<link rel="next" href="mendelian-genetics-mendel-in-his-own-words.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">RCC Genetics Laboratory Manual</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="course-description.html"><a href="course-description.html"><i class="fa fa-check"></i><b>1</b> Course Description</a><ul>
<li class="chapter" data-level="1.1" data-path="course-description.html"><a href="course-description.html#instructional-objectives"><i class="fa fa-check"></i><b>1.1</b> Instructional Objectives:</a></li>
<li class="chapter" data-level="1.2" data-path="course-description.html"><a href="course-description.html#student-learning-outcomes"><i class="fa fa-check"></i><b>1.2</b> Student Learning Outcomes:</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="how-to-do-well-in-this-class.html"><a href="how-to-do-well-in-this-class.html"><i class="fa fa-check"></i><b>2</b> How to do well in this class</a><ul>
<li class="chapter" data-level="" data-path="how-to-do-well-in-this-class.html"><a href="how-to-do-well-in-this-class.html#before-the-lab"><i class="fa fa-check"></i>Before the lab</a></li>
<li class="chapter" data-level="" data-path="how-to-do-well-in-this-class.html"><a href="how-to-do-well-in-this-class.html#during-the-lab"><i class="fa fa-check"></i>During the lab</a></li>
<li class="chapter" data-level="" data-path="how-to-do-well-in-this-class.html"><a href="how-to-do-well-in-this-class.html#after-the-lab"><i class="fa fa-check"></i>After the lab</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="lab-safety.html"><a href="lab-safety.html"><i class="fa fa-check"></i><b>3</b> Lab Safety</a><ul>
<li class="chapter" data-level="3.1" data-path="lab-safety.html"><a href="lab-safety.html#general-rules"><i class="fa fa-check"></i><b>3.1</b> General rules</a></li>
<li class="chapter" data-level="3.2" data-path="lab-safety.html"><a href="lab-safety.html#class-dissections"><i class="fa fa-check"></i><b>3.2</b> Class dissections</a></li>
<li class="chapter" data-level="3.3" data-path="lab-safety.html"><a href="lab-safety.html#clothing"><i class="fa fa-check"></i><b>3.3</b> Clothing</a></li>
<li class="chapter" data-level="3.4" data-path="lab-safety.html"><a href="lab-safety.html#handling-chemicals"><i class="fa fa-check"></i><b>3.4</b> Handling chemicals</a></li>
<li class="chapter" data-level="3.5" data-path="lab-safety.html"><a href="lab-safety.html#glassware"><i class="fa fa-check"></i><b>3.5</b> Glassware</a></li>
<li class="chapter" data-level="3.6" data-path="lab-safety.html"><a href="lab-safety.html#heating-substances"><i class="fa fa-check"></i><b>3.6</b> Heating substances</a></li>
<li class="chapter" data-level="3.7" data-path="lab-safety.html"><a href="lab-safety.html#handling-microbiology-materials"><i class="fa fa-check"></i><b>3.7</b> Handling microbiology materials</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="classical-genetics.html"><a href="classical-genetics.html"><i class="fa fa-check"></i>Classical Genetics</a><ul>
<li class="chapter" data-level="3.8" data-path="classical-genetics.html"><a href="classical-genetics.html#review-questions"><i class="fa fa-check"></i><b>3.8</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="mitosis-and-meiosis.html"><a href="mitosis-and-meiosis.html"><i class="fa fa-check"></i><b>4</b> Mitosis and Meiosis</a><ul>
<li class="chapter" data-level="4.1" data-path="mitosis-and-meiosis.html"><a href="mitosis-and-meiosis.html#view-prepared-slides"><i class="fa fa-check"></i><b>4.1</b> View Prepared Slides</a></li>
<li class="chapter" data-level="4.2" data-path="mitosis-and-meiosis.html"><a href="mitosis-and-meiosis.html#preparing-an-onion-root-tip-squash"><i class="fa fa-check"></i><b>4.2</b> Preparing an Onion root tip squash</a><ul>
<li class="chapter" data-level="4.2.1" data-path="mitosis-and-meiosis.html"><a href="mitosis-and-meiosis.html#experimental-procedures"><i class="fa fa-check"></i><b>4.2.1</b> Experimental procedures</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="mitosis-and-meiosis.html"><a href="mitosis-and-meiosis.html#review-questions-1"><i class="fa fa-check"></i><b>4.3</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html"><i class="fa fa-check"></i><b>5</b> Chromosomes And Karyotypes</a><ul>
<li class="chapter" data-level="5.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#chromosomes"><i class="fa fa-check"></i><b>5.1</b> Chromosomes</a></li>
<li class="chapter" data-level="5.2" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#karyotypes"><i class="fa fa-check"></i><b>5.2</b> Karyotypes</a></li>
<li class="chapter" data-level="5.3" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#observations-on-karyotypes"><i class="fa fa-check"></i><b>5.3</b> Observations on karyotypes</a><ul>
<li class="chapter" data-level="5.3.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#staining"><i class="fa fa-check"></i><b>5.3.1</b> Staining</a></li>
<li class="chapter" data-level="5.3.2" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#observations"><i class="fa fa-check"></i><b>5.3.2</b> Observations</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#human-karyotype"><i class="fa fa-check"></i><b>5.4</b> Human karyotype</a><ul>
<li class="chapter" data-level="5.4.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#ploidy"><i class="fa fa-check"></i><b>5.4.1</b> Ploidy</a></li>
<li class="chapter" data-level="5.4.2" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#aneuploidy"><i class="fa fa-check"></i><b>5.4.2</b> Aneuploidy</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#chromosome-abnormalities"><i class="fa fa-check"></i><b>5.5</b> Chromosome abnormalities</a><ul>
<li class="chapter" data-level="5.5.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#chromosomal-abnormalities-that-lead-to-disease-in-humans-include"><i class="fa fa-check"></i><b>5.5.1</b> Chromosomal abnormalities that lead to disease in humans include</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#chromosome-banding"><i class="fa fa-check"></i><b>5.6</b> Chromosome banding</a><ul>
<li class="chapter" data-level="5.6.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#types-of-banding"><i class="fa fa-check"></i><b>5.6.1</b> Types of banding</a></li>
<li class="chapter" data-level="5.6.2" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#classic-karyotype-cytogenetics"><i class="fa fa-check"></i><b>5.6.2</b> Classic karyotype cytogenetics</a></li>
<li class="chapter" data-level="5.6.3" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#multicolor-fish-mfish-and-spectral-karyotype-sky-technique"><i class="fa fa-check"></i><b>5.6.3</b> Multicolor FISH (mFISH) and spectral karyotype (SKY technique)</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#determine-the-karyotype"><i class="fa fa-check"></i><b>5.7</b> Determine the karyotype</a><ul>
<li class="chapter" data-level="5.7.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#experimental-procedures-1"><i class="fa fa-check"></i><b>5.7.1</b> Experimental procedures</a></li>
</ul></li>
<li class="chapter" data-level="5.8" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#review-questions-2"><i class="fa fa-check"></i><b>5.8</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="mendelian-genetics-mendel-in-his-own-words.html"><a href="mendelian-genetics-mendel-in-his-own-words.html"><i class="fa fa-check"></i><b>6</b> Mendelian Genetics: Mendel in His Own Words</a><ul>
<li class="chapter" data-level="6.1" data-path="mendelian-genetics-mendel-in-his-own-words.html"><a href="mendelian-genetics-mendel-in-his-own-words.html#experimental-procedures-2"><i class="fa fa-check"></i><b>6.1</b> Experimental Procedures</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html"><i class="fa fa-check"></i><b>7</b> Mendelian Genetics: A Practical Example</a><ul>
<li class="chapter" data-level="7.1" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#monohybrid-cross-experiment-1"><i class="fa fa-check"></i><b>7.1</b> Monohybrid Cross (Experiment 1)</a></li>
<li class="chapter" data-level="7.2" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#punnett-square"><i class="fa fa-check"></i><b>7.2</b> Punnett Square</a><ul>
<li class="chapter" data-level="7.2.1" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#experimental-procedures-3"><i class="fa fa-check"></i><b>7.2.1</b> Experimental Procedures</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#dihybrid-cross-experiment-2"><i class="fa fa-check"></i><b>7.3</b> Dihybrid Cross (Experiment 2)</a><ul>
<li class="chapter" data-level="7.3.1" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#experimental-procedures-4"><i class="fa fa-check"></i><b>7.3.1</b> Experimental Procedures</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#pearsons-chi-squared-chi2-and-goodness-of-fit-test"><i class="fa fa-check"></i><b>7.4</b> Pearson’s Chi-Squared (<span class="math inline">\(\chi^2\)</span>) And Goodness Of Fit Test</a><ul>
<li class="chapter" data-level="7.4.1" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#calculating-chi2-using-the-monohybrid-cross-data"><i class="fa fa-check"></i><b>7.4.1</b> Calculating <span class="math inline">\(\chi^2\)</span> Using The Monohybrid Cross Data</a></li>
<li class="chapter" data-level="7.4.2" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#testing-goodness-of-fit"><i class="fa fa-check"></i><b>7.4.2</b> Testing Goodness Of Fit</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#a-look-at-mendels-monohybrid-cross-data"><i class="fa fa-check"></i><b>7.5</b> A Look At Mendel’s Monohybrid Cross Data</a></li>
<li class="chapter" data-level="7.6" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#review-questions-3"><i class="fa fa-check"></i><b>7.6</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="molecular-genetics.html"><a href="molecular-genetics.html"><i class="fa fa-check"></i>Molecular Genetics</a></li>
<li class="chapter" data-level="8" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html"><i class="fa fa-check"></i><b>8</b> Molecular Laboratory Skills</a><ul>
<li class="chapter" data-level="8.1" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#exercise-1-using-graduated-serological-macropipettes"><i class="fa fa-check"></i><b>8.1</b> Exercise 1: using graduated (serological) macropipettes</a><ul>
<li class="chapter" data-level="8.1.1" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#experimental-procedures-5"><i class="fa fa-check"></i><b>8.1.1</b> Experimental procedures</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#exercise-2-using-air-displacement-micropipettes"><i class="fa fa-check"></i><b>8.2</b> Exercise 2: using air displacement micropipettes</a><ul>
<li class="chapter" data-level="8.2.1" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#experimental-procedures-6"><i class="fa fa-check"></i><b>8.2.1</b> Experimental procedures</a></li>
</ul></li>
<li class="chapter" data-level="8.3" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#exercise-3-using-a-microcentrifuge"><i class="fa fa-check"></i><b>8.3</b> Exercise 3: using a microcentrifuge</a><ul>
<li class="chapter" data-level="8.3.1" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#experimental-procedures-7"><i class="fa fa-check"></i><b>8.3.1</b> Experimental procedures</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#review-questions-4"><i class="fa fa-check"></i><b>8.4</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html"><i class="fa fa-check"></i><b>9</b> DNA Extraction and Polymerase Chain Reaction</a><ul>
<li class="chapter" data-level="9.1" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#dna"><i class="fa fa-check"></i><b>9.1</b> DNA</a></li>
<li class="chapter" data-level="9.2" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#polymerase-chain-reaction"><i class="fa fa-check"></i><b>9.2</b> Polymerase chain reaction</a></li>
<li class="chapter" data-level="9.3" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#dna-extraction"><i class="fa fa-check"></i><b>9.3</b> DNA Extraction</a></li>
<li class="chapter" data-level="9.4" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#experimental-procedures-8"><i class="fa fa-check"></i><b>9.4</b> Experimental Procedures</a></li>
<li class="chapter" data-level="9.5" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#gapdh-polymerase-chain-reaction"><i class="fa fa-check"></i><b>9.5</b> GAPDH Polymerase Chain Reaction</a></li>
<li class="chapter" data-level="9.6" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#experimental-procedures-9"><i class="fa fa-check"></i><b>9.6</b> Experimental Procedures</a></li>
<li class="chapter" data-level="9.7" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#review-questions-5"><i class="fa fa-check"></i><b>9.7</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html"><i class="fa fa-check"></i><b>10</b> Exonuclease I Digestion And Nested PCR</a><ul>
<li class="chapter" data-level="10.1" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html#exonuclease-i-digestion"><i class="fa fa-check"></i><b>10.1</b> Exonuclease I Digestion</a></li>
<li class="chapter" data-level="10.2" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html#experimental-procedures-10"><i class="fa fa-check"></i><b>10.2</b> Experimental Procedures</a></li>
<li class="chapter" data-level="10.3" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html#nested-polymerase-chain-reaction"><i class="fa fa-check"></i><b>10.3</b> Nested polymerase chain reaction</a></li>
<li class="chapter" data-level="10.4" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html#experimental-procedures-11"><i class="fa fa-check"></i><b>10.4</b> Experimental Procedures</a></li>
<li class="chapter" data-level="10.5" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html#review-questions-6"><i class="fa fa-check"></i><b>10.5</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html"><i class="fa fa-check"></i><b>11</b> Agarose Gel Electrophoresis, Ligation and Transformation</a><ul>
<li class="chapter" data-level="11.1" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#agarose-gel-electrophoresis"><i class="fa fa-check"></i><b>11.1</b> Agarose gel electrophoresis</a></li>
<li class="chapter" data-level="11.2" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#experimental-procedures-12"><i class="fa fa-check"></i><b>11.2</b> Experimental Procedures</a></li>
<li class="chapter" data-level="11.3" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#pcr-product-purification"><i class="fa fa-check"></i><b>11.3</b> PCR Product Purification</a></li>
<li class="chapter" data-level="11.4" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#experimental-procedures-13"><i class="fa fa-check"></i><b>11.4</b> Experimental Procedures</a></li>
<li class="chapter" data-level="11.5" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#ligation"><i class="fa fa-check"></i><b>11.5</b> Ligation</a></li>
<li class="chapter" data-level="11.6" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#experimental-procedures-14"><i class="fa fa-check"></i><b>11.6</b> Experimental Procedures</a></li>
<li class="chapter" data-level="11.7" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#transformation"><i class="fa fa-check"></i><b>11.7</b> Transformation</a></li>
<li class="chapter" data-level="11.8" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#experimental-procedures-15"><i class="fa fa-check"></i><b>11.8</b> Experimental Procedures</a></li>
<li class="chapter" data-level="11.9" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#review-questions-7"><i class="fa fa-check"></i><b>11.9</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="dna-sequencing.html"><a href="dna-sequencing.html"><i class="fa fa-check"></i><b>12</b> DNA sequencing</a><ul>
<li class="chapter" data-level="12.1" data-path="dna-sequencing.html"><a href="dna-sequencing.html#experimental-procedures-16"><i class="fa fa-check"></i><b>12.1</b> Experimental Procedures</a></li>
<li class="chapter" data-level="12.2" data-path="dna-sequencing.html"><a href="dna-sequencing.html#review-questions-8"><i class="fa fa-check"></i><b>12.2</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="dna-sequence-analysis.html"><a href="dna-sequence-analysis.html"><i class="fa fa-check"></i><b>13</b> DNA sequence analysis</a><ul>
<li class="chapter" data-level="13.1" data-path="dna-sequence-analysis.html"><a href="dna-sequence-analysis.html#experimental-procedures-17"><i class="fa fa-check"></i><b>13.1</b> Experimental procedures</a></li>
<li class="chapter" data-level="13.2" data-path="dna-sequence-analysis.html"><a href="dna-sequence-analysis.html#blast-search"><i class="fa fa-check"></i><b>13.2</b> BLAST search</a></li>
<li class="chapter" data-level="13.3" data-path="dna-sequence-analysis.html"><a href="dna-sequence-analysis.html#experimental-procedures-18"><i class="fa fa-check"></i><b>13.3</b> Experimental procedures</a></li>
<li class="chapter" data-level="13.4" data-path="dna-sequence-analysis.html"><a href="dna-sequence-analysis.html#review-questions-9"><i class="fa fa-check"></i><b>13.4</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="the-human-genome.html"><a href="the-human-genome.html"><i class="fa fa-check"></i><b>14</b> The Human Genome</a><ul>
<li class="chapter" data-level="14.1" data-path="the-human-genome.html"><a href="the-human-genome.html#the-human-genome-project"><i class="fa fa-check"></i><b>14.1</b> The Human Genome Project</a></li>
<li class="chapter" data-level="14.2" data-path="the-human-genome.html"><a href="the-human-genome.html#public-private-competition-around-the-human-genome-project"><i class="fa fa-check"></i><b>14.2</b> Public-Private Competition Around The Human Genome Project</a></li>
<li class="chapter" data-level="14.3" data-path="the-human-genome.html"><a href="the-human-genome.html#genome-annotation"><i class="fa fa-check"></i><b>14.3</b> Genome Annotation</a></li>
<li class="chapter" data-level="14.4" data-path="the-human-genome.html"><a href="the-human-genome.html#international-hapmap-project"><i class="fa fa-check"></i><b>14.4</b> International HapMap Project</a></li>
<li class="chapter" data-level="14.5" data-path="the-human-genome.html"><a href="the-human-genome.html#the-single-nucleotide-polymorphism-database-dbsnp-of-nucleotide-sequence-variation"><i class="fa fa-check"></i><b>14.5</b> The Single Nucleotide Polymorphism Database (dbSNP) of Nucleotide Sequence Variation</a></li>
<li class="chapter" data-level="14.6" data-path="the-human-genome.html"><a href="the-human-genome.html#the-1000-genomes-project"><i class="fa fa-check"></i><b>14.6</b> The 1000 Genomes Project</a></li>
<li class="chapter" data-level="14.7" data-path="the-human-genome.html"><a href="the-human-genome.html#personal-genomics"><i class="fa fa-check"></i><b>14.7</b> Personal genomics</a></li>
<li class="chapter" data-level="14.8" data-path="the-human-genome.html"><a href="the-human-genome.html#viewing-the-human-genome"><i class="fa fa-check"></i><b>14.8</b> Viewing The Human Genome</a><ul>
<li class="chapter" data-level="14.8.1" data-path="the-human-genome.html"><a href="the-human-genome.html#experimental-procedures-19"><i class="fa fa-check"></i><b>14.8.1</b> Experimental Procedures</a></li>
</ul></li>
<li class="chapter" data-level="14.9" data-path="the-human-genome.html"><a href="the-human-genome.html#experimental-procedures-20"><i class="fa fa-check"></i><b>14.9</b> Experimental Procedures</a></li>
<li class="chapter" data-level="14.10" data-path="the-human-genome.html"><a href="the-human-genome.html#review-questions-10"><i class="fa fa-check"></i><b>14.10</b> Review Questions</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Laboratory Manual For SCI200 Genetics at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chromosomes-and-karyotypes" class="section level1">
<h1><span class="header-section-number">5</span> Chromosomes And Karyotypes</h1>
<div id="chromosomes" class="section level2">
<h2><span class="header-section-number">5.1</span> Chromosomes</h2>
<p>A <a href="https://en.wikipedia.org/wiki/Chromosome">chromosome</a> is a DNA molecule with part or all of the genetic material (genome) of an organism. Most eukaryotic chromosomes include packaging proteins which, aided by chaperone proteins, bind to and condense the DNA molecule to prevent it from becoming an unmanageable tangle. The word chromosome comes from the Greek χρῶμα (chroma, “colour”) and σῶμα (soma, “body”), describing their strong staining by particular dyes. The term was coined by <a href="https://en.wikipedia.org/wiki/Heinrich_Wilhelm_Gottfried_von_Waldeyer-Hartz">Heinrich Wilhelm Gottfried von Waldeyer-Hartz</a>, referring to the term chromatin, which was introduced by <a href="https://en.wikipedia.org/wiki/Walther_Flemming">Walther Flemming</a>.</p>
<p>Chromosomes are normally visible under a light microscope only when the cell is undergoing the metaphase of cell division (where all chromosomes are aligned in the center of the cell in their condensed form). Before this happens, every chromosome is copied once (S phase), and the copy is joined to the original by a centromere, resulting either in an X-shaped structure (Figure <a href="chromosomes-and-karyotypes.html#fig:karyo">5.2</a>) if the centromere is located in the middle of the chromosome or a two-arm structure if the centromere is located near one of the ends. The original chromosome and the copy are now called sister chromatids. During metaphase the X-shape structure is called a metaphase chromosome. In this highly condensed form chromosomes are easiest to distinguish and study. In animal cells, chromosomes reach their highest compaction level in anaphase during segregation.</p>

<div class="figure" style="text-align: center"><span id="fig:chromosome"></span>
<img src="figures/karyotyping/Chromosome.svg" alt="Diagram of a replicated and condensed metaphase eukaryotic chromosome. (1) Chromatid – one of the two identical parts of the chromosome after S phase. (2) Centromere – the point where the two chromatids touch. (3) Short (p) arm. (4) Long (q) arm. By derivative work: Tryphon (talk)*Chromosome-upright.png: Original version: Magnus Manske, this version with upright chromosome: User:Dietzel65 [GFDL (http://www.gnu.org/copyleft/fdl.html) or CC-BY-SA-3.0 (http://creativecommons.org/licenses/by-sa/3.0/)], via Wikimedia Commons" width="70%" />
<p class="caption">
Figure 5.1: <a href="https://commons.wikimedia.org/wiki/File:Chromosome.svg">Diagram of a replicated and condensed metaphase eukaryotic chromosome. (1) Chromatid – one of the two identical parts of the chromosome after S phase. (2) Centromere – the point where the two chromatids touch. (3) Short (p) arm. (4) Long (q) arm.</a> By derivative work: Tryphon (talk)*Chromosome-upright.png: Original version: Magnus Manske, this version with upright chromosome: User:Dietzel65 [GFDL (<a href="http://www.gnu.org/copyleft/fdl.html" class="uri">http://www.gnu.org/copyleft/fdl.html</a>) or CC-BY-SA-3.0 (<a href="http://creativecommons.org/licenses/by-sa/3.0/" class="uri">http://creativecommons.org/licenses/by-sa/3.0/</a>)], via Wikimedia Commons
</p>
</div>
<p>Chromosomal recombination during meiosis and subsequent sexual reproduction play a significant role in genetic diversity. If these structures are manipulated incorrectly, through processes known as chromosomal instability and translocation, the cell may undergo mitotic catastrophe and die. Mutations in the cell can allow it to inappropriately evade apoptosis and lead to the progression of cancer.</p>
<p>Some use the term chromosome in a wider sense, to refer to the individualized portions of chromatin in cells, either visible or not under light microscopy. Others use the concept in a narrower sense, to refer to the individualized portions of chromatin during cell division, visible under light microscopy due to high condensation.</p>
<p>The prokaryotes – bacteria and archaea – typically have a single circular chromosome, but many variations exist. The chromosomes of most bacteria can range in size from only 130,000 base pairs in the endosymbiotic bacteria <em>Candidatus Hodgkinia cicadicola</em> to more than 14,000,000 base pairs in the soil-dwelling bacterium <em>Sorangium cellulosum</em>. Spirochaetes of the genus <em>Borrelia</em> are a notable exception to this arrangement, with bacteria such as <em>Borrelia burgdorferi</em>, the cause of Lyme disease, containing a single linear chromosome.</p>
<p>Chromosomes in eukaryotes are composed of chromatin fiber. Chromatin fiber is made of nucleosomes (histone octamers with part of a DNA strand attached to and wrapped around it). Chromatin fibers are packaged by proteins into a condensed structure called chromatin. Chromatin contains the vast majority of DNA and a small amount inherited maternally, can be found in the mitochondria. Chromatin is present in most cells, with a few exceptions, for example, red blood cells.</p>
<p>Chromatin allows the very long DNA molecules to fit into the cell nucleus. During cell division chromatin condenses further to form microscopically visible chromosomes. The structure of chromosomes varies through the cell cycle. During cellular division chromosomes are replicated, divided, and passed successfully to their daughter cells so as to ensure the genetic diversity and survival of their progeny. Chromosomes may exist as either duplicated or unduplicated. Unduplicated chromosomes are single double helixes, whereas duplicated chromosomes contain two identical copies (called chromatids or sister chromatids) joined by a centromere. Eukaryotes possess multiple large linear chromosomes contained in the cells’ nucleus. Each chromosome has one centromere, with one or two arms projecting from the centromere, although, under most circumstances, these arms are not visible as such. In addition, most eukaryotes have a small circular mitochondrial genome, and some eukaryotes may have additional small circular or linear cytoplasmic chromosomes.Eukaryotes (cells with nuclei such as those found in plants, fungi, and animals) possess multiple large linear chromosomes contained in the cell’s nucleus. Each chromosome has one centromere, with one or two arms projecting from the centromere, although, under most circumstances, these arms are not visible as such. In addition, most eukaryotes have a small circular mitochondrial genome, and some eukaryotes may have additional small circular or linear cytoplasmic chromosomes.</p>
<p>In the nuclear chromosomes of eukaryotes, the uncondensed DNA exists in a semi-ordered structure, where it is wrapped around histones (structural proteins), forming a composite material called chromatin.</p>
<p>Chromosomes in humans are divided into two types: autosomes (body chromosome(s)) and allosome (sex chromosome(s)). Certain genetic traits are linked to a person’s sex and are passed on through the sex chromosomes. The autosomes contain the rest of the genetic hereditary information. All act in the same way during cell division. Human cells have 23 pairs of chromosomes (22 pairs of autosomes and one pair of sex chromosomes), giving a total of 46 per cell. In addition to these, human cells have many hundreds of copies of the mitochondrial genome.</p>
</div>
<div id="karyotypes" class="section level2">
<h2><span class="header-section-number">5.2</span> Karyotypes</h2>
<p>A <a href="https://en.wikipedia.org/wiki/Karyotype">karyotype</a> is the number and appearance of chromosomes in the nucleus of a eukaryotic cell. The term is also used for the complete set of chromosomes in a species or in an individual organism and for a test that detects this complement or measures the number. The name was coined by the German anatomist, Heinrich von Waldeyer in 1888. It is New Latin from Ancient Greek κάρυον karyon, “kernel”, “seed”, or “nucleus”, and τύπος typos, “general form”).</p>
<p>Karyotypes describe the chromosome count of an organism and what these chromosomes look like under a light microscope. Attention is paid to their length, the position of the centromeres, banding pattern, any differences between the sex chromosomes, and any other physical characteristics. The preparation and study of karyotypes is part of cytogenetics.</p>
<p>The study of whole sets of chromosomes is sometimes known as karyology. The chromosomes are depicted (by rearranging a photomicrograph) in a standard format known as a karyogram or idiogram: in pairs, ordered by size and position of centromere for chromosomes of the same size as shown in Figure <a href="chromosomes-and-karyotypes.html#fig:karyo">5.2</a>.</p>

<div class="figure" style="text-align: center"><span id="fig:karyo"></span>
<img src="figures/karyotyping/human_male_karyotype.jpg" alt="Karyogram of human male using Giemsa staining" width="70%" />
<p class="caption">
Figure 5.2: <a href="https://commons.wikimedia.org/wiki/File:NHGRI_human_male_karyotype.png">Karyogram of human male using Giemsa staining</a>
</p>
</div>
<p>The basic number of chromosomes in the somatic cells of an individual or a species is called the somatic number and is designated 2n. In the germ-line (the sex cells) the chromosome number is n (humans: n = 23). Thus, in humans 2n = 46.</p>
<p>So, in normal diploid organisms, autosomal chromosomes are present in two copies. There may, or may not, be sex chromosomes. Polyploid cells have multiple copies of chromosomes and haploid cells have single copies.</p>
<p>The study of karyotypes is important for cell biology and genetics, and the results may be used in evolutionary biology (karyosystematics) and medicine. Karyotypes can be used for many purposes; such as to study chromosomal aberrations, cellular function, taxonomic relationships, and to gather information about past evolutionary events.</p>
<p>The German scientists Matthias <a href="https://en.wikipedia.org/wiki/Matthias_Jakob_Schleiden">Jakob Schleiden</a>, <a href="https://en.wikipedia.org/wiki/Rudolf_Virchow">Rudolf Virchow</a> and <a href="https://en.wikipedia.org/wiki/Otto_Bütschli">Otto Bütschli</a> were among the first scientists who recognized the structures now familiar as chromosomes. Chromosomes were first observed in plant cells by <a href="https://en.wikipedia.org/wiki/Carl_Nägeli">Carl Wilhelm von Nägeli</a> in 1842. Their behavior in animal (salamander) cells was described by Walther Flemming, the discoverer of mitosis, in 1882.</p>
<p>The next stage took place after the development of genetics in the early 20th century, when it was appreciated that chromosomes (that can be observed by karyotype) were the carrier of genes.</p>
<p>In his famous textbook The Cell in Development and Heredity, Wilson linked together the independent work of <a href="https://en.wikipedia.org/wiki/Theodor_Boveri">Theodor Boveri</a> and <a href="https://en.wikipedia.org/wiki/Walter_Sutton">Walter Sutton</a> (both around 1902) by naming the chromosome theory of inheritance the <a href="https://en.wikipedia.org/wiki/Boveri–Sutton_chromosome_theory">Boveri–Sutton chromosome theory</a> (the names are sometimes reversed).</p>
<p>Investigation into the human karyotype took many years to settle the most basic question: how many chromosomes does a normal diploid human cell contain? In 1912, Hans von Winiwarter reported 47 chromosomes in spermatogonia and 48 in oogonia, concluding an XX/XO sex determination mechanism. The number of human chromosomes was published in 1923 by Theophilus Painter. By inspection through the microscope, he counted 24 pairs, which would mean 48 chromosomes. His error was copied by others and it was not until 1956 that the true number, 46, was determined by Indonesia-born cytogeneticist <a href="https://en.wikipedia.org/wiki/Joe_Hin_Tjio">Joe Hin Tjio</a>. He was responsible for finding a new approach that settled the issue:</p>
<ul>
<li>Using cells in tissue culture</li>
<li>Pretreating cells in a hypotonic solution, which swells them and spreads the chromosomes</li>
<li>Arresting mitosis in metaphase by a solution of colchicine</li>
<li>Squashing the preparation on the slide forcing the chromosomes into a single plane</li>
<li>Cutting up a photomicrograph and arranging the result into an indisputable karyogram.</li>
</ul>
<p>The work took place in 1955, and was published in 1956. The karyotype of humans includes only 46 chromosomes. Rather interestingly, the great apes have 48 chromosomes. Human chromosome 2 is now known to be a result of an end-to-end fusion of two ancestral ape chromosomes.</p>
</div>
<div id="observations-on-karyotypes" class="section level2">
<h2><span class="header-section-number">5.3</span> Observations on karyotypes</h2>
<div id="staining" class="section level3">
<h3><span class="header-section-number">5.3.1</span> Staining</h3>
<p>The study of karyotypes is made possible by staining. Usually, a suitable dye, such as Giemsa, is applied after cells have been arrested during cell division by a solution of colchicine usually in metaphase or prometaphase when most condensed. In order for the Giemsa stain to adhere correctly, all chromosomal proteins must be digested and removed. For humans, white blood cells are used most frequently because they are easily induced to divide and grow in tissue culture. Sometimes observations may be made on non-dividing (interphase) cells. The sex of an unborn fetus can be determined by observation of interphase cells (see amniotic centesis and Barr body).</p>
</div>
<div id="observations" class="section level3">
<h3><span class="header-section-number">5.3.2</span> Observations</h3>
<p>Six different characteristics of karyotypes are usually observed and compared:</p>
<ol style="list-style-type: decimal">
<li>Differences in absolute sizes of chromosomes. Chromosomes can vary in absolute size by as much as twenty-fold between genera of the same family. For example, the legumes <em>Lotus tenuis</em> and <em>Vicia faba</em> each have six pairs of chromosomes, yet <em>V. faba</em> chromosomes are many times larger. These differences probably reflect different amounts of DNA duplication.</li>
<li>Differences in the position of centromeres. These differences probably came about through translocations.</li>
<li>Differences in relative size of chromosomes. These differences probably arose from segmental interchange of unequal lengths.</li>
<li>Differences in basic number of chromosomes. These differences could have resulted from successive unequal translocations which removed all the essential genetic material from a chromosome, permitting its loss without penalty to the organism (the dislocation hypothesis) or through fusion. Humans have one pair fewer chromosomes than the great apes. Human chromosome 2 appears to have resulted from the fusion of two ancestral chromosomes, and many of the genes of those two original chromosomes have been translocated to other chromosomes.</li>
<li>Differences in number and position of satellites. Satellites are small bodies attached to a chromosome by a thin thread.</li>
<li>Differences in degree and distribution of heterochromatic regions. Heterochromatin stains darker than euchromatin. Heterochromatin is packed tighter. Heterochromatin consists mainly of genetically inactive and repetitive DNA sequences as well as containing a larger amount of Adenine-Thymine pairs. Euchromatin is usually under active transcription and stains much lighter as it has less affinity for the giemsa stain. Euchromatin regions contain larger amounts of Guanine-Cytosine pairs. The staining technique using giemsa staining is called G banding and therefore produces the typical “G-Bands”.</li>
</ol>
<p>A full account of a karyotype may therefore include the number, type, shape and banding of the chromosomes, as well as other cytogenetic information.</p>
<p>Variation is often found:</p>
<ul>
<li>between the sexes,</li>
<li>between the germ-line and soma (between gametes and the rest of the body),</li>
<li>between members of a population (chromosome polymorphism), in geographic specialization, and in mosaics or otherwise abnormal individuals. Human karyotype.</li>
</ul>
</div>
</div>
<div id="human-karyotype" class="section level2">
<h2><span class="header-section-number">5.4</span> Human karyotype</h2>
<p>The normal human karyotypes contain 22 pairs of autosomal chromosomes and one pair of sex chromosomes (allosomes; Figure <a href="chromosomes-and-karyotypes.html#fig:karyo">5.2</a>). Normal karyotypes for females contain two X chromosomes and are denoted 46,XX; males have both an X and a Y chromosome denoted 46,XY. Any variation from the standard karyotype may lead to developmental abnormalities.</p>
<div id="ploidy" class="section level3">
<h3><span class="header-section-number">5.4.1</span> Ploidy</h3>
<p><a href="https://en.wikipedia.org/wiki/Ploidy">Ploidy</a> is the number of complete sets of chromosomes in a cell.</p>
</div>
<div id="aneuploidy" class="section level3">
<h3><span class="header-section-number">5.4.2</span> Aneuploidy</h3>
<p><a href="https://en.wikipedia.org/wiki/Aneuploidy">Aneuploidy</a> is the condition in which the chromosome number in the cells is not the typical number for the species. This would give rise to a chromosome abnormality such as an extra chromosome or one or more chromosomes lost. Abnormalities in chromosome number usually cause a defect in development. <a href="https://en.wikipedia.org/wiki/Down_syndrome">Down syndrome</a> (Figure <a href="chromosomes-and-karyotypes.html#fig:Down">5.5</a>) and <a href="https://en.wikipedia.org/wiki/Turner_syndrome">Turner syndrome</a> are examples of this.</p>
</div>
</div>
<div id="chromosome-abnormalities" class="section level2">
<h2><span class="header-section-number">5.5</span> Chromosome abnormalities</h2>
<p>Chromosome abnormalities can be numerical, as in the presence of extra or missing chromosomes, or structural, as in derivative chromosome, translocations, inversions, large-scale deletions or duplications. Numerical abnormalities, also known as aneuploidy, often occur as a result of nondisjunction during meiosis in the formation of a gamete; trisomies, in which three copies of a chromosome are present instead of the usual two, are common numerical abnormalities. Structural abnormalities often arise from errors in homologous recombination. Both types of abnormalities can occur in gametes and therefore will be present in all cells of an affected person’s body, or they can occur during mitosis and give rise to a genetic mosaic individual who has some normal and some abnormal cells.</p>
<div id="chromosomal-abnormalities-that-lead-to-disease-in-humans-include" class="section level3">
<h3><span class="header-section-number">5.5.1</span> Chromosomal abnormalities that lead to disease in humans include</h3>
<ul>
<li>Turner syndrome (Figure <a href="chromosomes-and-karyotypes.html#fig:Turner">5.3</a>) results from a single X chromosome (45,X or 45,X0).</li>
<li><a href="https://en.wikipedia.org/wiki/Klinefelter_syndrome">Klinefelter syndrome</a> (Figure <a href="chromosomes-and-karyotypes.html#fig:Klinefelter">5.4</a>), the most common male chromosomal disease, otherwise known as 47,XXY, is caused by an extra X chromosome.</li>
<li><a href="https://en.wikipedia.org/wiki/Edwards_syndrome">Edwards syndrome</a> is caused by trisomy (three copies) of chromosome 18.</li>
<li>Down syndrome (Figure <a href="chromosomes-and-karyotypes.html#fig:Down">5.5</a>), a common chromosomal disease, is caused by trisomy of chromosome 21.</li>
<li><a href="https://en.wikipedia.org/wiki/Patau_syndrome">Patau syndrome</a> is caused by trisomy of chromosome 13.</li>
<li><a href="https://en.wikipedia.org/wiki/Trisomy_9">Trisomy 9</a>, believed to be the 4th most common trisomy, has many long lived affected individuals but only in a form other than a full trisomy, such as trisomy 9p syndrome or mosaic trisomy 9. They often function quite well, but tend to have trouble with speech.</li>
<li>Also documented are trisomy 8 and trisomy 16, although they generally do not survive to birth.</li>
</ul>

<div class="figure" style="text-align: center"><span id="fig:Turner"></span>
<img src="figures/karyotyping/45,X.jpg" alt="45,X karyotype, showing an unpaired X at the lower right" width="70%" />
<p class="caption">
Figure 5.3: <a href="https://commons.wikimedia.org/wiki/File:45,X.jpg">45,X karyotype, showing an unpaired X at the lower right</a>
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Klinefelter"></span>
<img src="figures/karyotyping/Human_chromosomesXXY01.jpg" alt="Klinefelter syndrome karyotype (XXY)" width="70%" />
<p class="caption">
Figure 5.4: <a href="https://commons.wikimedia.org/wiki/File:Human_chromosomesXXY01.png">Klinefelter syndrome karyotype (XXY)</a>
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Down"></span>
<img src="figures/karyotyping/Down_Syndrome_Karyotype.jpg" alt="Karyotype for trisomy Down syndrome: notice the three copies of chromosome 21" width="70%" />
<p class="caption">
Figure 5.5: <a href="https://commons.wikimedia.org/wiki/File:Down_Syndrome_Karyotype.png">Karyotype for trisomy Down syndrome: notice the three copies of chromosome 21</a>
</p>
</div>
<p>Some disorders arise from loss of just a piece of one chromosome, including</p>
<ul>
<li><a href="https://en.wikipedia.org/wiki/Cri_du_chat_syndrome">Cri du chat</a> (cry of the cat), from a truncated short arm on chromosome 5. The name comes from the babies’ distinctive cry, caused by abnormal formation of the larynx.</li>
<li>1p36 Deletion syndrome, from the loss of part of the short arm of chromosome 1.</li>
<li>Angelman syndrome – 50% of cases have a segment of the long arm of chromosome 15 missing; a deletion of the maternal genes, example of imprinting disorder.</li>
<li>Prader-Willi syndrome – 50% of cases have a segment of the long arm of chromosome 15 missing; a deletion of the paternal genes, example of imprinting disorder.</li>
<li>Chromosomal abnormalities can also occur in cancerous cells of an otherwise genetically normal individual; one well-documented example is the Philadelphia chromosome, a translocation mutation commonly associated with chronic myelogenous leukemia and less often with acute lymphoblastic leukemia.</li>
</ul>
</div>
</div>
<div id="chromosome-banding" class="section level2">
<h2><span class="header-section-number">5.6</span> Chromosome banding</h2>
<p>Chromosomes display a banded pattern when treated with some stains. Bands are alternating light and dark stripes that appear along the lengths of chromosomes. Unique banding patterns are used to identify chromosomes and to diagnose chromosomal aberrations, including chromosome breakage, loss, duplication, translocation or inverted segments. A range of different chromosome treatments produce a range of banding patterns: G-bands, R-bands, C-bands, Q-bands, T-bands and NOR-bands.</p>
<p>Cytogenetics employs several techniques to visualize different aspects of chromosomes.</p>
<div id="types-of-banding" class="section level3">
<h3><span class="header-section-number">5.6.1</span> Types of banding</h3>
<ul>
<li>G-banding is obtained with Giemsa stain following digestion of chromosomes with trypsin. It yields a series of lightly and darkly stained bands — the dark regions tend to be heterochromatic, late-replicating and AT rich. The light regions tend to be euchromatic, early-replicating and GC rich. This method will normally produce 300–400 bands in a normal, human genome.</li>
<li>R-banding is the reverse of G-banding (the R stands for “reverse”). The dark regions are euchromatic (guanine-cytosine rich regions) and the bright regions are heterochromatic (thymine-adenine rich regions).</li>
<li>C-banding: Giemsa binds to constitutive heterochromatin, so it stains centromeres.The name is derived from centromeric or constitutive heterochromatin. The preparations undergo alkaline denaturation prior to staining leading to an almost complete depurination of the DNA. After washing the probe the remaining DNA is renatured again and stained with Giemsa solution consisting of methylene azure, methylene violet, methylene blue, and eosin. Heterochromatin binds a lot of the dye, while the rest of the chromosomes absorb only little of it. The C-bonding proved to be especially well-suited for the characterization of plant chromosomes.</li>
<li>Q-banding is a fluorescent pattern obtained using quinacrine for staining. The pattern of bands is very similar to that seen in G-banding.They can be recognized by a yellow fluorescence of differing intensity. Most part of the stained DNA is heterochromatin. Quinacrin (atebrin) binds both regions rich in AT and in GC, but only the AT-quinacrin-complex fluoresces. Since regions rich in AT are more common in heterochromatin than in euchromatin, these regions are labelled preferentially. The different intensities of the single bands mirror the different contents of AT. Other fluorochromes like DAPI or Hoechst 33258 lead also to characteristic, reproducible patterns. Each of them produces its specific pattern. In other words: the properties of the bonds and the specificity of the fluorochromes are not exclusively based on their affinity to regions rich in AT. Rather, the distribution of AT and the association of AT with other molecules like histones, for example, influences the binding properties of the fluorochromes.</li>
<li>T-banding: visualize telomeres. Silver staining: Silver nitrate stains the nucleolar organization region-associated protein. This yields a dark region where the silver is deposited, denoting the activity of rRNA genes within the NOR.</li>
</ul>
</div>
<div id="classic-karyotype-cytogenetics" class="section level3">
<h3><span class="header-section-number">5.6.2</span> Classic karyotype cytogenetics</h3>
<p>In the “classic” (depicted) karyotype, a dye, often Giemsa (G-banding), less frequently mepacrine (quinacrine), is used to stain bands on the chromosomes (Figure <a href="chromosomes-and-karyotypes.html#fig:karyo">5.2</a>). Giemsa is specific for the phosphate groups of DNA. Quinacrine binds to the adenine-thymine-rich regions. Each chromosome has a characteristic banding pattern that helps to identify them; both chromosomes in a pair will have the same banding pattern.</p>
<p>Karyotypes are arranged with the short arm of the chromosome on top, and the long arm on the bottom. Some karyotypes call the short and long arms p and q, respectively. In addition, the differently stained regions and sub-regions are given numerical designations from proximal to distal on the chromosome arms. For example, Cri du chat syndrome involves a deletion on the short arm of chromosome 5. It is written as 46,XX,5p-. The critical region for this syndrome is deletion of p15.2 (the locus on the chromosome), which is written as 46,XX,del(5)(p15.2).</p>
</div>
<div id="multicolor-fish-mfish-and-spectral-karyotype-sky-technique" class="section level3">
<h3><span class="header-section-number">5.6.3</span> Multicolor FISH (mFISH) and spectral karyotype (SKY technique)</h3>
<p>Multicolor FISH and the older spectral karyotyping are molecular cytogenetic techniques used to simultaneously visualize all the pairs of chromosomes in an organism in different colors (Figure <a href="chromosomes-and-karyotypes.html#fig:sky">5.6</a>). Fluorescently labeled probes for each chromosome are made by labeling chromosome-specific DNA with different fluorophores. Because there are a limited number of spectrally distinct fluorophores, a combinatorial labeling method is used to generate many different colors. Fluorophore combinations are captured and analyzed by a fluorescence microscope using up to 7 narrow-banded fluorescence filters or, in the case of spectral karyotyping, by using an interferometer attached to a fluorescence microscope. In the case of an mFISH image, every combination of fluorochromes from the resulting original images is replaced by a pseudo color in a dedicated image analysis software. Thus, chromosomes or chromosome sections can be visualized and identified, allowing for the analysis of chromosomal rearrangements. In the case of spectral karyotyping, image processing software assigns a pseudo color to each spectrally different combination, allowing the visualization of the individually colored chromosomes. Multicolor FISH is used to identify structural chromosome aberrations in cancer cells and other disease conditions when Giemsa banding or other techniques are not accurate enough.</p>

<div class="figure" style="text-align: center"><span id="fig:sky"></span>
<img src="figures/karyotyping/Spectralkaryotype98-300.jpg" alt="Spectral human karyotype" width="70%" />
<p class="caption">
Figure 5.6: <a href="https://en.wikipedia.org/wiki/File:Spectralkaryotype98-300.jpg">Spectral human karyotype</a>
</p>
</div>
</div>
</div>
<div id="determine-the-karyotype" class="section level2">
<h2><span class="header-section-number">5.7</span> Determine the karyotype</h2>
<div id="experimental-procedures-1" class="section level3">
<h3><span class="header-section-number">5.7.1</span> Experimental procedures</h3>
<ol style="list-style-type: decimal">
<li>Obtain a photomicrograph of a metaphase chromosome spread.</li>
<li>Write down the code associated with the photomicrograph.</li>
<li>Using scissors carefully cut out the largest chromosome on the page and paste it onto the “data page”.</li>
<li>Find the matching chromosome and paste it next to the first chromosome.</li>
<li>Repeat these steps until all chromosomes have been pasted onto the data page by decreasing size (as shown in Figure <a href="chromosomes-and-karyotypes.html#fig:karyo">5.2</a>). <strong>Cut out only one chromosome at a time or you will get confused and likely lose chromosomes.</strong></li>
<li>Determine the number of chromosomes on the data page. Any extra chromosome(s) are indication of an aneuploidy.</li>
<li>Determine the type of sex chromosomes on the data page.</li>
<li>Write down the karyotype.</li>
<li>Write down the suggested diagnosis.</li>
</ol>
</div>
</div>
<div id="review-questions-2" class="section level2">
<h2><span class="header-section-number">5.8</span> Review Questions</h2>
<ol style="list-style-type: decimal">
<li>What are chromosomes?</li>
<li>How many chromosomes are found in somatic cells in humans?</li>
<li>How many chromosomes are found in human gametes?</li>
<li>What is a karyotype?</li>
<li>What are examples of abnormal karyotypes in humans?</li>
<li>What is Down syndrome?</li>
<li>What is Turner syndrome?</li>
<li>What is Klinefelter syndrome?</li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="mitosis-and-meiosis.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="mendelian-genetics-mendel-in-his-own-words.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/genetics/edit/master/karyotype.Rmd",
"text": "Edit"
},
"download": ["_main.pdf", "_main.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
