<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>14 DNA sequence analysis | Laboratory Manual For SCI200 Genetics at Roxbury Community College</title>
  <meta name="description" content="This laboratory manual will accompany you during your laboratory for Genetics.">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="14 DNA sequence analysis | Laboratory Manual For SCI200 Genetics at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="cover.jpg" />
  <meta property="og:description" content="This laboratory manual will accompany you during your laboratory for Genetics." />
  <meta name="github-repo" content="nikolaussuchert/genetics" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="14 DNA sequence analysis | Laboratory Manual For SCI200 Genetics at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This laboratory manual will accompany you during your laboratory for Genetics." />
  <meta name="twitter:image" content="cover.jpg" />

<meta name="author" content="Nikolaus Sucher">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="dna-sequencing.html">
<link rel="next" href="the-human-genome.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">RCC Genetics Laboratory Manual</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="course-description.html"><a href="course-description.html"><i class="fa fa-check"></i><b>1</b> Course Description</a><ul>
<li class="chapter" data-level="1.1" data-path="course-description.html"><a href="course-description.html#instructional-objectives"><i class="fa fa-check"></i><b>1.1</b> Instructional Objectives:</a></li>
<li class="chapter" data-level="1.2" data-path="course-description.html"><a href="course-description.html#student-learning-outcomes"><i class="fa fa-check"></i><b>1.2</b> Student Learning Outcomes:</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="how-to-do-well-in-this-class.html"><a href="how-to-do-well-in-this-class.html"><i class="fa fa-check"></i><b>2</b> How to do well in this class</a><ul>
<li class="chapter" data-level="" data-path="how-to-do-well-in-this-class.html"><a href="how-to-do-well-in-this-class.html#before-the-lab"><i class="fa fa-check"></i>Before the lab</a></li>
<li class="chapter" data-level="" data-path="how-to-do-well-in-this-class.html"><a href="how-to-do-well-in-this-class.html#during-the-lab"><i class="fa fa-check"></i>During the lab</a></li>
<li class="chapter" data-level="" data-path="how-to-do-well-in-this-class.html"><a href="how-to-do-well-in-this-class.html#after-the-lab"><i class="fa fa-check"></i>After the lab</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="lab-safety.html"><a href="lab-safety.html"><i class="fa fa-check"></i><b>3</b> Lab Safety</a><ul>
<li class="chapter" data-level="3.1" data-path="lab-safety.html"><a href="lab-safety.html#general-rules"><i class="fa fa-check"></i><b>3.1</b> General rules</a></li>
<li class="chapter" data-level="3.2" data-path="lab-safety.html"><a href="lab-safety.html#class-dissections"><i class="fa fa-check"></i><b>3.2</b> Class dissections</a></li>
<li class="chapter" data-level="3.3" data-path="lab-safety.html"><a href="lab-safety.html#clothing"><i class="fa fa-check"></i><b>3.3</b> Clothing</a></li>
<li class="chapter" data-level="3.4" data-path="lab-safety.html"><a href="lab-safety.html#handling-chemicals"><i class="fa fa-check"></i><b>3.4</b> Handling chemicals</a></li>
<li class="chapter" data-level="3.5" data-path="lab-safety.html"><a href="lab-safety.html#glassware"><i class="fa fa-check"></i><b>3.5</b> Glassware</a></li>
<li class="chapter" data-level="3.6" data-path="lab-safety.html"><a href="lab-safety.html#heating-substances"><i class="fa fa-check"></i><b>3.6</b> Heating substances</a></li>
<li class="chapter" data-level="3.7" data-path="lab-safety.html"><a href="lab-safety.html#handling-microbiology-materials"><i class="fa fa-check"></i><b>3.7</b> Handling microbiology materials</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="classical-genetics.html"><a href="classical-genetics.html"><i class="fa fa-check"></i>Classical Genetics</a><ul>
<li class="chapter" data-level="3.8" data-path="classical-genetics.html"><a href="classical-genetics.html#review-questions"><i class="fa fa-check"></i><b>3.8</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="mitosis-and-meiosis.html"><a href="mitosis-and-meiosis.html"><i class="fa fa-check"></i><b>4</b> Mitosis and Meiosis</a><ul>
<li class="chapter" data-level="4.1" data-path="mitosis-and-meiosis.html"><a href="mitosis-and-meiosis.html#view-prepared-slides"><i class="fa fa-check"></i><b>4.1</b> View Prepared Slides</a></li>
<li class="chapter" data-level="4.2" data-path="mitosis-and-meiosis.html"><a href="mitosis-and-meiosis.html#preparing-an-onion-root-tip-squash"><i class="fa fa-check"></i><b>4.2</b> Preparing an Onion root tip squash</a><ul>
<li class="chapter" data-level="4.2.1" data-path="mitosis-and-meiosis.html"><a href="mitosis-and-meiosis.html#experimental-procedures"><i class="fa fa-check"></i><b>4.2.1</b> Experimental procedures</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="mitosis-and-meiosis.html"><a href="mitosis-and-meiosis.html#review-questions-1"><i class="fa fa-check"></i><b>4.3</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html"><i class="fa fa-check"></i><b>5</b> Chromosomes And Karyotypes</a><ul>
<li class="chapter" data-level="5.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#chromosomes"><i class="fa fa-check"></i><b>5.1</b> Chromosomes</a></li>
<li class="chapter" data-level="5.2" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#karyotypes"><i class="fa fa-check"></i><b>5.2</b> Karyotypes</a></li>
<li class="chapter" data-level="5.3" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#observations-on-karyotypes"><i class="fa fa-check"></i><b>5.3</b> Observations on karyotypes</a><ul>
<li class="chapter" data-level="5.3.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#staining"><i class="fa fa-check"></i><b>5.3.1</b> Staining</a></li>
<li class="chapter" data-level="5.3.2" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#observations"><i class="fa fa-check"></i><b>5.3.2</b> Observations</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#human-karyotype"><i class="fa fa-check"></i><b>5.4</b> Human karyotype</a><ul>
<li class="chapter" data-level="5.4.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#ploidy"><i class="fa fa-check"></i><b>5.4.1</b> Ploidy</a></li>
<li class="chapter" data-level="5.4.2" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#aneuploidy"><i class="fa fa-check"></i><b>5.4.2</b> Aneuploidy</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#chromosome-abnormalities"><i class="fa fa-check"></i><b>5.5</b> Chromosome abnormalities</a><ul>
<li class="chapter" data-level="5.5.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#chromosomal-abnormalities-that-lead-to-disease-in-humans-include"><i class="fa fa-check"></i><b>5.5.1</b> Chromosomal abnormalities that lead to disease in humans include</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#chromosome-banding"><i class="fa fa-check"></i><b>5.6</b> Chromosome banding</a><ul>
<li class="chapter" data-level="5.6.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#types-of-banding"><i class="fa fa-check"></i><b>5.6.1</b> Types of banding</a></li>
<li class="chapter" data-level="5.6.2" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#classic-karyotype-cytogenetics"><i class="fa fa-check"></i><b>5.6.2</b> Classic karyotype cytogenetics</a></li>
<li class="chapter" data-level="5.6.3" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#multicolor-fish-mfish-and-spectral-karyotype-sky-technique"><i class="fa fa-check"></i><b>5.6.3</b> Multicolor FISH (mFISH) and spectral karyotype (SKY technique)</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#determine-the-karyotype"><i class="fa fa-check"></i><b>5.7</b> Determine the karyotype</a><ul>
<li class="chapter" data-level="5.7.1" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#experimental-procedures-1"><i class="fa fa-check"></i><b>5.7.1</b> Experimental procedures</a></li>
</ul></li>
<li class="chapter" data-level="5.8" data-path="chromosomes-and-karyotypes.html"><a href="chromosomes-and-karyotypes.html#review-questions-2"><i class="fa fa-check"></i><b>5.8</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="mendelian-genetics-mendel-in-his-own-words.html"><a href="mendelian-genetics-mendel-in-his-own-words.html"><i class="fa fa-check"></i><b>6</b> Mendelian Genetics: Mendel in His Own Words</a><ul>
<li class="chapter" data-level="6.1" data-path="mendelian-genetics-mendel-in-his-own-words.html"><a href="mendelian-genetics-mendel-in-his-own-words.html#experimental-procedures-2"><i class="fa fa-check"></i><b>6.1</b> Experimental Procedures</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html"><i class="fa fa-check"></i><b>7</b> Mendelian Genetics: A Practical Example</a><ul>
<li class="chapter" data-level="7.1" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#monohybrid-cross-experiment-1"><i class="fa fa-check"></i><b>7.1</b> Monohybrid Cross (Experiment 1)</a></li>
<li class="chapter" data-level="7.2" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#punnett-square"><i class="fa fa-check"></i><b>7.2</b> Punnett Square</a><ul>
<li class="chapter" data-level="7.2.1" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#experimental-procedures-3"><i class="fa fa-check"></i><b>7.2.1</b> Experimental Procedures</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#dihybrid-cross-experiment-2"><i class="fa fa-check"></i><b>7.3</b> Dihybrid Cross (Experiment 2)</a><ul>
<li class="chapter" data-level="7.3.1" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#experimental-procedures-4"><i class="fa fa-check"></i><b>7.3.1</b> Experimental Procedures</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#pearsons-chi-squared-chi2-and-goodness-of-fit-test"><i class="fa fa-check"></i><b>7.4</b> Pearson’s Chi-Squared (<span class="math inline">\(\chi^2\)</span>) And Goodness Of Fit Test</a><ul>
<li class="chapter" data-level="7.4.1" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#calculating-chi2-using-the-monohybrid-cross-data"><i class="fa fa-check"></i><b>7.4.1</b> Calculating <span class="math inline">\(\chi^2\)</span> Using The Monohybrid Cross Data</a></li>
<li class="chapter" data-level="7.4.2" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#testing-goodness-of-fit"><i class="fa fa-check"></i><b>7.4.2</b> Testing Goodness Of Fit</a></li>
<li class="chapter" data-level="7.4.3" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#calculate-chi2-for-your-dihybrid-cross-data"><i class="fa fa-check"></i><b>7.4.3</b> Calculate <span class="math inline">\(\chi^2\)</span> For Your Dihybrid Cross Data</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#a-look-at-mendels-monohybrid-cross-data"><i class="fa fa-check"></i><b>7.5</b> A Look At Mendel’s Monohybrid Cross Data</a></li>
<li class="chapter" data-level="7.6" data-path="mendelian-genetics-a-practical-example.html"><a href="mendelian-genetics-a-practical-example.html#review-questions-3"><i class="fa fa-check"></i><b>7.6</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="molecular-genetics.html"><a href="molecular-genetics.html"><i class="fa fa-check"></i>Molecular Genetics</a></li>
<li class="chapter" data-level="8" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html"><i class="fa fa-check"></i><b>8</b> Molecular Laboratory Skills</a><ul>
<li class="chapter" data-level="8.1" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#exercise-1-using-graduated-serological-macropipettes"><i class="fa fa-check"></i><b>8.1</b> Exercise 1: using graduated (serological) macropipettes</a><ul>
<li class="chapter" data-level="8.1.1" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#experimental-procedures-5"><i class="fa fa-check"></i><b>8.1.1</b> Experimental procedures</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#exercise-2-using-air-displacement-micropipettes"><i class="fa fa-check"></i><b>8.2</b> Exercise 2: using air displacement micropipettes</a><ul>
<li class="chapter" data-level="8.2.1" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#experimental-procedures-6"><i class="fa fa-check"></i><b>8.2.1</b> Experimental procedures</a></li>
</ul></li>
<li class="chapter" data-level="8.3" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#exercise-3-using-a-microcentrifuge"><i class="fa fa-check"></i><b>8.3</b> Exercise 3: using a microcentrifuge</a><ul>
<li class="chapter" data-level="8.3.1" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#experimental-procedures-7"><i class="fa fa-check"></i><b>8.3.1</b> Experimental procedures</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="molecular-laboratory-skills.html"><a href="molecular-laboratory-skills.html#review-questions-4"><i class="fa fa-check"></i><b>8.4</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html"><i class="fa fa-check"></i><b>9</b> DNA Extraction and Polymerase Chain Reaction</a><ul>
<li class="chapter" data-level="9.1" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#dna"><i class="fa fa-check"></i><b>9.1</b> DNA</a></li>
<li class="chapter" data-level="9.2" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#polymerase-chain-reaction"><i class="fa fa-check"></i><b>9.2</b> Polymerase chain reaction</a></li>
<li class="chapter" data-level="9.3" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#dna-extraction"><i class="fa fa-check"></i><b>9.3</b> DNA Extraction</a></li>
<li class="chapter" data-level="9.4" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#experimental-procedures-8"><i class="fa fa-check"></i><b>9.4</b> Experimental Procedures</a></li>
<li class="chapter" data-level="9.5" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#gapdh-polymerase-chain-reaction"><i class="fa fa-check"></i><b>9.5</b> GAPDH Polymerase Chain Reaction</a></li>
<li class="chapter" data-level="9.6" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#experimental-procedures-9"><i class="fa fa-check"></i><b>9.6</b> Experimental Procedures</a></li>
<li class="chapter" data-level="9.7" data-path="dna-extraction-and-polymerase-chain-reaction.html"><a href="dna-extraction-and-polymerase-chain-reaction.html#review-questions-5"><i class="fa fa-check"></i><b>9.7</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html"><i class="fa fa-check"></i><b>10</b> Exonuclease I Digestion And Nested PCR</a><ul>
<li class="chapter" data-level="10.1" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html#exonuclease-i-digestion"><i class="fa fa-check"></i><b>10.1</b> Exonuclease I Digestion</a></li>
<li class="chapter" data-level="10.2" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html#experimental-procedures-10"><i class="fa fa-check"></i><b>10.2</b> Experimental Procedures</a></li>
<li class="chapter" data-level="10.3" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html#nested-polymerase-chain-reaction"><i class="fa fa-check"></i><b>10.3</b> Nested polymerase chain reaction</a></li>
<li class="chapter" data-level="10.4" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html#experimental-procedures-11"><i class="fa fa-check"></i><b>10.4</b> Experimental Procedures</a></li>
<li class="chapter" data-level="10.5" data-path="exonuclease-i-digestion-and-nested-pcr.html"><a href="exonuclease-i-digestion-and-nested-pcr.html#review-questions-6"><i class="fa fa-check"></i><b>10.5</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html"><i class="fa fa-check"></i><b>11</b> Agarose Gel Electrophoresis, Ligation and Transformation</a><ul>
<li class="chapter" data-level="11.1" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#agarose-gel-electrophoresis"><i class="fa fa-check"></i><b>11.1</b> Agarose gel electrophoresis</a></li>
<li class="chapter" data-level="11.2" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#experimental-procedures-12"><i class="fa fa-check"></i><b>11.2</b> Experimental Procedures</a></li>
<li class="chapter" data-level="11.3" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#pcr-product-purification"><i class="fa fa-check"></i><b>11.3</b> PCR Product Purification</a></li>
<li class="chapter" data-level="11.4" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#experimental-procedures-13"><i class="fa fa-check"></i><b>11.4</b> Experimental Procedures</a></li>
<li class="chapter" data-level="11.5" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#ligation"><i class="fa fa-check"></i><b>11.5</b> Ligation</a></li>
<li class="chapter" data-level="11.6" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#experimental-procedures-14"><i class="fa fa-check"></i><b>11.6</b> Experimental Procedures</a></li>
<li class="chapter" data-level="11.7" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#transformation"><i class="fa fa-check"></i><b>11.7</b> Transformation</a></li>
<li class="chapter" data-level="11.8" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#experimental-procedures-15"><i class="fa fa-check"></i><b>11.8</b> Experimental Procedures</a></li>
<li class="chapter" data-level="11.9" data-path="agarose-gel-electrophoresis-ligation-and-transformation.html"><a href="agarose-gel-electrophoresis-ligation-and-transformation.html#review-questions-7"><i class="fa fa-check"></i><b>11.9</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="plasmid-purification-and-restriction-digest.html"><a href="plasmid-purification-and-restriction-digest.html"><i class="fa fa-check"></i><b>12</b> Plasmid Purification And Restriction Digest</a><ul>
<li class="chapter" data-level="12.1" data-path="plasmid-purification-and-restriction-digest.html"><a href="plasmid-purification-and-restriction-digest.html#plasmid-purification"><i class="fa fa-check"></i><b>12.1</b> Plasmid Purification</a></li>
<li class="chapter" data-level="12.2" data-path="plasmid-purification-and-restriction-digest.html"><a href="plasmid-purification-and-restriction-digest.html#plasmid"><i class="fa fa-check"></i><b>12.2</b> Plasmid</a></li>
<li class="chapter" data-level="12.3" data-path="plasmid-purification-and-restriction-digest.html"><a href="plasmid-purification-and-restriction-digest.html#experimental-procedures-16"><i class="fa fa-check"></i><b>12.3</b> Experimental Procedures</a></li>
<li class="chapter" data-level="12.4" data-path="plasmid-purification-and-restriction-digest.html"><a href="plasmid-purification-and-restriction-digest.html#restriction-digest"><i class="fa fa-check"></i><b>12.4</b> Restriction Digest</a></li>
<li class="chapter" data-level="12.5" data-path="plasmid-purification-and-restriction-digest.html"><a href="plasmid-purification-and-restriction-digest.html#experimental-procedures-17"><i class="fa fa-check"></i><b>12.5</b> Experimental Procedures</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="dna-sequencing.html"><a href="dna-sequencing.html"><i class="fa fa-check"></i><b>13</b> DNA sequencing</a><ul>
<li class="chapter" data-level="13.1" data-path="dna-sequencing.html"><a href="dna-sequencing.html#experimental-procedures-18"><i class="fa fa-check"></i><b>13.1</b> Experimental Procedures</a></li>
<li class="chapter" data-level="13.2" data-path="dna-sequencing.html"><a href="dna-sequencing.html#review-questions-8"><i class="fa fa-check"></i><b>13.2</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="dna-sequence-analysis.html"><a href="dna-sequence-analysis.html"><i class="fa fa-check"></i><b>14</b> DNA sequence analysis</a><ul>
<li class="chapter" data-level="14.1" data-path="dna-sequence-analysis.html"><a href="dna-sequence-analysis.html#experimental-procedures-19"><i class="fa fa-check"></i><b>14.1</b> Experimental procedures</a></li>
<li class="chapter" data-level="14.2" data-path="dna-sequence-analysis.html"><a href="dna-sequence-analysis.html#blast-search"><i class="fa fa-check"></i><b>14.2</b> BLAST search</a></li>
<li class="chapter" data-level="14.3" data-path="dna-sequence-analysis.html"><a href="dna-sequence-analysis.html#experimental-procedures-20"><i class="fa fa-check"></i><b>14.3</b> Experimental procedures</a></li>
<li class="chapter" data-level="14.4" data-path="dna-sequence-analysis.html"><a href="dna-sequence-analysis.html#review-questions-9"><i class="fa fa-check"></i><b>14.4</b> Review Questions</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="the-human-genome.html"><a href="the-human-genome.html"><i class="fa fa-check"></i><b>15</b> The Human Genome</a><ul>
<li class="chapter" data-level="15.1" data-path="the-human-genome.html"><a href="the-human-genome.html#the-human-genome-project"><i class="fa fa-check"></i><b>15.1</b> The Human Genome Project</a></li>
<li class="chapter" data-level="15.2" data-path="the-human-genome.html"><a href="the-human-genome.html#public-private-competition-around-the-human-genome-project"><i class="fa fa-check"></i><b>15.2</b> Public-Private Competition Around The Human Genome Project</a></li>
<li class="chapter" data-level="15.3" data-path="the-human-genome.html"><a href="the-human-genome.html#genome-annotation"><i class="fa fa-check"></i><b>15.3</b> Genome Annotation</a></li>
<li class="chapter" data-level="15.4" data-path="the-human-genome.html"><a href="the-human-genome.html#international-hapmap-project"><i class="fa fa-check"></i><b>15.4</b> International HapMap Project</a></li>
<li class="chapter" data-level="15.5" data-path="the-human-genome.html"><a href="the-human-genome.html#the-single-nucleotide-polymorphism-database-dbsnp-of-nucleotide-sequence-variation"><i class="fa fa-check"></i><b>15.5</b> The Single Nucleotide Polymorphism Database (dbSNP) of Nucleotide Sequence Variation</a></li>
<li class="chapter" data-level="15.6" data-path="the-human-genome.html"><a href="the-human-genome.html#the-1000-genomes-project"><i class="fa fa-check"></i><b>15.6</b> The 1000 Genomes Project</a></li>
<li class="chapter" data-level="15.7" data-path="the-human-genome.html"><a href="the-human-genome.html#personal-genomics"><i class="fa fa-check"></i><b>15.7</b> Personal genomics</a></li>
<li class="chapter" data-level="15.8" data-path="the-human-genome.html"><a href="the-human-genome.html#viewing-the-human-genome"><i class="fa fa-check"></i><b>15.8</b> Viewing The Human Genome</a><ul>
<li class="chapter" data-level="15.8.1" data-path="the-human-genome.html"><a href="the-human-genome.html#experimental-procedures-21"><i class="fa fa-check"></i><b>15.8.1</b> Experimental Procedures</a></li>
</ul></li>
<li class="chapter" data-level="15.9" data-path="the-human-genome.html"><a href="the-human-genome.html#review-questions-10"><i class="fa fa-check"></i><b>15.9</b> Review Questions</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Laboratory Manual For SCI200 Genetics at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="dna-sequence-analysis" class="section level1">
<h1><span class="header-section-number">14</span> DNA sequence analysis</h1>
<p>In this laboratory session, we will analyze the DNA sequence data from our cloned GAPDH genes generated by the DNA sequencing facility. We will learn how to use various offline and online bioinformatics software tools. <a href="https://en.wikipedia.org/wiki/Bioinformatics">Bioinformatics</a> is an interdisciplinary field that develops methods and software tools for understanding biological data. As an interdisciplinary field of science, bioinformatics combines biology, computer science, mathematics and statistics to analyze and interpret biological data. Bioinformatics has been used for <em>in silico</em> analyses of biological queries using mathematical and statistical techniques.</p>
<p>In the last laboratory session (Chapter <a href="dna-sequencing.html#dna-sequencing">13</a>), we set up the DNA sequencing reactions that the laboratory technicians mailed to the sequencing facility. There, DNA sequencing was performed using the chain termination method which was invented by Frederick Sanger and colleauges in the 1970s. This sequencing method is commonly referred to as <a href="https://en.wikipedia.org/wiki/Sanger_sequencing">Sanger sequencing</a> (Figure <a href="dna-sequence-analysis.html#fig:sanger">14.1</a>). Frederick Sanger received a Nobel Prize for his invention.</p>
<div class="figure" style="text-align: center"><span id="fig:sanger"></span>
<img src="figures/sequencing/sanger.svg" alt="The Sanger (chain-termination) method for DNA sequencing. (1) A primer is annealed to a sequence, (2) Reagents are added to the primer and template, including: DNA polymerase, dNTPs, and a small amount of all four dideoxynucleotides (ddNTPs) labeled with fluorophores. During primer elongation, the random insertion of a ddNTP instead of a dNTP terminates synthesis of the chain because DNA polymerase cannot react with the missing hydroxyl. This produces all possible lengths of chains. (3) The products are separated on a single lane capillary gel, where the resulting bands are read by a imaging system. (4) This produces several hundred thousand nucleotides a day, data which require storage and subsequent computational analysis. By Estevezj [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0)], from [Wikimedia Commons.](https://commons.wikimedia.org/wiki/File:Sanger-sequencing.svg)" width="70%" />
<p class="caption">
Figure 14.1: The Sanger (chain-termination) method for DNA sequencing. (1) A primer is annealed to a sequence, (2) Reagents are added to the primer and template, including: DNA polymerase, dNTPs, and a small amount of all four dideoxynucleotides (ddNTPs) labeled with fluorophores. During primer elongation, the random insertion of a ddNTP instead of a dNTP terminates synthesis of the chain because DNA polymerase cannot react with the missing hydroxyl. This produces all possible lengths of chains. (3) The products are separated on a single lane capillary gel, where the resulting bands are read by a imaging system. (4) This produces several hundred thousand nucleotides a day, data which require storage and subsequent computational analysis. By Estevezj <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a>], from <a href="https://commons.wikimedia.org/wiki/File:Sanger-sequencing.svg">Wikimedia Commons.</a>
</p>
</div>
<p>The chain-termination method requires a single-stranded DNA template, a DNA primer, a DNA polymerase, normal deoxynucleosidetriphosphates (dNTPs), and modified di-deoxynucleotidetriphosphates (ddNTPs). These chain-terminating nucleotides lack a 3’-OH group required for the formation of a phosphodiestee bend between two nucleotides. Therefore, the DNA polymerase ceases extension of DNA whenever a modified ddNTP is incorporated. The ddNTPs thus <strong>terminate</strong> the elongation of the newly copied DNA strand. Originally, the ddNTPs were radioactively labelled and four separate sequencing reactions, one for each of the four ddNTPs, had to be set up. Following several rounds of template DNA extension, the resulting DNA fragments were heat denatured and separated by size using gel electrophoresis. Each of the four sequencing reactions was loaded in separate lanes (lanes A, T, G, C). The DNA bands were then visualized by autoradiography. The DNA sequence could be read off the X-ray film by reading from the shortest (at the bottom of the gel) to the longest (at the top of the gel) fragment across the four lanes of the gel.</p>
<p>Since its invention, Sanger sequencing has been improved by several modifications. For example, in cycle sequencing a thermostable DNA polymerase is used. This polymerase can be heated to 95 °C and still retain activity and the sequencing reaction can be repeated multiple times in the same tube by heating the mixture to denature the DNA and then allowing it to cool to anneal the primers and polymerize new strands (similar to PCR). Thus, less DNA is needed than for conventional sequencing reactions. Moreover, today, fluorescently labeled ddNTPs are used. Each ddNTP is labelled with a different fluorescent dye (emitting light at different wavelengths; e.g. blue, green, yellow and red). The labelled DNA fragments are separated using capillary electrophoresis. A laser is used to excite the fluorescence of the labeled ddNTPs and a video camera records the color signal which is digitized and stored on a computer as a digital chromatogram. The chromatogram is analyzed using computer software which generates a graph (also referred to a as a trace) of the intensity of each color against electrophoresis running time resulting in overlapping peaks and troughs of different color. The peaks are used to assign a corresponding sequence letter (A, T, C, G or N in case the software cannot unequivocally decide which base to call). The plotted peak intensities and associated base calls are saved in a seqeuncing data file with the extension “.ab1”. Part of a sequencing trace with associated base calls is shown in Fig. <a href="dna-sequence-analysis.html#fig:chromatogram">14.2</a>. A base is considered to be of high quality when its identity is unambiguous. A high-quality region of sequence has evenly spaced peaks that do only overlap at their base and has signal intensity in the proper range for the detection software. In today’s laboratory session, we will analyze the data files containing the results of our sequencing project.</p>
<div class="figure" style="text-align: center"><span id="fig:chromatogram"></span>
<img src="figures/sequencing/chromatogram.png" alt="DNA sequence chromatogram and base calls viewed with SnapGene software." width="70%" />
<p class="caption">
Figure 14.2: DNA sequence chromatogram and base calls viewed with SnapGene software.
</p>
</div>
<div id="experimental-procedures-19" class="section level2">
<h2><span class="header-section-number">14.1</span> Experimental procedures</h2>
<ol style="list-style-type: decimal">
<li>Open a web browser on your computer. If you are reading this in your browser, right click on the highlighted link to open the <a href="http://www.snapgene.com/products/snapgene_viewer/">SnapGene</a> web site in a new tab. Otherwise enter the link in your browser manually.</li>
<li>Download SnapGene by clicking on the button corresponding to the operating system of your computer.</li>
<li>Once your download has finished, install the program.</li>
<li>After the installation has completed, double click on the program icon and start the program.</li>
<li>A program window will open. Click on the “Open” icon and navigate to the folder where the sequence files are stored on your computer.<br />
</li>
<li>Click on the sequence file that ends with “.ab1”. Click open.</li>
<li>A new window will open showing the sequence chromatogram trace and associated base calls (Figure <a href="dna-sequence-analysis.html#fig:chromatogram">14.2</a>).<br />
</li>
<li>Locate two buttons with arrows pointing to the right and left in the top right corner of the SnapGene Viewer window.</li>
<li>Click on the left button (arrow points to the right). This will bring you to the 5’-end of the sequence.</li>
<li>Click on the right button (arrow points to the left) to go to the 3’-end of the sequence.</li>
<li>Look for a number next to the right button. It indicates the length of the sequence.</li>
<li>Using your trackpad (or computer mouse) scan slowly from the 5’ to the 3’ end of the sequence and look at the chromatogram trace.</li>
<li>Are the peaks sharp and clearly separated from each other with little overlap?</li>
<li>Was the software able to call the bases or are there any (or many) “N” labels above the trace?</li>
</ol>
</div>
<div id="blast-search" class="section level2">
<h2><span class="header-section-number">14.2</span> BLAST search</h2>
<p>Now that we have inspected our DNA sequences and ascertained their quality, we will use the BLAST (Basic Local Alignment Tool) program to compare it with all DNA sequences stored in Genbank. <a href="https://www.ncbi.nlm.nih.gov/genbank/">GenBank</a> is the NIH genetic sequence database, an annotated collection of all publicly available DNA sequences. GenBank is part of the International Nucleotide Sequence Database Collaboration, which comprises the DNA DataBank of Japan (DDBJ), the European Nucleotide Archive (ENA), and GenBank at NCBI. These three organizations exchange data on a daily basis.</p>
<p>There are several ways to search and retrieve data from GenBank:</p>
<ol style="list-style-type: decimal">
<li>Search GenBank for sequence identifiers and annotations with Entrez Nucleotide, which is divided into three divisions:
<ul>
<li>CoreNucleotide (the main collection)</li>
<li>dbEST (Expressed Sequence Tags)</li>
<li>dbGSS (Genome Survey Sequences).</li>
</ul></li>
<li>Search and align GenBank sequences to a query sequence using BLAST (Basic Local Alignment Search Tool).</li>
</ol>
<p>We will use BLAST to search te CoreNucleotide main collection.</p>
</div>
<div id="experimental-procedures-20" class="section level2">
<h2><span class="header-section-number">14.3</span> Experimental procedures</h2>
<ol style="list-style-type: decimal">
<li>Open a web browser on your computer. If you are reading this in your browser, right click on the highlighted link to open the web site of the <a href="https://www.nlm.nih.gov">U.S. National Library of Medicine</a> in a new tab (Fig. <a href="dna-sequence-analysis.html#fig:nlm">14.3</a>). Otherwise enter the link in your browser manually.</li>
</ol>

<div class="figure" style="text-align: center"><span id="fig:nlm"></span>
<img src="figures/bioinformatics/nlm.png" alt="The web page of the U.S. National Library of Medicine." width="70%" />
<p class="caption">
Figure 14.3: The web page of the U.S. National Library of Medicine.
</p>
</div>
<ol start="2" style="list-style-type: decimal">
<li>On the U.S. National Library of Medicine web page (Fig. <a href="dna-sequence-analysis.html#fig:nlm">14.3</a>), click on the right most square with the DNA icon and BLAST (Basic Local Alignment Tool) written on it.</li>
<li>On the newly opened page (Fig. <a href="dna-sequence-analysis.html#fig:blast">14.4</a>), you will see a row of squares displaying various titles. Click on the left most square that has Nucleotide Blast written on it.</li>
</ol>

<div class="figure" style="text-align: center"><span id="fig:blast"></span>
<img src="figures/bioinformatics/blast.png" alt="The Basic Local Alignment Search Tool (BLAST) start page." width="70%" />
<p class="caption">
Figure 14.4: The Basic Local Alignment Search Tool (BLAST) start page.
</p>
</div>
<ol start="4" style="list-style-type: decimal">
<li>A new page will open. Paste your DNA sequence into the white box on the top left (Fig. <a href="dna-sequence-analysis.html#fig:sequence">14.5</a>).</li>
</ol>

<div class="figure" style="text-align: center"><span id="fig:sequence"></span>
<img src="figures/bioinformatics/sequence_entry.png" alt="Nucleotide BLAST (BLASTN) sequence entry form." width="70%" />
<p class="caption">
Figure 14.5: Nucleotide BLAST (BLASTN) sequence entry form.
</p>
</div>
<ol start="5" style="list-style-type: decimal">
<li>Leave the default values unchanged and click on the oval blue button at the lower left that has BLAST written on it (Fig. <a href="dna-sequence-analysis.html#fig:click">14.6</a>). This will upload your sequence (referred to from now on as the <strong>Query</strong> sequence) to the server where it will be compared to all sequences on record.</li>
</ol>
<div class="figure" style="text-align: center"><span id="fig:click"></span>
<img src="figures/bioinformatics/click_blast.png" alt="After you have pasted your sequence into the sequence entry field, click the BLAST button at the lower left of the page." width="70%" />
<p class="caption">
Figure 14.6: After you have pasted your sequence into the sequence entry field, click the BLAST button at the lower left of the page.
</p>
</div>
<ol start="6" style="list-style-type: decimal">
<li>A new page will open that will be updated every 2 seconds while your query is being processed (Fig. <a href="dna-sequence-analysis.html#fig:status">14.7</a>).</li>
</ol>
<div class="figure" style="text-align: center"><span id="fig:status"></span>
<img src="figures/bioinformatics/job_id.png" alt="The BLASTN query status updated page." width="70%" />
<p class="caption">
Figure 14.7: The BLASTN query status updated page.
</p>
</div>
<ol start="7" style="list-style-type: decimal">
<li>Once the search has completed, the BLASTN results page will open (Fig. <a href="dna-sequence-analysis.html#fig:results">14.8</a>).</li>
</ol>
<div class="figure" style="text-align: center"><span id="fig:results"></span>
<img src="figures/bioinformatics/results.png" alt="The BLASTN results page." width="70%" />
<p class="caption">
Figure 14.8: The BLASTN results page.
</p>
</div>
<ol start="8" style="list-style-type: decimal">
<li>Click on the “+” sign in front of “Graphic Summary” (Fig. <a href="dna-sequence-analysis.html#fig:results">14.8</a>). A graphical summary of your results will be shown (Fig. <a href="dna-sequence-analysis.html#fig:graphic">14.9</a>. The light green line on top represents your query sequence, below are shown any retreived sequences that align with your sequence.</li>
</ol>
<div class="figure" style="text-align: center"><span id="fig:graphic"></span>
<img src="figures/bioinformatics/graphic_summary.png" alt="Graphic summary of the BLASTN search results." width="70%" />
<p class="caption">
Figure 14.9: Graphic summary of the BLASTN search results.
</p>
</div>
<ol start="9" style="list-style-type: decimal">
<li>Click on the “+” sign in front of “Description” (Fig. <a href="dna-sequence-analysis.html#fig:description">14.10</a>). A list of sequences that align with your query sequence will be shown (Fig. <a href="dna-sequence-analysis.html#fig:description">14.10</a>.</li>
</ol>
<div class="figure" style="text-align: center"><span id="fig:description"></span>
<img src="figures/bioinformatics/results_description.png" alt="List of sequences producing significant alignments with your query sequence." width="70%" />
<p class="caption">
Figure 14.10: List of sequences producing significant alignments with your query sequence.
</p>
</div>
<ol start="10" style="list-style-type: decimal">
<li>Click on the first sequence. A new page will showing the alignment of the query sequence with the retrieved sequence (Fig. <a href="dna-sequence-analysis.html#fig:alignment">14.11</a>).</li>
</ol>
<div class="figure" style="text-align: center"><span id="fig:alignment"></span>
<img src="figures/bioinformatics/alignment.png" alt="Alignment of the best retrieved sequence with the query sequence." width="70%" />
<p class="caption">
Figure 14.11: Alignment of the best retrieved sequence with the query sequence.
</p>
</div>
<ol start="11" style="list-style-type: decimal">
<li>Which sequence matches yours? How good is the alignment?</li>
</ol>
</div>
<div id="review-questions-9" class="section level2">
<h2><span class="header-section-number">14.4</span> Review Questions</h2>
<ol style="list-style-type: decimal">
<li>What are the ingredients of a sequencing reaction based on the chain-termination method (Sanger sequencing)?</li>
<li>What is GenBank?</li>
<li>What is BLAST?</li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="dna-sequencing.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="the-human-genome.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/genetics/edit/master/bioinformatics.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
